# translation of ja.po to Japanese
# anaconda/ja.po
# Copyright (C) 2001 Red Hat, Inc.
# SATO Satoru <ssato@redhat.com>
# James Hashida <khashida@redhat.com>, 2002,2003.
# Noriko Mizumoto <noriko@redhat.com>, 2003, 2004, 2006.
# Yoshinari Takaoka <mumumu@mx.edit.ne.jp>, 2004.
# Tadashi Jokagi <fedora@elf.no-ip.org>, 2004, 2005.
# Naoki Shigematsu <sigematu@linuxml.net>, 2004.
# Hirofumi Saito <hi_saito@yk.rim.or.jp>, 2004, 2005, 2006.
# Kiyoto James Hashida <khashida@redhat.com>, 2006, 2007.
# Hyu_gabaru Ryu_ichi <hyu_gabaru@yahoo.co.jp>, 2007.
# Kiyoto Hashida <khashida@redhat.com>, 2007.
# Makoto Mizukami <mmk9060@aol.com>, 2008.
#
msgid ""
msgstr ""
"Project-Id-Version: ja\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2008-03-28 10:06-0400\n"
"PO-Revision-Date: 2008-04-03 22:02+0900\n"
"Last-Translator: Makoto Mizukami <mmk9060@aol.com>\n"
"Language-Team: Japanese <fedora-trans-ja@redhat.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: ../anaconda:289
#, c-format
msgid "Error processing %%ksappend lines: %s"
msgstr "%%ksappend 行目を処理中にエラーが発生しました: %s"

#: ../anaconda:292
#, c-format
msgid "Unknown error processing %%ksappend lines: %s"
msgstr "%%ksappend 行目を処理中に不明なエラーが発生しました: %s"

#: ../anaconda:306 ../cmdline.py:78 ../gui.py:1187 ../text.py:509
#, c-format, python-format
msgid ""
"The following error was found while parsing your kickstart configuration:\n"
"\n"
"%s"
msgstr ""
"キックスタート設定の解析中に次のエラーが見つかりました。\n"
"\n"
"%s"

#: ../anaconda:423
msgid "Press <enter> for a shell"
msgstr "シェルを起動するには、<enter> を押してください。"

#: ../anaconda:438 ../gui.py:234 ../rescue.py:271 ../rescue.py:339
#: ../rescue.py:366 ../rescue.py:376 ../rescue.py:452 ../rescue.py:458
#: ../text.py:537 ../text.py:699 ../textw/constants_text.py:42
#: ../textw/network_text.py:44 ../textw/network_text.py:54
#: ../textw/network_text.py:76 ../textw/network_text.py:82
#: ../textw/network_text.py:230 ../textw/network_text.py:815
#: ../textw/network_text.py:823 ../loader2/cdinstall.c:145
#: ../loader2/cdinstall.c:146 ../loader2/cdinstall.c:200
#: ../loader2/cdinstall.c:203 ../loader2/cdinstall.c:342
#: ../loader2/cdinstall.c:347 ../loader2/cdinstall.c:351
#: ../loader2/cdinstall.c:421 ../loader2/dirbrowser.c:145
#: ../loader2/driverdisk.c:98 ../loader2/driverdisk.c:235
#: ../loader2/driverdisk.c:266 ../loader2/driverdisk.c:299
#: ../loader2/driverdisk.c:337 ../loader2/driverdisk.c:351
#: ../loader2/driverdisk.c:365 ../loader2/driverdisk.c:375
#: ../loader2/driverdisk.c:535 ../loader2/driverdisk.c:572
#: ../loader2/driverselect.c:75 ../loader2/driverselect.c:199
#: ../loader2/hdinstall.c:141 ../loader2/hdinstall.c:252
#: ../loader2/hdinstall.c:388 ../loader2/hdinstall.c:438
#: ../loader2/hdinstall.c:470 ../loader2/hdinstall.c:540
#: ../loader2/hdinstall.c:583 ../loader2/hdinstall.c:596 ../loader2/kbd.c:138
#: ../loader2/kickstart.c:132 ../loader2/kickstart.c:142
#: ../loader2/kickstart.c:185 ../loader2/kickstart.c:284
#: ../loader2/kickstart.c:372 ../loader2/kickstart.c:529 ../loader2/lang.c:115
#: ../loader2/lang.c:376 ../loader2/loader.c:344 ../loader2/loader.c:379
#: ../loader2/loader.c:414 ../loader2/loader.c:431 ../loader2/loader.c:445
#: ../loader2/loader.c:480 ../loader2/loader.c:997 ../loader2/loader.c:1159
#: ../loader2/mediacheck.c:62 ../loader2/mediacheck.c:101
#: ../loader2/mediacheck.c:108 ../loader2/mediacheck.c:117
#: ../loader2/method.c:120 ../loader2/method.c:346 ../loader2/method.c:420
#: ../loader2/modules.c:336 ../loader2/modules.c:348 ../loader2/net.c:277
#: ../loader2/net.c:320 ../loader2/net.c:750 ../loader2/net.c:1094
#: ../loader2/net.c:1647 ../loader2/net.c:1670 ../loader2/net.c:1863
#: ../loader2/nfsinstall.c:68 ../loader2/nfsinstall.c:153
#: ../loader2/nfsinstall.c:283 ../loader2/nfsinstall.c:301
#: ../loader2/nfsinstall.c:342 ../loader2/telnetd.c:93
#: ../loader2/urlinstall.c:78 ../loader2/urlinstall.c:101
#: ../loader2/urlinstall.c:185 ../loader2/urlinstall.c:196
#: ../loader2/urlinstall.c:437 ../loader2/urlinstall.c:446
#: ../loader2/urlinstall.c:455 ../loader2/urls.c:280 ../loader2/urls.c:315
#: ../loader2/urls.c:321 ../loader2/urls.c:333 ../loader2/urls.c:405
msgid "OK"
msgstr "OK"

#: ../anaconda:445
msgid ""
"You do not have enough RAM to use the graphical installer.  Starting text "
"mode."
msgstr ""
"グラフィカルインストーラーを使用するにはメモリが足りません。テキストモードで"
"起動します"

#: ../anaconda:460
msgid "No video hardware found, assuming headless"
msgstr "ビデオハードウェアが検出されませんでした。モニターがないとみなします。"

#: ../anaconda:467 ../anaconda:958
msgid "Unable to instantiate a X hardware state object."
msgstr "X ハードウェア状態オブジェクトのインスタンス生成ができません。"

#: ../anaconda:517
msgid "Starting graphical installation..."
msgstr "グラフィカルインストールを開始中..."

#: ../anaconda:838
msgid "Install class forcing text mode installation"
msgstr "テキストモードインストールを強制するインストールクラス"

#: ../anaconda:878
msgid "Graphical installation not available...  Starting text mode."
msgstr ""
"グラフィカルインストールは使用できません...テキストモードを起動中です。"

#: ../anaconda:886
msgid "DISPLAY variable not set. Starting text mode!"
msgstr "DISPLAY 変数がセットされていません。テキストモードで起動します!"

#: ../autopart.py:888
#, python-format
msgid ""
"Error resizing partition %s.\n"
"\n"
"%s"
msgstr ""
"パーティション %s をリサイズ中にエラーが発生しました。\n"
"\n"
"%s"

#: ../autopart.py:891
#, python-format
msgid "Start of partition %s was moved when resizing"
msgstr "パーティション %s の開始シリンダが、リサイズ中に移動されました。"

#: ../autopart.py:983
msgid "Could not allocate cylinder-based partitions as primary partitions.\n"
msgstr ""
"シリンダーベースのパーティションを基本パーティションとして割り当てることがで"
"きませんでした。\n"

#: ../autopart.py:988
msgid "Could not allocate partitions as primary partitions.\n"
msgstr ""
"パーティションを基本パーティションとして割り当てることができませんでした。\n"
"\n"

#: ../autopart.py:993
msgid "Could not allocate cylinder-based partitions.\n"
msgstr "シリンダーベースのパーティションを割り当てることができませんでした。\n"

#: ../autopart.py:1058
#, python-format
msgid ""
"Boot partition %s doesn't belong to a BSD disk label. SRM won't be able to "
"boot from this partition. Use a partition belonging to a BSD disk label or "
"change this device disk label to BSD."
msgstr ""
"ブートパーティション %s は BSD ディスクラベルに属しません。SRM はこのパーティ"
"ションからブートできません。 BSD ディスクラベルに属するパーティションを使用す"
"るか、このデバイスディスクラベルを BSD に変更してください。"

#: ../autopart.py:1060
#, python-format
msgid ""
"Boot partition %s doesn't belong to a disk with enough free space at its "
"beginning for the bootloader to live on. Make sure that there's at least 5MB "
"of free space at the beginning of the disk that contains /boot"
msgstr ""
"ブートパーティション %s は、ブートローダーが格納されるべき先頭部分に充分な空"
"き領域を持つディスクに属していません。ディスクの先頭に /boot を格納する空き領"
"域が最低 5 MB あることを確認してください。"

#: ../autopart.py:1062
#, python-format
msgid ""
"Boot partition %s isn't a VFAT partition.  EFI won't be able to boot from "
"this partition."
msgstr ""
"ブートパーティション %s は VFAT パーティションではありません。 EFI はこのパー"
"ティションから起動できません。"

#: ../autopart.py:1064
msgid ""
"The boot partition must entirely be in the first 4GB of the disk.  "
"OpenFirmware won't be able to boot this installation."
msgstr ""
"ブートパーティションは全体がディスクの先頭 4GB 以内に位置しなければなりませ"
"ん。オープンファームウェアをこのインストールでブートさせることはできません。"

#: ../autopart.py:1066
#, python-format
msgid ""
"Boot partition %s is not a Linux filesystem, such as ext3.  The system won't "
"be able to boot from this partition."
msgstr ""
"ブートパーティション %s は ext3 のようなLinux ファイルシステムではありませ"
"ん。システムをこのパーティションから起動することはできません。"

#: ../autopart.py:1069
#, python-format
msgid ""
"Boot partition %s may not meet booting constraints for your architecture."
msgstr ""
"ブートパーティション %s は、ご使用のアーキテクチャに関するブート制限を満たし"
"ていない可能性があります。"

#: ../autopart.py:1094
#, python-format
msgid ""
"Adding this partition would not leave enough disk space for already "
"allocated logical volumes in %s."
msgstr ""
"このパーティションを追加すると、既に割り当てられている %s 内の論理ボリューム"
"に十分な領域が残りません。"

#: ../autopart.py:1286
msgid "Requested Partition Does Not Exist"
msgstr "要求されたパーティションは存在しません"

#: ../autopart.py:1287
#, python-format
msgid ""
"Unable to locate partition %s to use for %s.\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""
"%s に使用するパーティション %s が見つかりません。\n"
"\n"
"OK を押すとインストーラを終了します。"

#: ../autopart.py:1312
msgid "Requested Raid Device Does Not Exist"
msgstr "要求された RAID デバイスは存在しません"

#: ../autopart.py:1313
#, python-format
msgid ""
"Unable to locate raid device %s to use for %s.\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""
"%s に使用する RAID デバイス %s が見つかりません。\n"
"\n"
"OK を押すとインストーラを終了します。"

#: ../autopart.py:1342
msgid "Requested Volume Group Does Not Exist"
msgstr "要求されたボリュームグループは存在しません"

#: ../autopart.py:1343
#, python-format
msgid ""
"Unable to locate volume group %s to use for %s.\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""
"%s に使用するボリュームグループ %s が見つかりません。\n"
"\n"
"OK を押すとインストーラを終了します。"

#: ../autopart.py:1380
msgid "Requested Logical Volume Does Not Exist"
msgstr "要求された論理ボリュームは存在しません"

#: ../autopart.py:1381
#, python-format
msgid ""
"Unable to locate logical volume %s to use for %s.\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""
"%s に使用する論理ボリューム %s が見つかりません。\n"
"\n"
"OK を押すとインストーラを終了します。"

#: ../autopart.py:1516 ../autopart.py:1563
msgid "Automatic Partitioning Errors"
msgstr "自動パーティション設定のエラーです"

#: ../autopart.py:1517
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%s\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""
"パーティション設定中に次のエラーが発生しました:\n"
"\n"
"%s\n"
"\n"
"OK を押すとインストーラを終了します。"

#: ../autopart.py:1527
msgid "Warnings During Automatic Partitioning"
msgstr "自動パーティション設定中の警告"

#: ../autopart.py:1528
#, python-format
msgid ""
"Following warnings occurred during automatic partitioning:\n"
"\n"
"%s"
msgstr ""
"自動パーティション設定中に次の警告が発生しました:\n"
"\n"
"%s"

#: ../autopart.py:1542 ../autopart.py:1559
msgid ""
"\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""
"\n"
"\n"
"「OK」を押すとインストーラを終了します。"

#: ../autopart.py:1543 ../iw/partition_gui.py:1023
#: ../textw/partition_text.py:246
msgid "Error Partitioning"
msgstr "パーティション設定エラー"

#: ../autopart.py:1544
#, python-format
msgid ""
"Could not allocate requested partitions: \n"
"\n"
"%s.%s"
msgstr ""
"要求されたパーティションを設定できませんでした: \n"
"\n"
"%s.%s"

#: ../autopart.py:1561
msgid ""
"\n"
"\n"
"Press 'OK' to choose a different partitioning option."
msgstr ""
"\n"
"\n"
"「OK」を押して別のパーティション設定オプションを選択してください。"

#: ../autopart.py:1564
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%s\n"
"\n"
"This can happen if there is not enough space on your hard drive(s) for the "
"installation. %s"
msgstr ""
"パーティション作成中に次のエラーが発生しました:\n"
"\n"
"%s\n"
"\n"
"このエラーは、インストールに十分な領域がハードドライブ上にない場合に発生しま"
"す。%s"

#: ../autopart.py:1575
msgid "Unrecoverable Error"
msgstr "修復不能なエラー"

#: ../autopart.py:1576
msgid "Your system will now be rebooted."
msgstr "システムが再起動します。"

#: ../autopart.py:1700
msgid ""
"Automatic Partitioning sets partitions based on the selected installation "
"type. You also can customize the partitions once they have been created.\n"
"\n"
"The manual disk partitioning tool, Disk Druid, allows you to create "
"partitions in an interactive environment. You can set the file system types, "
"mount points, partition sizes, and more."
msgstr ""
"自動パーティション設定は選択したインストールタイプに応じてパーティション設定"
"を行います。作成されたパーティションをさらにカスタマイズすることもできま"
"す。\n"
"\n"
"手動ディスクパーティション設定ツールの Disk Druid を使用すると対話式の環境で"
"パーティションを設定することができます。ファイルシステムの種類、マウントポイ"
"ント、パーティションのサイズ、その他が設定できます。"

#: ../autopart.py:1711
msgid ""
"Before automatic partitioning can be set up by the installation program, you "
"must choose how to use the space on your hard drives."
msgstr ""
"インストールプログラムで自動パーティション設定する前に、ハードドライブの領域"
"の使用法を選択しておく必要があります。"

#: ../autopart.py:1716
msgid "Remove all partitions on this system"
msgstr "このシステムのすべてのパーティションを削除"

#: ../autopart.py:1717
msgid "Remove all Linux partitions on this system"
msgstr "このシステムのすべての Linux パーティションを削除"

#: ../autopart.py:1718
msgid "Keep all partitions and use existing free space"
msgstr "すべてのパーティションを保持し、既存の空き領域を使用"

#: ../backend.py:126
#, python-format
msgid "Upgrading %s\n"
msgstr "%s のアップグレード\n"

#: ../backend.py:128
#, python-format
msgid "Installing %s\n"
msgstr "%s のインストール\n"

#: ../bootloader.py:44 ../bootloader.py:236 ../image.py:87
#: ../partedUtils.py:365 ../partedUtils.py:395 ../partedUtils.py:1106
#: ../partedUtils.py:1259 ../upgrade.py:432 ../yuminstall.py:1463
#: ../yuminstall.py:1490 ../iw/autopart_type.py:180 ../iw/blpasswidget.py:149
#: ../iw/task_gui.py:53 ../iw/upgrade_swap_gui.py:192
#: ../iw/upgrade_swap_gui.py:200 ../iw/upgrade_swap_gui.py:207
#: ../textw/bootloader_text.py:127 ../textw/bootloader_text.py:452
#: ../textw/partition_text.py:250 ../textw/upgrade_text.py:186
msgid "Warning"
msgstr "警告"

#: ../bootloader.py:45
msgid ""
"Your filesystems have already been activated.  You cannot go back past this "
"point.\n"
"\n"
"Would you like to continue with the installation?"
msgstr ""
"指定されたパーティション設定オプションが有効になっています。ここからディスク"
"編集画面に戻ることはできません。インストールを続けますか?"

#: ../bootloader.py:49 ../exception.py:348 ../exception.py:363
#: ../exception.py:381 ../fsset.py:1715 ../fsset.py:1972 ../fsset.py:2711
#: ../fsset.py:2718 ../gui.py:1192 ../gui.py:1326 ../gui.py:1403
#: ../image.py:96 ../livecd.py:361 ../packages.py:152 ../upgrade.py:113
#: ../upgrade.py:133 ../yuminstall.py:718 ../yuminstall.py:793
#: ../yuminstall.py:799 ../yuminstall.py:970 ../yuminstall.py:1018
#: ../yuminstall.py:1251 ../yuminstall.py:1288
msgid "_Exit installer"
msgstr "インストーラを終了する(_E)"

#: ../bootloader.py:49 ../fsset.py:1973 ../image.py:97 ../image.py:250
#: ../kickstart.py:985 ../kickstart.py:1023 ../partedUtils.py:1262
#: ../upgrade.py:113 ../upgrade.py:133 ../yuminstall.py:975
#: ../iw/partition_gui.py:1037
msgid "_Continue"
msgstr "続行(_C)"

#: ../bootloader.py:167
msgid "Bootloader"
msgstr "ブートローダー"

#: ../bootloader.py:167
msgid "Installing bootloader..."
msgstr "ブートローダーをインストール中です..."

#: ../bootloader.py:237
msgid ""
"No kernel packages were installed on your system.  Your boot loader "
"configuration will not be changed."
msgstr ""
"システムにカーネルパッケージはインストールされなかったため、ブートローダー設"
"定は変更されません。"

#: ../cmdline.py:53
msgid "Completed"
msgstr "完了"

#: ../cmdline.py:61
msgid "In progress...   "
msgstr "進捗中...   "

#: ../cmdline.py:90
msgid "Can't have a question in command line mode!"
msgstr "コマンドラインモードでは質問できません!"

#: ../cmdline.py:109
msgid "Parted exceptions can't be handled in command line mode!"
msgstr "Parted の例外はコマンドラインモードでは処理できません!"

#: ../constants.py:75
msgid ""
"An unhandled exception has occurred.  This is most likely a bug.  Please "
"save a copy of the detailed exception and file a bug report"
msgstr ""
"未処理の例外が発生しました。バグの可能性が考えられます。全てのメッセージをコ"
"ピーし、バグとして報告してください。"

#: ../constants.py:81
msgid " with the provider of this software."
msgstr "このソフトウェアの供給者と共に"

#: ../constants.py:85
#, python-format
msgid " against anaconda at %s"
msgstr "anaconda の %s に対して"

#: ../exception.py:344 ../exception.py:359 ../exception.py:377
msgid "Dump Written"
msgstr "dump が書き込まれました"

#: ../exception.py:345 ../exception.py:360
msgid ""
"Your system's state has been successfully written to the disk. The installer "
"will now exit."
msgstr ""
"システム状態のディスク書き込みに成功しました。インストーラを終了します。"

#: ../exception.py:351 ../exception.py:368 ../exception.py:384
msgid "Dump Not Written"
msgstr "dump が書き込まれませんでした"

#: ../exception.py:352 ../exception.py:369
msgid "There was a problem writing the system state to the disk."
msgstr "システム状態を書き込んでいる際に問題が発生しました。"

#: ../exception.py:378
msgid ""
"Your system's state has been successfully written to the remote host.  The "
"installer will now exit."
msgstr ""
"システムの状態が正常にリモートホストに書き込まれました。インストーラを終了し"
"ます。"

#: ../exception.py:385
msgid "There was a problem writing the system state to the remote host."
msgstr "リモートホストにシステム状態を書き込んでいる際に問題が発生しました。"

#: ../fsset.py:548
msgid "Checking"
msgstr "チェック中"

#: ../fsset.py:549
#, python-format
msgid "Checking filesystem on %s..."
msgstr "/dev/%s のファイルシステムを検査中..."

#: ../fsset.py:560 ../fsset.py:1015
msgid "Resizing"
msgstr "リサイズ中"

#: ../fsset.py:561 ../fsset.py:1016
#, python-format
msgid "Resizing filesystem on %s..."
msgstr "ファイルシステム %s のリサイズ中..."

#: ../fsset.py:704 ../fsset.py:1685 ../fsset.py:1716 ../fsset.py:1792
#: ../fsset.py:1860 ../fsset.py:1910 ../fsset.py:1995 ../fsset.py:2008
#: ../image.py:267 ../livecd.py:354 ../partIntfHelpers.py:412 ../text.py:445
#: ../yuminstall.py:374 ../yuminstall.py:438 ../yuminstall.py:609
#: ../yuminstall.py:714 ../yuminstall.py:981 ../yuminstall.py:1022
#: ../yuminstall.py:1256 ../yuminstall.py:1281 ../iw/autopart_type.py:95
#: ../iw/autopart_type.py:170 ../iw/autopart_type.py:314
#: ../iw/netconfig_dialog.py:265 ../iw/osbootwidget.py:214
#: ../iw/osbootwidget.py:223 ../iw/raid_dialog_gui.py:671
#: ../iw/raid_dialog_gui.py:710 ../iw/task_gui.py:43 ../iw/task_gui.py:231
#: ../textw/grpselect_text.py:123 ../textw/partition_text.py:1651
#: ../textw/partition_text.py:1657 ../textw/partition_text.py:1679
#: ../textw/upgrade_text.py:174 ../textw/upgrade_text.py:181
#: ../loader2/cdinstall.c:146 ../loader2/cdinstall.c:421
#: ../loader2/driverdisk.c:98 ../loader2/driverdisk.c:299
#: ../loader2/driverdisk.c:337 ../loader2/driverdisk.c:365
#: ../loader2/driverdisk.c:375 ../loader2/driverdisk.c:439
#: ../loader2/hdinstall.c:141 ../loader2/hdinstall.c:252
#: ../loader2/hdinstall.c:438 ../loader2/hdinstall.c:540
#: ../loader2/hdinstall.c:583 ../loader2/hdinstall.c:596
#: ../loader2/kickstart.c:284 ../loader2/lang.c:115 ../loader2/loader.c:344
#: ../loader2/loader.c:445 ../loader2/loader.c:997 ../loader2/mediacheck.c:62
#: ../loader2/mediacheck.c:101 ../loader2/mediacheck.c:108
#: ../loader2/method.c:120 ../loader2/method.c:346 ../loader2/method.c:420
#: ../loader2/nfsinstall.c:153 ../loader2/nfsinstall.c:283
#: ../loader2/nfsinstall.c:301 ../loader2/telnetd.c:93
#: ../loader2/urlinstall.c:78 ../loader2/urlinstall.c:101
#: ../loader2/urlinstall.c:185 ../loader2/urlinstall.c:196
#: ../loader2/urls.c:315 ../loader2/urls.c:321
msgid "Error"
msgstr "エラー"

#: ../fsset.py:705
#, python-format
msgid ""
"An error occurred migrating %s to ext3.  It is possible to continue without "
"migrating this file system if desired.\n"
"\n"
"Would you like to continue without migrating %s?"
msgstr ""
"%s を ext3 に転換する際にエラーが発生しました。このファイルシステムを転換させ"
"ないで続行することも可能です。\n"
"\n"
"%s を転換しないで続行しますか?"

#: ../fsset.py:1538
msgid "EFI System Partition"
msgstr "EFI システムパーティション"

#: ../fsset.py:1542
msgid "RAID Device"
msgstr "RAID デバイス"

#: ../fsset.py:1546 ../fsset.py:1552
msgid "Apple Bootstrap"
msgstr "Apple ブートストラップ"

#: ../fsset.py:1557 ../partitions.py:1121
msgid "PPC PReP Boot"
msgstr "PPC PReP ブート"

#: ../fsset.py:1560
msgid "First sector of boot partition"
msgstr "ブートパーティションの最初のセクタ"

#: ../fsset.py:1561
msgid "Master Boot Record (MBR)"
msgstr "マスタブートレコード(MBR)"

#: ../fsset.py:1686
#, python-format
msgid ""
"An error occurred trying to initialize swap on device %s.  This problem is "
"serious, and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"デバイス %s 上の swap を初期化中にエラーが発生しました。深刻な問題となるため"
"インストールを続行できません。\n"
"\n"
"[Enter] を押してインストーラを終了してください。"

#: ../fsset.py:1715 ../packages.py:368 ../rescue.py:305 ../rescue.py:307
#: ../textw/upgrade_text.py:132 ../loader2/cdinstall.c:200
#: ../loader2/cdinstall.c:203 ../loader2/method.c:393
msgid "Skip"
msgstr "スキップする"

#: ../fsset.py:1736
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     /dev/%s\n"
"\n"
"is a version 0 Linux swap partition. If you want to use this device, you "
"must reformat as a version 1 Linux swap partition. If you skip it, the "
"installer will ignore it during the installation."
msgstr ""
" swap デバイス:\n"
"\n"
"     /dev/%s\n"
"\n"
"はバージョン 0 Linux  swap パーティションです。このデバイスを使用する場合は、"
"バージョン 1 Linux  swap パーティションとして再フォーマットする必要がありま"
"す。省略すると、インストーラーはインストール過程でこれを無視します。"

#: ../fsset.py:1743
msgid "Reformat"
msgstr "再フォーマットする"

#: ../fsset.py:1747
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     /dev/%s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend partition, "
"which means your system is hibernating. To perform an upgrade, please shut "
"down your system rather than hibernating it."
msgstr ""
" swap デバイス:\n"
"\n"
"     /etc/fstab ファイル内の /dev/%s\n"
"\n"
"は現在、ソフトウェアサスペンドパーティションとして使用されており、これはシス"
"テムがハイバネート状態であることを意味します。アップグレードを実行するには、"
"システムをハイバネートするのではなく、シャットダウンしてください。"

#: ../fsset.py:1755
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     /dev/%s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend partition, "
"which means your system is hibernating. If you are performing a new install, "
"make sure the installer is set to format all swap partitions."
msgstr ""
" /etc/fstab ファイル内の swap デバイス:\n"
"\n"
"     /dev/%s\n"
"\n"
"は現在、ソフトウェアサスペンドパーティションとして使用されており、これはシス"
"テムがハイバネート状態であることを意味します。新規のインストールを実行してい"
"る場合は、インストーラーがすべての swap パーティションをフォーマットするよう"
"設定されていることを確認してください。"

#: ../fsset.py:1765
msgid ""
"\n"
"\n"
"Choose Skip if you want the installer to ignore this partition during the "
"upgrade.  Choose Format to reformat the partition as swap space."
msgstr ""
"\n"
"\n"
"アップグレード中、インストーラーにこのパーティションを無視させたい場合は、"
"「スキップする」を選択します。「フォーマットする」を選択してこのパーティショ"
"ンを swap 領域として再フォーマットします。"

#: ../fsset.py:1770 ../iw/partition_gui.py:373
msgid "Format"
msgstr "フォーマットする"

#: ../fsset.py:1776
#, python-format
msgid ""
"Error enabling swap device %s: %s\n"
"\n"
"Devices in /etc/fstab should be specified by label, not by device name.\n"
"\n"
"Press OK to exit the installer."
msgstr ""
"スワップデバイス %s を有効にしている際にエラーが発生しました: %s\n"
"\n"
"/etc/fstab 内のデバイスはデバイス名ではなく、ラベルで指定されるべきです。\n"
"\n"
"「OK」を押してインストーラを終了してください。"

#: ../fsset.py:1781
#, python-format
msgid ""
"Error enabling swap device %s: %s\n"
"\n"
"The /etc/fstab on your upgrade partition does not reference a valid swap "
"partition.\n"
"\n"
"Press OK to exit the installer"
msgstr ""
"スワップデバイス %s を有効にしている際にエラーが発生しました: %s\n"
"\n"
"アップグレードしたパーティションの /etc/fstab が有効な スワップパーティション"
"を定義していません。\n"
"\n"
"OK を押してインストーラを終了してください。"

#: ../fsset.py:1787
#, python-format
msgid ""
"Error enabling swap device %s: %s\n"
"\n"
"This most likely means this swap partition has not been initialized.\n"
"\n"
"Press OK to exit the installer."
msgstr ""
" swap デバイス %s を有効にしている際にエラー発生: %s\n"
"\n"
"swap パーティションが初期化されていない可能性があります。\n"
"\n"
"[OK] を押してインストーラを終了してください。"

#: ../fsset.py:1861
#, python-format
msgid ""
"An error occurred trying to format %s.  This problem is serious, and the "
"install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"%s をフォーマット中にエラーが発生しました。深刻な問題となるためインストールを"
"続行できません\n"
"\n"
"<Enter> を押してインストーラを終了してください。"

#: ../fsset.py:1911
#, python-format
msgid ""
"An error occurred trying to migrate %s.  This problem is serious, and the "
"install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"%s を転換中にエラーが発生しました。深刻な問題のためインストールを続行できませ"
"ん。\n"
"\n"
"<Enter> を押してインストーラを終了してください。"

#: ../fsset.py:1940 ../fsset.py:1949
msgid "Invalid mount point"
msgstr "無効なマウントポイント"

#: ../fsset.py:1941
#, python-format
msgid ""
"An error occurred when trying to create %s.  Some element of this path is "
"not a directory. This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"%s を作成中にエラーが発生しました。このパスのどこかがディレクトリではありませ"
"ん。致命的なエラーとなるためインストールを続行できません。\n"
"\n"
"<Enter> を押してインストーラを終了してください。"

#: ../fsset.py:1950
#, python-format
msgid ""
"An error occurred when trying to create %s: %s.  This is a fatal error and "
"the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"%s を作成中にエラーが発生しました: %s。致命的なエラーとなるためインストールを"
"続行できません。\n"
"\n"
"<Enter> を押してインストーラを終了してください。"

#: ../fsset.py:1964
msgid "Unable to mount filesystem"
msgstr "ファイルシステムをマウントできません"

#: ../fsset.py:1965
#, python-format
msgid ""
"An error occurred mounting device %s as %s.  You may continue installation, "
"but there may be problems."
msgstr ""
"デバイス %s を %s としてマウントする際にエラーが発生しました。インストールは"
"続行できますが、問題が生じる場合があります。"

#: ../fsset.py:1981
#, python-format
msgid ""
"Error mounting device %s as %s: %s\n"
"\n"
"Devices in /etc/fstab should be specified by label, not by device name.\n"
"\n"
"Press OK to exit the installer."
msgstr ""
"デバイス %s を %s としてマウントする際にエラーが発生しました:%s。\n"
"\n"
"/etc/fstab 内のデバイスはデバイス名ではなく、ラベルで指定されるべきです。\n"
"\n"
"「OK」を押してインストーラを終了してください。"

#: ../fsset.py:1988
#, python-format
msgid ""
"Error mounting device %s as %s: %s\n"
"\n"
"This most likely means this partition has not been formatted.\n"
"\n"
"Press OK to exit the installer."
msgstr ""
"デバイス %s を %s としてマウントする際にエラーが発生しました:%s。\n"
"\n"
"このパーティションがフォーマットされていない可能性があります。\n"
"\n"
"「OK」を押してインストーラを終了してください。"

#: ../fsset.py:2009
msgid ""
"Error finding / entry.\n"
"\n"
"This is most likely means that your fstab is incorrect.\n"
"\n"
"Press OK to exit the installer."
msgstr ""
"/ エントリを検索中にエラーが発生しました。\n"
"\n"
"fstab が正しくない可能性があります。\n"
"\n"
"OK を押してインストーラを終了してください。"

#: ../fsset.py:2703
msgid "Duplicate Labels"
msgstr "ラベルの重複"

#: ../fsset.py:2704
#, python-format
msgid ""
"Multiple devices on your system are labelled %s.  Labels across devices must "
"be unique for your system to function properly.\n"
"\n"
"Please fix this problem and restart the installation process."
msgstr ""
"システム上の複数のデバイスに %s というラベルが付けられています。システムが正"
"しく機能するためには、デバイスに固有のラベルを付ける必要があります。\n"
"\n"
"この問題を解決してからインストールをやり直してください。"

#: ../fsset.py:2713
msgid "Invalid Label"
msgstr "無効なラベル"

#: ../fsset.py:2714
#, python-format
msgid ""
"An invalid label was found on device %s.  Please fix this problem and "
"restart the installation process."
msgstr ""
"デバイス %s に不当なラベルが付けられています。この問題を解決してからインス"
"トールをやり直してください。"

#: ../fsset.py:2877
msgid "Formatting"
msgstr "フォーマット中"

#: ../fsset.py:2878
#, python-format
msgid "Formatting %s file system..."
msgstr "%s ファイルシステムのフォーマット中..."

#: ../gui.py:106
msgid "An error occurred copying the screenshots over."
msgstr "スクリーンショットのコピー中にエラーが発生しました。"

#: ../gui.py:117
msgid "Screenshots Copied"
msgstr "スクリーンショットがコピーされました"

#: ../gui.py:118
msgid ""
"The screenshots have been saved into the directory:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"You can access these when you reboot and login as root."
msgstr ""
"スクリーンショットは次のディレクトリに保存されました。\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"再起動して root でログインするとアクセスできます。"

#: ../gui.py:161
msgid "Saving Screenshot"
msgstr "スクリーンショットを保存中"

#: ../gui.py:162
#, python-format
msgid "A screenshot named '%s' has been saved."
msgstr "「%s」 と言う名前のスクリーンショットが保存されました。"

#: ../gui.py:165
msgid "Error Saving Screenshot"
msgstr "スクリーンショット保存中にエラー発生"

#: ../gui.py:166
msgid ""
"An error occurred while saving the screenshot.  If this occurred during "
"package installation, you may need to try several times for it to succeed."
msgstr ""
"スクリーンショット保存中にエラーが発生しました。パッケージインストール中に発"
"生した場合、完了するには数回試行する必要があるかもしれません。"

#: ../gui.py:231 ../text.py:534
msgid "Fix"
msgstr "修正"

#: ../gui.py:232 ../rescue.py:228 ../text.py:535
#: ../textw/bootloader_text.py:72 ../textw/constants_text.py:54
#: ../loader2/driverdisk.c:479 ../loader2/driverdisk.c:489
#: ../loader2/hdinstall.c:334
msgid "Yes"
msgstr "はい"

#: ../gui.py:233 ../rescue.py:228 ../rescue.py:230 ../text.py:536
#: ../textw/bootloader_text.py:72 ../textw/constants_text.py:58
#: ../loader2/driverdisk.c:479 ../loader2/driverdisk.c:489
msgid "No"
msgstr "いいえ"

#: ../gui.py:235 ../text.py:538 ../text.py:669 ../loader2/net.c:92
#: ../loader2/net.c:338 ../loader2/net.c:589 ../loader2/net.c:695
#: ../loader2/net.c:820 ../loader2/net.c:828 ../loader2/net.c:1224
#: ../loader2/net.c:1230
msgid "Retry"
msgstr "再試行"

#: ../gui.py:236 ../text.py:539
msgid "Ignore"
msgstr "無視"

#: ../gui.py:237 ../gui.py:856 ../gui.py:1403 ../partIntfHelpers.py:244
#: ../partIntfHelpers.py:535 ../text.py:108 ../text.py:109 ../text.py:540
#: ../textw/bootloader_text.py:202 ../textw/constants_text.py:46
#: ../loader2/dirbrowser.c:145 ../loader2/driverdisk.c:236
#: ../loader2/kickstart.c:372 ../loader2/loader.c:379 ../loader2/loader.c:480
msgid "Cancel"
msgstr "取り消し"

#: ../gui.py:582 ../text.py:466
msgid "Installation Key"
msgstr "インストールキー"

#: ../gui.py:656 ../gui.py:664
msgid "Error with passphrase"
msgstr "パスフレーズが違います。"

#: ../gui.py:657
msgid "The passphrases you entered were different.  Please try again."
msgstr "入力したパスフレーズが異ります。やり直してください。"

#: ../gui.py:665
msgid "The passphrase must be at least eight characters long."
msgstr "パスフレーズは最低 6 文字以上でなければなりません。"

#: ../gui.py:691 ../text.py:276
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""
"デバイス %s は暗号化されています。インストール中にこのデバイスの内容にアクセ"
"スするには、以下にデバイスのパスフレーズを入力する必要があります。"

#: ../gui.py:868 ../gui.py:869 ../gui.py:981 ../gui.py:982
#: tmp/anaconda.glade.h:2
msgid "_Debug"
msgstr "デバッグ(_D)"

#: ../gui.py:1025 ../gui.py:1027 ../gui.py:1400 ../livecd.py:112
msgid "Exit installer"
msgstr "インストーラーを終了する"

#: ../gui.py:1027 ../text.py:254 ../text.py:262
msgid "Debug"
msgstr "デバッグ"

#: ../gui.py:1029 ../text.py:258
msgid "Exception Occurred"
msgstr "例外が発生しました"

#: ../gui.py:1189 ../text.py:511
msgid "Error Parsing Kickstart Config"
msgstr "キックスタート設定ファイル解析エラー"

#: ../gui.py:1235
msgid "default:LTR"
msgstr "デフォルト:LTR"

#: ../gui.py:1315 ../text.py:664
msgid "Error!"
msgstr "エラー!"

#: ../gui.py:1316 ../text.py:665
#, python-format
msgid ""
"An error occurred when attempting to load an installer interface component.\n"
"\n"
"className = %s"
msgstr ""
"インストーラーインターフェースコンポーネントを読み込もうとしてエラーが発生し"
"ました。\n"
"\n"
"className = %s"

#: ../gui.py:1321 ../image.py:165 ../image.py:191 ../packages.py:417
#: ../packages.py:422
msgid "_Exit"
msgstr "終了(_E)"

#: ../gui.py:1322 ../image.py:191 ../yuminstall.py:606 ../yuminstall.py:970
#: ../yuminstall.py:1018 ../yuminstall.py:1251
msgid "_Retry"
msgstr "再試行(_R)"

#: ../gui.py:1325 ../packages.py:189 ../packages.py:421
msgid "The installer will now exit..."
msgstr "インストーラーが終了します..."

#: ../gui.py:1328 ../packages.py:424
msgid "Your system will now be rebooted..."
msgstr "システムが再起動します..."

#: ../gui.py:1329 ../image.py:250 ../packages.py:190 ../packages.py:425
#: ../partedUtils.py:1261 ../yuminstall.py:1304
msgid "_Reboot"
msgstr "再起動(_R)"

#: ../gui.py:1331
msgid "Exiting"
msgstr "終了"

#: ../gui.py:1401
msgid "Are you sure you wish to exit the installation?"
msgstr "本当にインストールを終了しますか?"

#: ../gui.py:1410
#, python-format
msgid "%s Installer"
msgstr "%s インストーラー"

#: ../gui.py:1417
msgid "Unable to load title bar"
msgstr "タイトルバーを読み込めません"

#: ../gui.py:1470
msgid "Install Window"
msgstr "インストールウィンドウ"

#: ../image.py:88
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error."
msgstr ""
"この ISO イメージ %s のサイズは、2048 バイトの倍数ではありません。これは、こ"
"のコンピューターへの転送の際に、それが破壊されたことを意味します。\n"
"\n"
"終了して、インストールを中止することをお勧めしますが、もしこれが間違いだと思"
"う場合は、続行することができます。 "

#: ../image.py:157
msgid "Couldn't Mount ISO Source"
msgstr "ISO ソースをマウントできない"

#: ../image.py:158
#, python-format
msgid ""
"An error occurred mounting the source device %s.  This may happen if your "
"ISO images are located on an advanced storage device like LVM or RAID, or if "
"there was a problem mounting a partition.  Click exit to abort the "
"installation."
msgstr ""
"ソースデバイス %s のマウントでエラーが起きました。LVM や RAID のような進化し"
"たストレージ装置に ISO イメージが置かれたときや、パーティションのマウントに問"
"題がある場合に発生します。「終了」をクリックしてインストールを中止してくださ"
"い。"

#: ../image.py:182
msgid "Missing ISO 9660 Image"
msgstr "ISO 9660 イメージがありません"

#: ../image.py:183
#, python-format
msgid ""
"The installer has tried to mount image #%s, but cannot find it on the hard "
"drive.\n"
"\n"
"Please copy this image to the drive and click Retry. Click Exit  to abort "
"the installation."
msgstr ""
"インストーラーはイメージ #%s のマウントを試行しましたが、ハードドライブ上に見"
"つかりませんでした。\n"
"\n"
"このイメージをドライブにコピーしてから再試行をクリックしてください。「終了」"
"をクリックするとインストールを中止します。"

#: ../image.py:241
msgid "Required Install Media"
msgstr "必要なインストールメディア"

#: ../image.py:242
#, python-format
msgid ""
"The software you have selected to install will require the following discs:\n"
"\n"
"%s\n"
"Please have these ready before proceeding with the installation.  If you "
"need to abort the installation and exit please select \"Reboot\"."
msgstr ""
"選択したソフトウェアをインストールするには次の ディスク が必要です。\n"
"\n"
"%s\n"
"インストールを続行する前にこれらの ディスク を用意してください。インストール"
"を中止して操作を終了する必要がある場合は「再起動」を選択します。"

#: ../image.py:250 ../livecd.py:360 ../packages.py:371 ../packages.py:422
#: ../packages.py:425 ../yuminstall.py:718 ../yuminstall.py:799
#: ../yuminstall.py:1288 ../yuminstall.py:1304
msgid "_Back"
msgstr "戻る(_B)"

#: ../image.py:268
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""
"ディスク をアンマウント中にエラーが発生しました。tty2 のシェルから %s にアク"
"セスしていないことを確認してから [OK] をクリックして再試行してください。"

#: ../installclass.py:71
msgid "Install on System"
msgstr "システム上にインストール"

#: ../iscsi.py:205 ../iscsi.py:206
msgid "Initializing iSCSI initiator"
msgstr "iSCSI イニシエーターの初期化中"

#: ../kickstart.py:86
#, python-format
msgid ""
"There was an error running the kickstart script at line %s.  You may examine "
"the output in %s.  This is a fatal error and your install will be aborted.  "
"Press the OK button to exit the installer."
msgstr ""
"キックスタートスクリプトの %s 行目の処理中にエラーが発生しました。%s 内の出力"
"を調査した方が良いでしょう。致命的なエラーの為、インストールは中止されま"
"す。\n"
"OK を押すとインストーラーを終了します。"

#: ../kickstart.py:101 ../kickstart.py:103
msgid "Scriptlet Failure"
msgstr "Scriptlet の失敗"

#: ../kickstart.py:928 ../kickstart.py:945
msgid "Running..."
msgstr "実行中..."

#: ../kickstart.py:929
msgid "Running post-install scripts"
msgstr "post-install スクリプトを実行中"

#: ../kickstart.py:946
msgid "Running pre-install scripts"
msgstr "pre-install スクリプトを実行中"

#: ../kickstart.py:977
msgid "Missing Package"
msgstr "見つからないパッケージ"

#: ../kickstart.py:978
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist. Would you like to continue or abort your installation?"
msgstr ""
"パッケージ '%s'をインストールするように指定しています。このパッケージは存在し"
"ません。インストールを続行しますか、中止しますか?"

#: ../kickstart.py:984 ../kickstart.py:1022
msgid "_Abort"
msgstr "中止(_A)"

#: ../kickstart.py:1014
msgid "Missing Group"
msgstr "見つからないグループ"

#: ../kickstart.py:1015
#, python-format
msgid ""
"You have specified that the group '%s' should be installed. This group does "
"not exist. Would you like to continue or abort your installation?"
msgstr ""
"グループ '%s'をインストールするように指定しています。このグループは存在しませ"
"ん。インストールを継続しますか、中止しますか?"

#: ../livecd.py:107
msgid "Unable to find image"
msgstr "イメージを検出できません。"

#: ../livecd.py:108
#, python-format
msgid ""
"The given location isn't a valid %s live CD to use as an installation source."
msgstr ""
"与えられた場所は、インストールソースとして使用するための有効な %s ライブ ディ"
"スク ではありません。"

#: ../livecd.py:167
msgid "Copying live image to hard drive."
msgstr "ライブイメージをハードドライブに転送中。"

#: ../livecd.py:203
msgid "Doing post-installation"
msgstr "ポストインストールの実行"

#: ../livecd.py:204
msgid ""
"Performing post-installation filesystem changes.  This may take several "
"minutes..."
msgstr ""
"ポストインストールのファイルシステムの変更を開始しています。これには、数分か"
"かることがあります..."

#: ../livecd.py:355
#, python-format
msgid ""
"The root filesystem you created is not large enough for this live image "
"(%.2f MB required)."
msgstr ""
"作成したルートファイルシステムのサイズが、このライブイメージが必要としている"
"サイズより小さすぎます(最低 %.2f MB は必要です)。"

#: ../network.py:57
msgid "Hostname must be 64 or less characters in length."
msgstr "ホスト名は 64 文字以下の長さにしてください。"

#: ../network.py:60
msgid "Hostname must start with a valid character in the range 'a-z' or 'A-Z'"
msgstr "ホスト名は 「a-z」 又は 「A-Z」 の範囲の有効な文字で始めてください。"

#: ../network.py:65
msgid "Hostnames can only contain the characters 'a-z', 'A-Z', '-', or '.'"
msgstr ""
"ホスト名には 「a-z」、 「A-Z」、 「-」、 「.」以外の文字は使用できません。"

#: ../network.py:95
msgid "IP address is missing."
msgstr "IP アドレスがありません。"

#: ../network.py:99
msgid ""
"IPv4 addresses must contain four numbers between 0 and 255, separated by "
"periods."
msgstr ""
"IPv4 アドレスは、ピリオドで区切られた 0 から 255 までの 4 つの数字を含まなけ"
"ればなりません。"

#: ../network.py:102
#, python-format
msgid "'%s' is not a valid IPv6 address."
msgstr "%s は有効な IPv6 アドレスではありません。"

#: ../network.py:104
#, python-format
msgid "'%s' is an invalid IP address."
msgstr "'%s' は、有効な IP アドレスではありません。"

#: ../packages.py:147
msgid "Resizing Failed"
msgstr "リサイズに失敗"

#: ../packages.py:148
#, python-format
msgid "There was an error encountered resizing the device %s."
msgstr "%s のリサイズ中にエラーが発生しました。"

#: ../packages.py:188
msgid "LVM operation failed"
msgstr "LVM の操作に失敗しました。"

#: ../packages.py:342
msgid "Invalid Key"
msgstr "無効なキー"

#: ../packages.py:343
msgid "The key you entered is invalid."
msgstr "入力したキーは無効です。"

#: ../packages.py:371
msgid "_Skip"
msgstr "スキップする (_S)"

#: ../packages.py:403 ../packages.py:426
msgid "Warning! This is pre-release software!"
msgstr "警告! これはベータ版のソフトウェアです!"

#: ../packages.py:404
#, python-format
msgid ""
"Thank you for downloading this pre-release of %s.\n"
"\n"
"This is not a final release and is not intended for use on production "
"systems.  The purpose of this release is to collect feedback from testers, "
"and it is not suitable for day to day usage.\n"
"\n"
"To report feedback, please visit:\n"
"\n"
"   %s\n"
"\n"
"and file a report against '%s'.\n"
msgstr ""
"本 %s のプレリリースをダウンロードして頂きありがとうございます。\n"
"\n"
"これは最終リリースではありません。また実稼働用システムでの使用を目的としたも"
"のではありません。このリリースの目的は、テスターからのフィードバックを収集す"
"ることであり、日常の使用には適しません。\n"
"\n"
"フィードバックを報告するには\n"
"\n"
"    %s\n"
"\n"
"上記ののサイトに行き、「%s」に対して報告をファイルしてください。\n"

#: ../packages.py:417
msgid "_Install anyway"
msgstr "とにかくインストールする(_I)"

#: ../partedUtils.py:238 ../textw/partition_text.py:577
msgid "Foreign"
msgstr "その他"

#: ../partedUtils.py:366
#, python-format
msgid ""
"The device %s is LDL formatted instead of CDL formatted.  LDL formatted "
"DASDs are not supported for usage during an install of %s.  If you wish to "
"use this disk for installation, it must be re-initialized causing the loss "
"of ALL DATA on this drive.\n"
"\n"
"Would you like to reformat this DASD using CDL format?"
msgstr ""
"デバイス %s は、ディスクL 形式ではなく、LDL 形式でフォーマットされています。"
"LDL 形式でフォーマットされた DASD の利用は %s のインストールではサポートされ"
"ていません。インストール用にこのディスクを使用するには、再度初期化する必要が"
"ありますが、このドライブ上の「すべてのデータ」を失うことになります。\n"
"\n"
"この DASD を ディスクL 形式で初期化しますか?"

#: ../partedUtils.py:396
#, python-format
msgid ""
"/dev/%s currently has a %s partition layout.  To use this drive for the "
"installation of %s, it must be re-initialized, causing the loss of ALL DATA "
"on this drive.\n"
"\n"
"Would you like to re-initialize this drive?"
msgstr ""
"/dev/%s は現在、%s パーティションレイアウトになっています。%s のインストール"
"にこのディスクを使用するには、再度初期化する必要がありますが、このドライブ上"
"にあるすべてのデータを失うことになります。\n"
"\n"
"このドライブを再度初期化しますか?"

#: ../partedUtils.py:405
msgid "_Ignore drive"
msgstr "ドライブを無視(_I)"

#: ../partedUtils.py:406
msgid "_Re-initialize drive"
msgstr "ドライブの再初期化 (_R)"

#: ../partedUtils.py:994
msgid "Initializing"
msgstr "初期化中"

#: ../partedUtils.py:995
#, python-format
msgid "Please wait while formatting drive %s...\n"
msgstr "ドライブ %s をフォーマット中です、しばらくお待ちください...\n"

#: ../partedUtils.py:1083
#, python-format
msgid ""
"The partition table on device %s was unreadable. To create new partitions it "
"must be initialized, causing the loss of ALL DATA on this drive.\n"
"\n"
"This operation will override any previous installation choices about which "
"drives to ignore.\n"
"\n"
"Would you like to initialize this drive, erasing ALL DATA?"
msgstr ""
"デバイス %s のパーティションテーブルが読み込めませんでした。新規パーティショ"
"ンを作成するには初期化が必要ですが、本ドライブにあるすべてのデータを失うこと"
"になります。\n"
"\n"
"この操作を行うと、以前のインストールで選択した無視するドライブの選択をすべて"
"上書きしてしまいます。\n"
"\n"
"すべてのデータを消去して、このドライブを初期化しますか?"

#: ../partedUtils.py:1097
#, python-format
msgid ""
"The partition table on device %s (%s) was unreadable. To create new "
"partitions it must be initialized, causing the loss of ALL DATA on this "
"drive.\n"
"\n"
"This operation will override any previous installation choices about which "
"drives to ignore.\n"
"\n"
"Would you like to initialize this drive, erasing ALL DATA?"
msgstr ""
"デバイス %s にあるパーティションテーブルを読み込めませんでした (%s)。新規パー"
"ティションを作成するには初期化が必要ですが、本ドライブにあるすべてのデータを"
"失うことになります。\n"
"\n"
"この操作を行うと、以前のインストールで選択した無視するドライブの選択をすべて"
"上書きしてしまいます。\n"
"\n"
"すべてのデータを消去して、このドライブを初期化しますか?"

#: ../partedUtils.py:1251
#, python-format
msgid ""
"The drive /dev/%s has more than 15 partitions on it.  The SCSI subsystem in "
"the Linux kernel does not allow for more than 15 partitons at this time.  "
"You will not be able to make changes to the partitioning of this disk or use "
"any partitions beyond /dev/%s15 in %s"
msgstr ""
"ドライブ /dev/%s には 15 を超えるパーティションがあります。現在の Linux カー"
"ネルのSCSI サブシステムは 15 を超えるパーティションを認めていません。このディ"
"スクのパーティション構成に変更を加えることはできず、/dev/%s15 を超える %sのど"
"のパーティションも使用できません。"

#: ../partedUtils.py:1342
msgid "No Drives Found"
msgstr "ドライブがみつかりませんでした。"

#: ../partedUtils.py:1343
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems. Please check your hardware for the cause of this problem."
msgstr ""
"エラー発生 - 新規ファイルシステム作成のための有効なデバイスが見つかりません。"
"ハードウェアをチェックして、この問題の原因を調査してください。"

#: ../partIntfHelpers.py:42
msgid "Please enter a volume group name."
msgstr "ボリュームグループ名を入力してください。"

#: ../partIntfHelpers.py:46
msgid "Volume Group Names must be less than 128 characters"
msgstr "ボリュームグループ名は 128 文字以下にしてください"

#: ../partIntfHelpers.py:49
#, python-format
msgid "Error - the volume group name %s is not valid."
msgstr "エラー - ボリュームグループ名 %s は有効ではありません。"

#: ../partIntfHelpers.py:54
msgid ""
"Error - the volume group name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""
"エラー - ボリュームグループ名に不適切な文字または空白が含まれています。ボ"
"リュームグループ名として使用できる符号は文字、数字、「.」、「_」になります。"

#: ../partIntfHelpers.py:64
msgid "Please enter a logical volume name."
msgstr "論理ボリューム名を入力してください。"

#: ../partIntfHelpers.py:68
msgid "Logical Volume Names must be less than 128 characters"
msgstr "論理ボリューム名は 128 文字以下にしてください"

#: ../partIntfHelpers.py:72
#, python-format
msgid "Error - the logical volume name %s is not valid."
msgstr "エラー - 論理ボリューム名 %s は有効ではありません。"

#: ../partIntfHelpers.py:78
msgid ""
"Error - the logical volume name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""
"エラー - 論理ボリューム名に不適切な文字または空白が含まれています。論理ボ"
"リューム名に使用できる符号は文字、数字、「.」、「_」になります。"

#: ../partIntfHelpers.py:102
#, python-format
msgid ""
"The mount point %s is invalid.  Mount points must start with '/' and cannot "
"end with '/', and must contain printable characters and no spaces."
msgstr ""
"マウントポイント %s は無効です。マウントポイントは 「/」 で始まらなければなら"
"ず、「/」 で終ることはできません。また、空白がなく表示可能な文字で構成されて"
"いなければなりません。"

#: ../partIntfHelpers.py:109
msgid "Please specify a mount point for this partition."
msgstr "このパーティションに対するマウントポイントを指定してください。"

#: ../partIntfHelpers.py:119
#, python-format
msgid "This partition is part of the RAID device /dev/md%s."
msgstr "このパーティションは RAID デバイス /dev/md%s の一部です。"

#: ../partIntfHelpers.py:122
msgid "This partition is part of a RAID device."
msgstr "このパーティションは RAID デバイスの一部です。"

#: ../partIntfHelpers.py:127
#, python-format
msgid "This partition is part of the LVM volume group '%s'."
msgstr "このパーティションは LVM ボリュームグループ 「%s」 の一部です。"

#: ../partIntfHelpers.py:130
msgid "This partition is part of a LVM volume group."
msgstr "このパーティションは LVM ボリュームグループの一部です。"

#: ../partIntfHelpers.py:145 ../partIntfHelpers.py:153
#: ../partIntfHelpers.py:160 ../partIntfHelpers.py:170
#: ../partIntfHelpers.py:194
msgid "Unable To Delete"
msgstr "削除できません"

#: ../partIntfHelpers.py:146
msgid "You must first select a partition to delete."
msgstr "まず削除するパーティションを選択してください。"

#: ../partIntfHelpers.py:154
msgid "You cannot delete free space."
msgstr "空白領域は削除できません。"

#: ../partIntfHelpers.py:161
msgid "You cannot delete a partition of a LDL formatted DASD."
msgstr "DASD 形式でフォーマットされた LDL パーティションは削除できません。"

#: ../partIntfHelpers.py:171
#, python-format
msgid ""
"You cannot delete this partition, as it is an extended partition which "
"contains %s"
msgstr "これは %s を含む拡張パーティションなので削除できません。"

#: ../partIntfHelpers.py:189 ../iw/raid_dialog_gui.py:607
msgid "This partition is holding the data for the hard drive install."
msgstr ""
"このパーティションはハードドライブインストール用のデータを保有しています。"

#: ../partIntfHelpers.py:195
msgid ""
"You cannot delete this partition:\n"
"\n"
msgstr ""
"このパーティションは削除できません:\n"
"\n"

#: ../partIntfHelpers.py:240 ../partIntfHelpers.py:534
#: ../iw/lvm_dialog_gui.py:800
msgid "Confirm Delete"
msgstr "削除の確認"

#: ../partIntfHelpers.py:241
#, python-format
msgid "You are about to delete all partitions on the device '/dev/%s'."
msgstr ""
"デバイス 「/dev/%s」上のすべてのパーティションを削除しようとしています。"

#: ../partIntfHelpers.py:244 ../partIntfHelpers.py:535
#: ../iw/lvm_dialog_gui.py:803 ../iw/lvm_dialog_gui.py:1152
#: ../iw/osbootwidget.py:103 ../iw/partition_gui.py:1380
msgid "_Delete"
msgstr "削除(_D)"

#: ../partIntfHelpers.py:301
msgid "Notice"
msgstr "通知"

#: ../partIntfHelpers.py:302
#, python-format
msgid ""
"The following partitions were not deleted because they are in use:\n"
"\n"
"%s"
msgstr ""
"次のパーティションは使用中のため削除されませんでした。\n"
"\n"
"%s"

#: ../partIntfHelpers.py:318 ../partIntfHelpers.py:331
#: ../partIntfHelpers.py:357 ../partIntfHelpers.py:368
msgid "Unable To Edit"
msgstr "編集できません"

#: ../partIntfHelpers.py:319
msgid "You must select a partition to edit"
msgstr "編集するパーティションを選択してください"

#: ../partIntfHelpers.py:331 ../partIntfHelpers.py:369
msgid ""
"You cannot edit this partition:\n"
"\n"
msgstr ""
"このパーティションは編集できません:\n"
"\n"

#: ../partIntfHelpers.py:358
#, python-format
msgid ""
"You cannot edit this partition, as it is an extended partition which "
"contains %s"
msgstr "これは %s を含む拡張パーティションなので編集できません。"

#: ../partIntfHelpers.py:390
msgid "Format as Swap?"
msgstr "swap としてフォーマットしますか?"

#: ../partIntfHelpers.py:391
#, python-format
msgid ""
"/dev/%s has a partition type of 0x82 (Linux swap) but does not appear to be "
"formatted as a Linux swap partition.\n"
"\n"
"Would you like to format this partition as a swap partition?"
msgstr ""
"/dev/%s にはタイプ 0x82 (Linux swap) のパーティションがありますが、Linux "
"swap パーティションとしてフォーマットされるようには表示されていません。\n"
"\n"
"このパーティションを swap パーティションとしてフォーマットしますか?"

#: ../partIntfHelpers.py:411
#, python-format
msgid "You need to select at least one hard drive to install %s."
msgstr ""
"%s をインストールするには、少なくともハードドライブを 1 つ選択しなければなり"
"ません。"

#: ../partIntfHelpers.py:416
msgid ""
"You have chosen to use a pre-existing partition for this installation "
"without formatting it. We recommend that you format this partition to make "
"sure files from a previous operating system installation do not cause "
"problems with this installation of Linux. However, if this partition "
"contains files that you need to keep, such as home directories, then "
"continue without formatting this partition."
msgstr ""
"既存のパーティションをフォーマットせずに、インストールに使用するよう選択して"
"います。前のオペレーティングシステムのファイル群が原因でこの Linux のインス"
"トールに支障をきたさないよう、パーティションのフォーマットを行うことを推奨し"
"ます。尚、ユーザーのホームディレクトリなど保持しておく必要があるファイルをあ"
"る場合は、パーティションのフォーマットを行わずに続行します。"

#: ../partIntfHelpers.py:424
msgid "Format?"
msgstr "フォーマットしますか?"

#: ../partIntfHelpers.py:424 ../iw/partition_gui.py:1035
msgid "_Modify Partition"
msgstr "パーティションの変更(_M)"

#: ../partIntfHelpers.py:424
msgid "Do _Not Format"
msgstr "フォーマットをしない(_N)"

#: ../partIntfHelpers.py:432
msgid "Error with Partitioning"
msgstr "パーティション設定エラー"

#: ../partIntfHelpers.py:433
#, python-format
msgid ""
"The following critical errors exist with your requested partitioning scheme. "
"These errors must be corrected prior to continuing with your install of %s.\n"
"\n"
"%s"
msgstr ""
"要求されたパーティション設定に次のような致命的なエラーが存在しています。 %s "
"のインストールを続行する前に、これらのエラーを修正する必要があります。\n"
"\n"
"%s"

#: ../partIntfHelpers.py:447
msgid "Partitioning Warning"
msgstr "パーティション設定の警告"

#: ../partIntfHelpers.py:448
#, python-format
msgid ""
"The following warnings exist with your requested partition scheme.\n"
"\n"
"%s\n"
"\n"
"Would you like to continue with your requested partitioning scheme?"
msgstr ""
"要求されたパーティション設定について次のような警告があります。\n"
"\n"
"%s\n"
"\n"
"このパーティション設定のまま続行しますか?"

#: ../partIntfHelpers.py:462 ../iw/partition_gui.py:669
msgid ""
"The following pre-existing partitions have been selected to be formatted, "
"destroying all data."
msgstr ""
"次の既存のパーティションをフォーマットするよう選択されています。これによりす"
"べてのデータが消去されます。"

#: ../partIntfHelpers.py:465
msgid ""
"Select 'Yes' to continue and format these partitions, or 'No' to go back and "
"change these settings."
msgstr ""
"処理を続行してこれらのパーティションをフォーマットするには、「はい」を選択し"
"ます。前に戻り設定を変更するには「いいえ」を選択します。"

#: ../partIntfHelpers.py:471
msgid "Format Warning"
msgstr "フォーマット警告"

#: ../partIntfHelpers.py:519
#, python-format
msgid ""
"You are about to delete the volume group \"%s\".\n"
"\n"
"ALL logical volumes in this volume group will be lost!"
msgstr ""
"ボリュームグループ \"%s\" を削除しようとしています。\n"
" \n"
"このボリュームグループ内のすべての論理ボリュームが消去されます!"

#: ../partIntfHelpers.py:523
#, python-format
msgid "You are about to delete the logical volume \"%s\"."
msgstr "論理ボリューム \"%s\" を削除しようとしています。"

#: ../partIntfHelpers.py:526
msgid "You are about to delete a RAID device."
msgstr "RAID デバイスを削除しようとしています。"

#: ../partIntfHelpers.py:529
#, python-format
msgid "You are about to delete the /dev/%s partition."
msgstr "/dev/%s パーティションを削除しようとしています。"

#: ../partIntfHelpers.py:532
msgid "The partition you selected will be deleted."
msgstr "選択したパーティションは削除されます。"

#: ../partIntfHelpers.py:542
msgid "Confirm Reset"
msgstr "リセットの確認"

#: ../partIntfHelpers.py:543
msgid ""
"Are you sure you want to reset the partition table to its original state?"
msgstr "本当に、パーティションテーブル設定をリセットして元の状態に戻しますか?"

#: ../partitions.py:84
msgid "Installation cannot continue."
msgstr "インストールを続行できません。"

#: ../partitions.py:85
msgid ""
"The partitioning options you have chosen have already been activated. You "
"can no longer return to the disk editing screen. Would you like to continue "
"with the installation process?"
msgstr ""
"指定されたパーティション設定オプションは既に選択されています。ここからディス"
"ク編集画面に戻ることはできません。インストールを続けますか?"

#: ../partitions.py:113
msgid "Writing partitioning to disk"
msgstr "パーティション設定をディスクに書き込み中"

#: ../partitions.py:114
msgid ""
"The partitioning options you have selected will now be written to disk.  Any "
"data on deleted or reformatted partitions will be lost"
msgstr ""
"選択したディスク分割のオプションを書き込みます。削除及び再フォーマットするよ"
"うに選択したパーティションにあるデータはすべて失われます。"

#: ../partitions.py:119
msgid "Go _back"
msgstr "戻る(_B)"

#: ../partitions.py:120
msgid "_Write changes to disk"
msgstr "変更をディスクに書き込む (_W)"

#: ../partitions.py:200 ../textw/constants_text.py:50
#: ../loader2/cdinstall.c:351 ../loader2/driverdisk.c:236
#: ../loader2/driverdisk.c:267 ../loader2/driverdisk.c:351
#: ../loader2/driverselect.c:75 ../loader2/driverselect.c:173
#: ../loader2/driverselect.c:199 ../loader2/hdinstall.c:334
#: ../loader2/hdinstall.c:388 ../loader2/kbd.c:138 ../loader2/loader.c:415
#: ../loader2/loader.c:431 ../loader2/loader.c:1159 ../loader2/loader.c:1181
#: ../loader2/net.c:277 ../loader2/net.c:320 ../loader2/net.c:750
#: ../loader2/net.c:1094 ../loader2/net.c:1863 ../loader2/nfsinstall.c:68
#: ../loader2/urls.c:280 ../loader2/urls.c:405
msgid "Back"
msgstr "戻る"

#: ../partitions.py:200 ../rescue.py:305 ../iw/partition_gui.py:572
#: ../textw/network_text.py:60 ../loader2/cdinstall.c:117
#: ../loader2/cdinstall.c:125 ../loader2/driverdisk.c:440
msgid "Continue"
msgstr "続行"

#: ../partitions.py:204
msgid "Confirm"
msgstr "確認"

#: ../partitions.py:205
#, python-format
msgid ""
"Are you sure you want to skip entering a passphrase for device %s?\n"
"\n"
"If you skip this step the device's contents will not be available during "
"installation."
msgstr ""
"本当にデバイス %s のパスフレーズの入力を省略しますか?\n"
"\n"
"このステップをスキップすると、インストール中のデバイス内のコンテンツへのアク"
"セスはできません。"

#: ../partitions.py:1026
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""
"%s のインストールを継続するために必要な root パーティション (/) が定義されて"
"いません。"

#: ../partitions.py:1031
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""
"root パーティションが 250 MB 以下になっています。通常、 %s のインストールには"
"小さすぎます。"

#: ../partitions.py:1053
msgid ""
"You must create an EFI System Partition of type FAT and a size of 50 "
"megabytes."
msgstr ""
"EFI システムパーティションを作成する必要があります。これは、50 メガバイトの"
"FAT システムである必要があります。"

#: ../partitions.py:1067
msgid ""
"Your boot partition isn't on one of the first four partitions and thus won't "
"be bootable."
msgstr ""
"ブートパーティションが最初の 4 つのパーティションのいずれにもなっていないため"
"ブートできません。"

#: ../partitions.py:1088
msgid "You must create an Apple Bootstrap partition."
msgstr "Apple Bootstrap パーティションを作成しなくてはなりません。"

#: ../partitions.py:1110
msgid "You must create a PPC PReP Boot partition."
msgstr "PPC PReP Boot パーティションを作成しなくてはなりません。"

#: ../partitions.py:1118 ../partitions.py:1129
#, python-format
msgid ""
"Your %s partition is less than %s megabytes which is lower than recommended "
"for a normal %s install."
msgstr ""
"%s パーティションの容量が %s MB未満であり、通常の %s のインストールでの推奨容"
"量には不足しています。"

#: ../partitions.py:1163
msgid ""
"Installing on a USB device.  This may or may not produce a working system."
msgstr ""
"USB デバイス上にインストールしています。稼動するシステムが作成されることもあ"
"りますが、されないこともあります。"

#: ../partitions.py:1166
msgid ""
"Installing on a FireWire device.  This may or may not produce a working "
"system."
msgstr ""
"FireWire デバイス上にインストールしています。稼動するシステムが作成されること"
"もありますが、されないこともあります。"

#: ../partitions.py:1176
msgid "Bootable partitions can only be on RAID1 devices."
msgstr "RAID1 デバイス上にしか起動可能なパーティションは存在できません。"

#: ../partitions.py:1181
msgid "Bootable partitions cannot be on a logical volume."
msgstr "起動可能なパーティションは論理ボリューム上には存在できません。"

#: ../partitions.py:1187
msgid "Bootable partitions cannot be on a RAID device."
msgstr "起動可能なパーティションを RAID デバイス上に作成することはできません。"

#: ../partitions.py:1194
#, python-format
msgid "Bootable partitions cannot be on an %s filesystem."
msgstr ""
"起動可能なパーティションを、ファイルシステム %s に作成することはできません。"

#: ../partitions.py:1198
msgid "Bootable partitions cannot be on an encrypted block device"
msgstr ""
"起動可能なパーティションを暗号化されたブロックデバイス上に作成することはでき"
"ません。"

#: ../partitions.py:1204
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""
"swap パーティションが未指定です。すべての場合において必ず必要というわけではあ"
"りませんが、ほとんどのインストールにおいて明らかに性能を向上させます。"

#: ../partitions.py:1211
#, python-format
msgid ""
"You have specified more than 32 swap devices.  The kernel for %s only "
"supports 32 swap devices."
msgstr ""
"32 個を超える swap デバイスが指定されていますが、 %s のカーネルは 32個の "
"swap デバイスしかサポートしません。"

#: ../partitions.py:1222
#, python-format
msgid ""
"You have allocated less swap space (%dM) than available RAM (%dM) on your "
"system.  This could negatively impact performance."
msgstr ""
"確保した swap 領域 (%d MB)が、システムで実際に利用可能な RAM 容量 (%d MB)より"
"も小さくなっています。これによって性能が劣化する可能性があります。"

#: ../partitions.py:1517
msgid "the partition in use by the installer."
msgstr "このパーティションはインストーラーが使用中です。"

#: ../partitions.py:1520
msgid "a partition which is a member of a RAID array."
msgstr "RAID アレイのメンバーであるパーティションです。"

#: ../partitions.py:1523
msgid "a partition which is a member of a LVM Volume Group."
msgstr "LVM ボリュームグループのメンバーであるパーティションです。"

#: ../partRequests.py:273
#, python-format
msgid "The mount point %s must be formatted during live CD installs."
msgstr ""
"マウントポイント %s をライブCDからのインストール中にフォーマットしなければな"
"りません。"

#: ../partRequests.py:278
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""
"マウントポイントが不適切です。 %s ディレクトリは / ファイルシステム上になけれ"
"ばなりません。"

#: ../partRequests.py:281
#, python-format
msgid ""
"The mount point %s cannot be used.  It must be a symbolic link for proper "
"system operation.  Please select a different mount point."
msgstr ""
"マウントポイント %s は使用できません。正常なシステム動作が行われるためのシン"
"ボリックリンクのはずです。別のマウントポイントを選択してください。"

#: ../partRequests.py:290
msgid "This mount point must be on a linux file system."
msgstr ""
"このマウントポイントは linux のファイルシステム上になければなりません。"

#: ../partRequests.py:311
#, python-format
msgid ""
"The mount point \"%s\" is already in use, please choose a different mount "
"point."
msgstr ""
"マウントポイント \"%s\" はすでに使用されています。別のマウントポイントを選択"
"してください。"

#: ../partRequests.py:325
#, python-format
msgid ""
"The size of the %s partition (%10.2f MB) exceeds the maximum size of %10.2f "
"MB."
msgstr ""
"%s パーティションのサイズ (%10.2f MB) は最大サイズ %10.2f MB を超えています。"

#: ../partRequests.py:533
#, python-format
msgid ""
"The size of the requested partition (size = %s MB) exceeds the maximum size "
"of %s MB."
msgstr ""
"要求されたパーティションのサイズ（サイズ = %s MB) は最大サイズの %s MBを超え"
"ています。"

#: ../partRequests.py:538
#, python-format
msgid "The size of the requested partition is negative! (size = %s MB)"
msgstr "要求されたパーティションのサイズ(サイズ = %s MB)が負数です!"

#: ../partRequests.py:542
msgid "Partitions can't start below the first cylinder."
msgstr "第一シリンダーよりも前からパーティションを始めることはできません。"

#: ../partRequests.py:545
msgid "Partitions can't end on a negative cylinder."
msgstr "パーティションが負のシリンダーで終ることはできません。"

#: ../partRequests.py:746
msgid "No members in RAID request, or not RAID level specified."
msgstr ""
"RAID 要求の中にメンバーが含まれていない、または RAID レベルが未指定です。"

#: ../partRequests.py:751
#, python-format
msgid "A RAID device of type %s requires at least %s members."
msgstr "タイプ %s の RAID デバイスは最低でも %s メンバーを必要とします。"

#: ../partRequests.py:760
#, python-format
msgid ""
"This RAID device can have a maximum of %s spares. To have more spares you "
"will need to add members to the RAID device."
msgstr ""
"この RAID デバイスは最大 %s スペアを持つことができます。スペアを増やすには、 "
"RAID デバイスにメンバーを追加する必要があります。"

#: ../partRequests.py:1026
msgid ""
"Logical volume size must be larger than the volume group's physical extent "
"size."
msgstr ""
"論理ボリュームのサイズは、ボリュームグループの物理エクステントのサイズより大"
"きくなければなりません。"

#: ../rescue.py:157
msgid "Starting Interface"
msgstr "インターフェースを起動中"

#: ../rescue.py:158
#, python-format
msgid "Attempting to start %s"
msgstr "%s の起動を試行中"

#: ../rescue.py:200
msgid "When finished please exit from the shell and your system will reboot."
msgstr "完了したらシェルを終了してください。システムが再起動します。"

#: ../rescue.py:226
msgid "Setup Networking"
msgstr "ネットワークの設定"

#: ../rescue.py:227
msgid "Do you want to start the network interfaces on this system?"
msgstr "このシステムでネットワークインターフェースを開始しますか?"

#: ../rescue.py:267 ../text.py:695
msgid "Cancelled"
msgstr "取り消されました"

#: ../rescue.py:268 ../text.py:696
msgid "I can't go to the previous step from here. You will have to try again."
msgstr "直前のステップに戻ることはできません。もう一度やり直してください。"

#: ../rescue.py:294 ../rescue.py:361 ../rescue.py:369 ../rescue.py:447
msgid "Rescue"
msgstr "レスキュー"

#: ../rescue.py:295
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.\n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""
"レスキュー環境はこれから Linux インストールを探し、ディレクトリ %s の下にマウ"
"ントしようと試みます。これにより、ご使用のシステムに必要な変更を行うことがで"
"きます。このプロセスを続ける場合、「続行」を選択してください。また 「読み取り"
"専用」を選択すると、「読み取りと書き込み」ではなく「読み取り専用」でのファイ"
"ルシステムのマウントを選択することもできます。\n"
"\n"
"何らかの原因でこのプロセスが失敗する場合は「スキップする」を選択し、このプロ"
"セスをとばして直接コマンドシェルに行くことができます。\n"
"\n"

#: ../rescue.py:305 ../rescue.py:310
msgid "Read-Only"
msgstr "読み取り専用"

#: ../rescue.py:336
msgid "System to Rescue"
msgstr "レスキュー対象システム"

#: ../rescue.py:337
msgid "What partition holds the root partition of your installation?"
msgstr ""
"インストールの root パーティションを格納しているパーティションを指定してくだ"
"さい。"

#: ../rescue.py:339 ../rescue.py:343 ../text.py:669 ../text.py:671
msgid "Exit"
msgstr "終了"

#: ../rescue.py:362
msgid ""
"Your system had dirty file systems which you chose not to mount.  Press "
"return to get a shell from which you can fsck and mount your partitions.  "
"The system will reboot automatically when you exit from the shell."
msgstr ""
"システムに損傷のあるファイルシステムがありマウントしないことを選択していま"
"す。[Enter] を押すとシェルが起動され、そこで fsck やパーティションのマウント"
"ができます。シェルを終了するとシステムは自動的に再起動します。"

#: ../rescue.py:370
#, python-format
msgid ""
"Your system has been mounted under %s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %s\n"
"\n"
"The system will reboot automatically when you exit from the shell."
msgstr ""
"システムを %s 下にマウントしています。\n"
"\n"
"シェルを起動するには [Enter] を押してください。そのシステムを root 環境として"
"利用するには次のコマンドを実行します。\n"
"\n"
"\tchroot %s\n"
"\n"
"シェルを終了するとシステムは自動的に再起動します。"

#: ../rescue.py:448
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell. The system will reboot automatically when you "
"exit from the shell."
msgstr ""
"システムのいくつか、あるいはすべてをマウントしようと試みている間にエラーが発"
"生しました。いくつかのパーティションは %s の下にマウントされています。\n"
"\n"
"シェルを起動するには [Enter] を押してください。シェルを終了するとシステムは自"
"動的に再起動します。"

#: ../rescue.py:454
msgid "Rescue Mode"
msgstr "レスキューモード"

#: ../rescue.py:455
msgid ""
"You don't have any Linux partitions. Press return to get a shell. The system "
"will reboot automatically when you exit from the shell."
msgstr ""
"Linux パーティションがありません。シェルを起動するには [Enter] を押してくださ"
"い。シェルを終了するとシステムは自動的に再起動します。"

#: ../rescue.py:468
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr "システムは %s ディレクトリ配下にマウントされます。"

#: ../text.py:185 ../text.py:251 ../text.py:264 tmp/exnSave.glade.h:1
msgid "Save"
msgstr "保存"

#: ../text.py:188
msgid "Save to Disk"
msgstr "ディスクに保存"

#: ../text.py:189
msgid "Save to Remote"
msgstr "リモートに保存 (_R)"

#: ../text.py:203
msgid "Host"
msgstr "ホスト"

#: ../text.py:205
msgid "Remote path"
msgstr "リモートパス"

#: ../text.py:207
msgid "User name"
msgstr "ユーザ名"

#: ../text.py:209
msgid "Password"
msgstr "パスワード"

#: ../text.py:283 tmp/lukspassphrase.glade.h:5
msgid "Passphrase"
msgstr "パスフレーズ"

#: ../text.py:291 tmp/lukspassphrase.glade.h:6
msgid "This is a global passphrase"
msgstr "これはグローバルパスフレーズです。"

#: ../text.py:347
msgid "Help not available"
msgstr "ヘルプ情報はありません"

#: ../text.py:348
msgid "No help is available for this step of the install."
msgstr "このインストールステップについてのヘルプはありません"

#: ../text.py:446
msgid "Repository editing is not available in text mode."
msgstr "レポジトリの編集は、テキストモードではできません。"

#: ../text.py:471
#, python-format
msgid "Please enter your %(instkey)s"
msgstr "%(instkey)s を入力してください。"

#: ../text.py:488 tmp/instkey.glade.h:6
#, no-c-format, python-format
msgid "Skip entering %(instkey)s"
msgstr "%(instkey)s の入力をスキップ"

#: ../text.py:564 ../loader2/lang.c:63
#, c-format, python-format
msgid "Welcome to %s for %s"
msgstr "%s (%s 用)へようこそ"

#: ../text.py:566
#, python-format
msgid "Welcome to %s"
msgstr "%s へようこそ"

#: ../text.py:569
msgid ""
" <F1> for help | <Tab> between elements | <Space> selects | <F12> next screen"
msgstr " <F1> ヘルプ | <Tab> 項目間の移動 | <Space> 選択 | <F12> 次の画面"

#: ../text.py:571
msgid ""
"  <Tab>/<Alt-Tab> between elements   |  <Space> selects   |  <F12> next "
"screen"
msgstr "  <Tab>/<Alt-Tab> 項目間の移動   |  <Space> 選択   |  <F12> 次の画面"

#: ../upgrade.py:105 ../upgrade.py:126
msgid "Proceed with upgrade?"
msgstr "アップグレードしますか?"

#: ../upgrade.py:106
#, python-format
msgid ""
"You have choosen the upgrade for %s architecture, but the installed system "
"is for %s architecture. \n"
"\n"
msgstr ""
"選択したアップデートは %s アーキテクチャのものですが、インストールされたシス"
"テムのアーキテクチャは %s です。\n"
"\n"

#: ../upgrade.py:110
#, python-format
msgid "Would you like to upgrade  the installed system to the %s architecture?"
msgstr ""
"インストールされたシステムを、%s アーキテクチャにアップグレードしますか?"

#: ../upgrade.py:127
msgid ""
"The file systems of the Linux installation you have chosen to upgrade have "
"already been mounted. You cannot go back past this point. \n"
"\n"
msgstr ""
"アップグレードの選択をした Linux インストールのファイルシステムは既にマウント"
"されています。ここから元に戻ることはできません。\n"
"\n"

#: ../upgrade.py:131
msgid "Would you like to continue with the upgrade?"
msgstr "アップグレードを続行しますか?"

#: ../upgrade.py:229 ../upgrade.py:237
msgid "Dirty File Systems"
msgstr "損傷ファイルシステム"

#: ../upgrade.py:230
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Please boot your Linux installation, let the file systems be "
"checked and shut down cleanly to upgrade.\n"
"%s"
msgstr ""
"ご使用の Linux システム用の次のファイルシステムは正常にアンマウントされません"
"でした。アップグレードするには、 Linux インストールを起動し、ファイルシステム"
"のチェックをしてから正常なシャットダウンを行ってください。\n"
"%s"

#: ../upgrade.py:238
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""
"ご使用の Linux システム用の次のファイルシステムは正常にアンマウントされません"
"でした。とにかくそれらをマウントしますか?\n"
"%s"

#: ../upgrade.py:379 ../upgrade.py:385
msgid "Mount failed"
msgstr "マウントに失敗しました"

#: ../upgrade.py:380
msgid ""
"One or more of the file systems listed in the /etc/fstab on your Linux "
"system cannot be mounted. Please fix this problem and try to upgrade again."
msgstr ""
"ご使用の Linux システムの /etc/fstab に記載されたひとつまたは複数のファイルシ"
"ステムがマウントできません。この問題を解決してからアップグレードをやり直して"
"ください。"

#: ../upgrade.py:386
msgid ""
"One or more of the file systems listed in the /etc/fstab of your Linux "
"system are inconsistent and cannot be mounted.  Please fix this problem and "
"try to upgrade again."
msgstr ""
"ご使用の Linux システムの /etc/fstab に記載されたひとつまたは複数のファイルシ"
"ステムが矛盾しているためマウントできません。この問題を解決してからアップグ"
"レードをやり直してください。"

#: ../upgrade.py:403
msgid ""
"The following files are absolute symbolic links, which we do not support "
"during an upgrade. Please change them to relative symbolic links and restart "
"the upgrade.\n"
"\n"
msgstr ""
"次のファイルはアップグレード中にはサポートしていない絶対シンボリックリンクで"
"す。相対シンボリックリンクに変更してからアップグレードを再開してください。\n"
"\n"

#: ../upgrade.py:409
msgid "Absolute Symlinks"
msgstr "絶対 Symlink"

#: ../upgrade.py:420
msgid ""
"The following are directories which should instead be symbolic links, which "
"will cause problems with the upgrade.  Please return them to their original "
"state as symbolic links and restart the upgrade.\n"
"\n"
msgstr ""
"これらは、シンボリックリンクであるべきディレクトリであり、アップグレード時に"
"起きる問題の原因となります。シンボリックリンクとして本来の状態に戻してから"
"アップグレードを再開してください。\n"
"\n"

#: ../upgrade.py:426
msgid "Invalid Directories"
msgstr "無効なディレクトリ"

#: ../upgrade.py:433
#, python-format
msgid "%s not found"
msgstr "%s が見つかりませんでした"

#: ../vnc.py:145
#, python-format
msgid "%s %s installation on host %s"
msgstr "%s %s インストール、ホスト %s"

#: ../vnc.py:147
#, python-format
msgid "%s %s installation"
msgstr "%s %s インストール"

#: ../vnc.py:174
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "ホスト %s 上の vnc クライアントに接続を試行中..."

#: ../vnc.py:188
msgid "Connected!"
msgstr "接続しました!"

#: ../vnc.py:191
msgid "Will try to connect again in 15 seconds..."
msgstr "15 秒後に再接続を試みます..."

#: ../vnc.py:197
#, python-format
msgid "Giving up attempting to connect after %d tries!\n"
msgstr "接続を %d 回試行したら、接続試行を中止します!\n"

#: ../vnc.py:206
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""
"インストールを開始するには、ご使用の vnc クライアントを %s に手動で接続してく"
"ださい。"

#: ../vnc.py:208
msgid "Please manually connect your vnc client to begin the install."
msgstr ""
"インストールを開始するには、ご使用の vnc クライアントを手動で接続してくださ"
"い。"

#: ../vnc.py:211
msgid "Starting VNC..."
msgstr "VNC を起動中..."

#: ../vnc.py:236
msgid "The VNC server is now running."
msgstr "VNC サーバーが稼動しています。"

#: ../vnc.py:249
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""
"\n"
"\n"
"Listen モードの VNC Viewer に接続するよう選択しました。\n"
"これは、パスワードを必要としません。もしパスワードを設定すると、\n"
"VNC Viewew への接続失敗の原因になります。\n"
"\n"

#: ../vnc.py:254
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the self.password=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""
"\n"
"\n"
"警告!!! VNC サーバーがパスワードなしで実行されています!\n"
"サーバーの安全性を確保するには、self.password=<password>\n"
"ブートオプションを使うことができます。\n"
"\n"

#: ../vnc.py:258
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""
"\n"
"\n"
"パスワードを使って VNC を実行するように選択しました。\n"
"\n"

#: ../vnc.py:260
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""
"\n"
"\n"
"不明なエラーが発生しました。作業を中断します。\n"

#: ../vnc.py:285 ../vnc.py:374
msgid "VNC Configuration"
msgstr "VNC の設定"

#: ../vnc.py:288 ../vnc.py:378
msgid "No password"
msgstr "パスワードなし"

#: ../vnc.py:290 ../vnc.py:381
msgid ""
"A password will prevent unauthorized listeners connecting and monitoring "
"your installation progress.  Please enter a password to be used for the "
"installation"
msgstr ""
"パスワードによって許可のないリスナーが接続してインストールの進捗を傍受するの"
"を防ぎます。インストールに使用するパスワードを入力してください。"

#: ../vnc.py:298 ../vnc.py:389 ../textw/userauth_text.py:44
msgid "Password:"
msgstr "パスワード:"

#: ../vnc.py:299 ../vnc.py:390 ../textw/userauth_text.py:46
msgid "Password (confirm):"
msgstr "パスワード (確認):"

#: ../vnc.py:317 ../vnc.py:412 ../textw/userauth_text.py:67
msgid "Password Mismatch"
msgstr "パスワードが一致しません"

#: ../vnc.py:318 ../vnc.py:413 ../textw/userauth_text.py:68
msgid "The passwords you entered were different. Please try again."
msgstr "入力したパスワードが異なっています。再度やり直してください。"

#: ../vnc.py:323 ../vnc.py:418 ../textw/userauth_text.py:63
msgid "Password Length"
msgstr "パスワードの長さ"

#: ../vnc.py:324 ../vnc.py:419
msgid "The password must be at least six characters long."
msgstr "パスワードは最低 6 文字以上でなければなりません。"

#: ../vnc.py:355
msgid "Unable to Start X"
msgstr "X を起動できません"

#: ../vnc.py:356
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"install or continue with a text mode install?"
msgstr ""
"ご使用のマシンで X を起動できませんでした。VNC を起動して別のコンピューターか"
"らこのコンピューターに接続しグラフィカルインストールを行ないますか、あるいは"
"テキストモードのインストールで続行しますか?"

#: ../vnc.py:363 ../vnc.py:366
msgid "Use text mode"
msgstr "テキストモードを使用する"

#: ../vnc.py:364
msgid "Start VNC"
msgstr "VNC を起動する"

#: ../yuminstall.py:72
#, python-format
msgid "%s MB"
msgstr "%s MB"

#: ../yuminstall.py:75
#, python-format
msgid "%s KB"
msgstr "%s KB"

#: ../yuminstall.py:78
#, python-format
msgid "%s Byte"
msgstr "%s バイト"

#: ../yuminstall.py:80
#, python-format
msgid "%s Bytes"
msgstr "%s バイト"

#: ../yuminstall.py:128
msgid "Processing"
msgstr "処理中"

#: ../yuminstall.py:129
msgid "Preparing transaction from installation source..."
msgstr "インストールソースからのトランザクションを準備中..."

#: ../yuminstall.py:160
#, python-format
msgid "<b>Installing %s</b> (%s)\n"
msgstr "<b>%s のインストール</b> (%s)\n"

#: ../yuminstall.py:205
#, python-format
msgid "%s of %s packages completed"
msgstr "%s / %s パッケージが完了しました。"

#: ../yuminstall.py:217
msgid "Finishing upgrade"
msgstr "アップグレードの終了"

#: ../yuminstall.py:218
msgid "Finishing upgrade process.  This may take a little while..."
msgstr "アップグレードプロセスを終了しています。数分かかることがあります..."

#: ../yuminstall.py:355
msgid "Copying File"
msgstr "ファイルをコピー中"

#: ../yuminstall.py:356
msgid "Transferring install image to hard drive..."
msgstr "インストールイメージをハードドライブに転送中..."

#: ../yuminstall.py:366
msgid ""
"An error occurred transferring the install image to your hard drive.  This "
"is probably due to bad media."
msgstr ""
"インストールイメージをハードドライブへ転送している際にエラーが発生しました。"
"ディスクが破損している可能性があります。"

#: ../yuminstall.py:370
msgid ""
"An error occurred transferring the install image to your hard drive. You are "
"probably out of disk space."
msgstr ""
"インストールイメージをハードドライブへ転送中にエラーが発生しました。おそらく"
"ディスク容量が不足しています。"

#: ../yuminstall.py:417
msgid "Change Disc"
msgstr "ディスクの変更"

#: ../yuminstall.py:418
#, python-format
msgid "Please insert %s disc %d to continue."
msgstr "%s ディスク %d を挿入して続行します。"

#: ../yuminstall.py:432
msgid "Wrong Disc"
msgstr "不適切なディスクです"

#: ../yuminstall.py:433
#, python-format
msgid "That's not the correct %s disc."
msgstr "正しい %s ディスクではありません。"

#: ../yuminstall.py:439
msgid "Unable to access the disc."
msgstr "ディスクにアクセスできません。"

#: ../yuminstall.py:604 ../yuminstall.py:606
msgid "Re_boot"
msgstr "再起動(_B)"

#: ../yuminstall.py:604
msgid "_Eject"
msgstr "ディスクを取り出す (_E)"

#: ../yuminstall.py:610
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file, a corrupt "
"package or corrupt media.  Please verify your installation source.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"likely require reinstallation.\n"
"\n"
msgstr ""
"ファイル %s を開けません。ファイルが見つからないか、パッケージやメディアが破"
"損している可能性があります。インストール元を検証してください。\n"
"\n"
"終了するとシステムは矛盾した状態となり、再インストールを必要とする可能性が高"
"くなります。\n"
"\n"

#: ../yuminstall.py:650
msgid "Retrying"
msgstr "再試行中"

#: ../yuminstall.py:650
msgid "Retrying package download..."
msgstr "パッケージの再ダウンロード..."

#: ../yuminstall.py:715
#, python-format
msgid ""
"There was an error running your transaction for the following reason: %s\n"
msgstr "トランザクションの実行中に次のような理由でエラーが発生しました。%s\n"

#: ../yuminstall.py:748 ../yuminstall.py:749
msgid "file conflicts"
msgstr "ファイル競合"

#: ../yuminstall.py:750
msgid "older package(s)"
msgstr "旧パッケージ"

#: ../yuminstall.py:751
msgid "insufficient disk space"
msgstr "ディスク容量の不足"

#: ../yuminstall.py:752
msgid "insufficient disk inodes"
msgstr "ディスク inode の不足"

#: ../yuminstall.py:753
msgid "package conflicts"
msgstr "パッケージ競合"

#: ../yuminstall.py:754
msgid "package already installed"
msgstr "パッケージは既にインストールされています"

#: ../yuminstall.py:755
msgid "required package"
msgstr "必要なパッケージ"

#: ../yuminstall.py:756
msgid "package for incorrect arch"
msgstr "不適切なアーキテクチャ用のパッケージ"

#: ../yuminstall.py:757
msgid "package for incorrect os"
msgstr "不適切な OS 用のパッケージ"

#: ../yuminstall.py:771
msgid "You need more space on the following file systems:\n"
msgstr "以下のファイルシステムは容量を増やす必要があります。\n"

#: ../yuminstall.py:784
#, python-format
msgid ""
"There were file conflicts when checking the packages to be installed:\n"
"%s\n"
msgstr ""
"インストールするパッケージに、競合するファイルがみつかりました:\n"
"%s\n"

#: ../yuminstall.py:787
#, python-format
msgid ""
"There was an error running your transaction for the following reason(s): %"
"s.\n"
msgstr "トランザクションの実行中に次のような理由でエラーが発生しました: %s\n"

#: ../yuminstall.py:791
msgid "Error Running Transaction"
msgstr "トランザクション実行中にエラーが発生しました。"

#: ../yuminstall.py:796
msgid "Error running transaction"
msgstr "トランザクション実行中にエラーが発生"

#: ../yuminstall.py:956
msgid "Retrieving installation information..."
msgstr "インストール情報を読み出し中..."

#: ../yuminstall.py:958
#, python-format
msgid "Retrieving installation information for %s..."
msgstr "%s に関するインストール情報を読み出し中..."

#: ../yuminstall.py:970 ../textw/bootloader_text.py:277
#: ../textw/constants_text.py:62 ../textw/partition_text.py:1428
msgid "Edit"
msgstr "編集"

#: ../yuminstall.py:982 ../yuminstall.py:1257
#, python-format
msgid ""
"Unable to read package metadata. This may be due to a missing repodata "
"directory.  Please ensure that your install tree has been correctly "
"generated.\n"
"\n"
"%s"
msgstr ""
"パッケージのメタデータを読めません。repodata ディレクトリが見つからないのが原"
"因かもしれません。インストールツリーが正しく構築されているか確認してくださ"
"い。\n"
"\n"
"%s"

#: ../yuminstall.py:1023
msgid ""
"Unable to read group information from repositories.  This is a problem with "
"the generation of your install tree."
msgstr ""
"レポジトリからグループ情報を読み取ることはできません。これは、インストールツ"
"リーの生成の問題です。"

#: ../yuminstall.py:1056 ../iw/GroupSelector.py:545
msgid "Uncategorized"
msgstr "カテゴリなし"

#: ../yuminstall.py:1282
#, python-format
msgid ""
"Your selected packages require %d MB of free space for installation, but you "
"do not have enough available.  You can change your selections or exit the "
"installer."
msgstr ""
"選択したパッケージはインストールに %d MB の空き領域を必要としますが、充分な空"
"きがありません。選択を変更するか、またはインストーラーを終了してください。"

#: ../yuminstall.py:1301
msgid "Reboot?"
msgstr "再起動しますか?"

#: ../yuminstall.py:1302
msgid "The system will be rebooted now."
msgstr "システムが再起動します。"

#: ../yuminstall.py:1464
#, python-format
msgid ""
"You appear to be upgrading from a system which is too old to upgrade to this "
"version of %s.  Are you sure you wish to continue the upgrade process?"
msgstr ""
"%s のこのバージョンにアップグレードを行うには古過ぎるシステムからアップグレー"
"トしているようです。本当にこのアップグレードのプロセスを続行しますか?"

#: ../yuminstall.py:1491
#, python-format
msgid ""
"The arch of the release of %s you are upgrading to appears to be %s which "
"does not match your previously installed arch of %s.  This is likely to not "
"succeed.  Are you sure you wish to continue the upgrade process?"
msgstr ""
"アップグレードしようとしている %s のリリースパッケージのアーキテクチャは %s "
"で、既にインストールされているパッケージのアーキテクチャ、%s と一致しません。"
"この状況では、インストールに失敗するかもしれません。本当にこのアップグレード"
"プロセスを続行しますか?"

#: ../yuminstall.py:1538
msgid "Post Upgrade"
msgstr "Post Upgrade"

#: ../yuminstall.py:1539
msgid "Performing post upgrade configuration..."
msgstr "post upgrade の設定を実行中..."

#: ../yuminstall.py:1541
msgid "Post Install"
msgstr "Post Install"

#: ../yuminstall.py:1542
msgid "Performing post install configuration..."
msgstr "post install の設定を実行中..."

#: ../yuminstall.py:1764
msgid "Installation Progress"
msgstr "インストールの進捗"

#: ../yuminstall.py:1799
msgid "Install Starting"
msgstr "インストールを開始しています"

#: ../yuminstall.py:1800
msgid "Starting install process.  This may take several minutes..."
msgstr "インストールプロセスを開始しています。数分かかることがあります..."

#: ../yuminstall.py:1837
msgid "Dependency Check"
msgstr "依存関係チェック"

#: ../yuminstall.py:1838
msgid "Checking dependencies in packages selected for installation..."
msgstr "インストール対象パッケージの依存関係をチェック中..."

#: ../zfcp.py:49
msgid "You have not specified a device number or the number is invalid"
msgstr "デバイス番号を指定していないか番号が無効です。"

#: ../zfcp.py:51
msgid "You have not specified a worldwide port name or the name is invalid."
msgstr "WWPN を指定していないか、そのポート名が無効です。"

#: ../zfcp.py:53
msgid "You have not specified a FCP LUN or the number is invalid."
msgstr "FCP FUN を指定していないか、番号が無効です。"

#: ../iw/account_gui.py:48
msgid "Root _Password:"
msgstr "ルートパスワード (_P):"

#: ../iw/account_gui.py:50
msgid "_Confirm:"
msgstr "確認 (_C):"

#: ../iw/account_gui.py:87
msgid "Caps Lock is on."
msgstr "Caps Lock がオンになっています。"

#: ../iw/account_gui.py:97 ../iw/account_gui.py:105 ../iw/account_gui.py:112
#: ../iw/account_gui.py:132 ../textw/userauth_text.py:71
msgid "Error with Password"
msgstr "パスワードにエラー"

#: ../iw/account_gui.py:98
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""
"続行するには、 root パスワードを入力してから、もう一度同じパスワードを入力し"
"て確認しなければなりません。"

#: ../iw/account_gui.py:106
msgid "The passwords you entered were different.  Please try again."
msgstr "入力したパスワードが異ります。やり直してください。"

#: ../iw/account_gui.py:113
msgid "The root password must be at least six characters long."
msgstr "root パスワードは最低 6 文字以上でなければなりません。"

#: ../iw/account_gui.py:120 ../textw/userauth_text.py:78
msgid "Weak Password"
msgstr "パスワードが弱すぎます。"

#: ../iw/account_gui.py:121 ../textw/userauth_text.py:79
#, python-format
msgid ""
"Weak password provided: %s\n"
"\n"
"Would you like to continue with this password?"
msgstr ""
"入力されたパスワードが弱すぎます: %s\n"
"\n"
"本当にこのパスワードを使いますか?"

#: ../iw/account_gui.py:133 ../textw/userauth_text.py:72
msgid ""
"Requested password contains non-ASCII characters, which are not allowed."
msgstr ""
"要求されたパスワードには、パスワードに使用できない非 ASCII 文字が含まれていま"
"す。"

#: ../iw/autopart_type.py:96
msgid ""
"No partitions are available to resize.  Only physical partitions with "
"specific filesystems can be resized."
msgstr ""
"リサイズできるパーティションがありません。特定のファイルシステムでフォーマッ"
"トされた、物理パーティションのみリサイズできます。"

#: ../iw/autopart_type.py:179
msgid "Do you really want to boot from disk which is not used for instalation?"
msgstr "本当にインストールに使用されないディスクからシステムをブートしますか?"

#: ../iw/autopart_type.py:260
msgid "Invalid Initiator Name"
msgstr "無効なイニシエーター名です"

#: ../iw/autopart_type.py:261
msgid "You must provide an initiator name."
msgstr "イニシエーター名を入力しなければなりません。"

#: ../iw/autopart_type.py:283
msgid "Error with Data"
msgstr "データにエラー発生"

#: ../iw/autopart_type.py:403 ../textw/partition_text.py:1517
msgid "Remove all partitions on selected drives and create default layout"
msgstr ""
"選択したドライブ上のすべてのパーティションを削除してデフォルトのレイアウトを"
"作成します。"

#: ../iw/autopart_type.py:404 ../textw/partition_text.py:1518
msgid "Remove Linux partitions on selected drives and create default layout"
msgstr ""
"選択したドライブ上の linux パーティションを削除してデフォルトのレイアウトを作"
"成します。"

#: ../iw/autopart_type.py:405
msgid "Resize existing partition and create default layout in free space"
msgstr ""
"現存するパーティションのサイズを変更し、空きスペースに標準のレイアウトを作成"
"します。"

#: ../iw/autopart_type.py:406 ../textw/partition_text.py:1519
msgid "Use free space on selected drives and create default layout"
msgstr ""
"選択したドライブ上の空き領域を使用して、デフォルトレイアウトを作成します。"

#: ../iw/autopart_type.py:407 ../textw/partition_text.py:1520
msgid "Create custom layout"
msgstr "カスタムレイアウトを作成します。"

#: ../iw/blpasswidget.py:45
msgid "_Use a boot loader password"
msgstr "ブートローダーパスワードを使用(_U)"

#: ../iw/blpasswidget.py:46
msgid ""
"A boot loader password prevents users from changing kernel options, "
"increasing security."
msgstr ""
"ブートローダーパスワードによって、ユーザがカーネルオプションを変更してしまう"
"のを防ぎ、より高度なシステムセキュリティを確保します。"

#: ../iw/blpasswidget.py:77
msgid "Change _password"
msgstr "パスワードを変更(_P)"

#: ../iw/blpasswidget.py:100
msgid "Enter Boot Loader Password"
msgstr "ブートローダーパスワードを入力"

#: ../iw/blpasswidget.py:106
msgid ""
"Enter a boot loader password and then confirm it.  (Note that your BIOS "
"keymap may be different than the actual keymap you are used to.)"
msgstr ""
"ブートローダーのパスワードを入力してからそのパスワードの確認をしてください。"
"(BIOS のキーマップが実際に使用しているキーマップとは異なっている可能性がある"
"ので注意してください)"

#: ../iw/blpasswidget.py:113
msgid "_Password:"
msgstr "パスワード(_P):"

#: ../iw/blpasswidget.py:119
msgid "Con_firm:"
msgstr "確認(_F):"

#: ../iw/blpasswidget.py:140
msgid "Passwords don't match"
msgstr "パスワードが一致しません"

#: ../iw/blpasswidget.py:141 ../textw/bootloader_text.py:443
msgid "Passwords do not match"
msgstr "パスワードが一致しません"

#: ../iw/blpasswidget.py:150 ../textw/bootloader_text.py:453
msgid ""
"Your boot loader password is shorter than six characters.  We recommend a "
"longer boot loader password.\n"
"\n"
"Would you like to continue with this password?"
msgstr ""
"ブートローダーパスワードが 6 文字未満になっています。ブートローダーのパスワー"
"ドはこれより長くすることを推奨します。\n"
"\n"
"このパスワードで続行しますか?"

#: ../iw/bootloader_main_gui.py:34 ../textw/bootloader_text.py:46
#: ../textw/bootloader_text.py:111 ../textw/bootloader_text.py:168
#: ../textw/bootloader_text.py:286 ../textw/bootloader_text.py:395
msgid "Boot Loader Configuration"
msgstr "ブートローダーの設定"

#: ../iw/bootloader_main_gui.py:170 ../iw/bootloader_main_gui.py:175
#: ../iw/bootloader_main_gui.py:212
#, python-format
msgid "_Install boot loader on /dev/%s."
msgstr "ブートローダーを /dev/%s にインストールする。 (_I)"

#: ../iw/bootloader_main_gui.py:218
msgid "_Change device"
msgstr "デバイスの変更 (_C)"

#: ../iw/congrats_gui.py:31
msgid "Congratulations"
msgstr "おめでとうございます"

#: ../iw/congrats_gui.py:71
msgid ""
"Press the \"Reboot\" button to reboot your system.\n"
"\n"
msgstr "「再起動」ボタンを押すとシステムを再起動します。\n"

#: ../iw/congrats_gui.py:75
#, python-format
msgid ""
"Congratulations, the installation is complete.\n"
"\n"
"%s%s"
msgstr ""
"おめでとうございます。インストールが完了しました。\n"
"\n"
"%s%s"

#: ../iw/examine_gui.py:35
msgid "Upgrade Examine"
msgstr "アップグレードテスト"

#: ../iw/examine_gui.py:57
#, python-format
msgid "_Install %s"
msgstr "インストール(_I) %s"

#: ../iw/examine_gui.py:59
msgid ""
"Choose this option to freshly install your system. Existing software and "
"data may be overwritten depending on your configuration choices."
msgstr ""
"新しくシステムをインストールするにはこのオプションを選択します。 既存のソフト"
"ウェアやデータは選択した設定によっては上書きされるかもしれません。"

#: ../iw/examine_gui.py:63
msgid "_Upgrade an existing installation"
msgstr "既存インストールのアップグレード(_U)"

#: ../iw/examine_gui.py:65
#, python-format
msgid ""
"Choose this option if you would like to upgrade your existing %s system.  "
"This option preserves the existing data on your drives."
msgstr ""
"既存の %s システムをアップグレードする場合に、このオプションを選択します。こ"
"のオプションはドライブ上の既存データを保持します。"

#: ../iw/examine_gui.py:112 ../iw/pixmapRadioButtonGroup_gui.py:203
msgid "The following installed system will be upgraded:"
msgstr "次のインストール済みシステムがアップグレードされます。"

#: ../iw/examine_gui.py:125
msgid "Unknown Linux system"
msgstr "不明な Linux システム"

#: ../iw/GroupSelector.py:210
#, python-format
msgid "Packages in %s"
msgstr "%s に含まれるパッケージ"

#: ../iw/GroupSelector.py:482
#, python-format
msgid "%d of %d optional package selected"
msgid_plural "%d of %d optional packages selected"
msgstr[0] "%d 個中 %d 個の追加パッケージが選択されました。"
msgstr[1] "%d 個中 %d 個の追加パッケージが選択されました。"

#: ../iw/GroupSelector.py:485
#, python-format
msgid "<i>%s</i>"
msgstr "<i>%s</i>"

#: ../iw/language_gui.py:30 ../textw/language_text.py:44
msgid "Language Selection"
msgstr "言語の選択"

#: ../iw/language_gui.py:76 ../textw/language_text.py:45 ../loader2/lang.c:374
msgid "What language would you like to use during the installation process?"
msgstr "インストール時に使用する言語を選択してください。"

#: ../iw/lvm_dialog_gui.py:123 ../iw/lvm_dialog_gui.py:170
#: ../iw/lvm_dialog_gui.py:181 ../iw/lvm_dialog_gui.py:221
#: ../iw/lvm_dialog_gui.py:295 ../iw/lvm_dialog_gui.py:624
#: ../iw/lvm_dialog_gui.py:706 ../iw/lvm_dialog_gui.py:925
#: ../textw/partition_text.py:1275 ../textw/partition_text.py:1294
msgid "Not enough space"
msgstr "容量が足りません"

#: ../iw/lvm_dialog_gui.py:124
msgid ""
"The physical extent size cannot be changed because otherwise the space "
"required by the currently defined logical volumes will be increased to more "
"than the available space."
msgstr ""
"物理エクステントサイズは変更できません。変更すると現在定義されている論理ボ"
"リュームが必要とする領域が使用可能な領域以上に増加してしまうためです。"

#: ../iw/lvm_dialog_gui.py:133
msgid "Confirm Physical Extent Change"
msgstr "物理エクステントの変更を確認"

#: ../iw/lvm_dialog_gui.py:134
msgid ""
"This change in the value of the physical extent will require the sizes of "
"the current logical volume requests to be rounded up in size to an integer "
"multiple of the physical extent.\n"
"\n"
"This change will take effect immediately."
msgstr ""
"この物理エクステントの値の変更により、現在の論理ボリューム要求サイズを物理エ"
"クステントの整数倍に切り上げる必要があります。\n"
"\n"
"この変更は直ちに反映されます。"

#: ../iw/lvm_dialog_gui.py:143 ../iw/lvm_dialog_gui.py:203
#: ../iw/network_gui.py:142 ../iw/network_gui.py:146 ../iw/network_gui.py:166
msgid "C_ontinue"
msgstr "続行(_O)"

#: ../iw/lvm_dialog_gui.py:171
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"10.2f MB) is larger than the smallest physical volume (%10.2f MB) in the "
"volume group."
msgstr ""
"物理エクステントのサイズは変更できません。選択された値 (%10.2f MB) がボリュー"
"ムグループ内の最小物理ボリューム (%10.2f MB) よりも大きいためです。"

#: ../iw/lvm_dialog_gui.py:182
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"10.2f MB) is too large compared to the size of the smallest physical volume "
"(%10.2f MB) in the volume group."
msgstr ""
"物理エクステントのサイズを変更することはできません。ボリュームグループ内の最"
"小物理ボリューム (%10.2f MB) のサイズと比較して選択された値 (%10.2f MB) が大"
"きすぎるためです。"

#: ../iw/lvm_dialog_gui.py:196
msgid "Too small"
msgstr "小さすぎます"

#: ../iw/lvm_dialog_gui.py:197
msgid ""
"This change in the value of the physical extent will waste substantial space "
"on one or more of the physical volumes in the volume group."
msgstr ""
"物理エクステントの値を変更すると、ボリュームグループ内の1つまたは複数の物理ボ"
"リュームの相当な領域を浪費することになります。"

#: ../iw/lvm_dialog_gui.py:222
#, python-format
msgid ""
"The physical extent size cannot be changed because the resulting maximum "
"logical volume size (%10.2f MB) is smaller than one or more of the currently "
"defined logical volumes."
msgstr ""
"物理エクステントのサイズは変更できません。結果としてできる最大論理ボリューム"
"サイズ (%10.2f MB) が現在定義されている 1 つまたは複数の論理ボリュームよりも"
"小さくなるためです。"

#: ../iw/lvm_dialog_gui.py:296
msgid ""
"You cannot remove this physical volume because otherwise the volume group "
"will be too small to hold the currently defined logical volumes."
msgstr ""
"この物理ボリュームは削除できません。削除すると現在定義されている論理ボリュー"
"ムを保持するにはボリュームグループが小さくなりすぎるためです。"

#: ../iw/lvm_dialog_gui.py:376 ../textw/partition_text.py:1114
msgid "Make Logical Volume"
msgstr "論理ボリュームを作成"

#: ../iw/lvm_dialog_gui.py:379
#, python-format
msgid "Edit Logical Volume: %s"
msgstr "論理ボリュームを編集: %s"

#: ../iw/lvm_dialog_gui.py:381 ../textw/partition_text.py:1112
msgid "Edit Logical Volume"
msgstr "論理ボリュームを編集"

#: ../iw/lvm_dialog_gui.py:394 ../iw/partition_dialog_gui.py:322
#: ../iw/raid_dialog_gui.py:321
msgid "_Mount Point:"
msgstr "マウントポイント(_M):"

#: ../iw/lvm_dialog_gui.py:402 ../iw/raid_dialog_gui.py:334
msgid "_File System Type:"
msgstr "ファイルシステムタイプ(_F):"

#: ../iw/lvm_dialog_gui.py:412 ../iw/raid_dialog_gui.py:344
msgid "Original File System Type:"
msgstr "元のファイルシステムタイプ:"

#: ../iw/lvm_dialog_gui.py:417 ../iw/raid_dialog_gui.py:349
msgid "Unknown"
msgstr "不明"

#: ../iw/lvm_dialog_gui.py:423 ../iw/partition_dialog_gui.py:374
#: ../iw/raid_dialog_gui.py:355
msgid "Original File System Label:"
msgstr "元のファイルシステムラベル:"

#: ../iw/lvm_dialog_gui.py:432
msgid "_Logical Volume Name:"
msgstr "論理ボリューム名(_L):"

#: ../iw/lvm_dialog_gui.py:440 ../textw/partition_text.py:305
msgid "Logical Volume Name:"
msgstr "論理ボリューム名:"

#: ../iw/lvm_dialog_gui.py:448 ../iw/partition_dialog_gui.py:386
msgid "_Size (MB):"
msgstr "サイズ (MB)(_S):"

#: ../iw/lvm_dialog_gui.py:454 ../iw/partition_dialog_gui.py:403
#: ../textw/partition_text.py:320 ../textw/partition_text.py:397
#: ../textw/partition_text.py:480 ../textw/partition_text.py:588
msgid "Size (MB):"
msgstr "サイズ (MB):"

#: ../iw/lvm_dialog_gui.py:469
#, python-format
msgid "(Max size is %s MB)"
msgstr "(最大サイズは %s MB)"

#: ../iw/lvm_dialog_gui.py:478 ../iw/partition_dialog_gui.py:334
#: ../iw/partition_ui_helpers_gui.py:300 ../iw/raid_dialog_gui.py:329
msgid "_Encrypt"
msgstr "暗号化 (_E)"

#: ../iw/lvm_dialog_gui.py:543 ../textw/partition_text.py:1264
msgid "Illegal size"
msgstr "不適切なサイズ"

#: ../iw/lvm_dialog_gui.py:544 ../textw/partition_text.py:1265
msgid "The requested size as entered is not a valid number greater than 0."
msgstr "入力された要求サイズは 0 より大きい有効な数値ではありません。"

#: ../iw/lvm_dialog_gui.py:579
msgid "Mount point in use"
msgstr "使用中のマウントポイント"

#: ../iw/lvm_dialog_gui.py:580
#, python-format
msgid "The mount point \"%s\" is in use. Please pick another."
msgstr "マウントポイント \"%s\" は使用中です。他を選択してください。"

#: ../iw/lvm_dialog_gui.py:591 ../textw/partition_text.py:1235
msgid "Illegal Logical Volume Name"
msgstr "不適切な論理ボリューム名"

#: ../iw/lvm_dialog_gui.py:610 ../textw/partition_text.py:1252
msgid "Illegal logical volume name"
msgstr "不適切な論理ボリューム名"

#: ../iw/lvm_dialog_gui.py:611 ../textw/partition_text.py:1253
#, python-format
msgid "The logical volume name \"%s\" is already in use. Please pick another."
msgstr ""
"論理ボリューム名 \"%s\" はすでに使用中です。他の名前を選択してください。"

#: ../iw/lvm_dialog_gui.py:625
#, python-format
msgid ""
"The current requested size (%10.2f MB) is larger than the maximum logical "
"volume size (%10.2f MB). To increase this limit you can create more Physical "
"Volumes from unpartitioned disk space and add them to this Volume Group."
msgstr ""
"現在の要求サイズ (%10.2f MB) は、最大論理ボリュームサイズ (%10.2f MB) より大"
"きくなります。この上限を増やす場合は、パーティションが未設定のディスク領域か"
"ら物理ボリュームを作成し、このボリュームグループに追加します。"

#: ../iw/lvm_dialog_gui.py:683 ../iw/partition_dialog_gui.py:197
#: ../iw/partition_dialog_gui.py:209 ../iw/partition_dialog_gui.py:268
#: ../iw/raid_dialog_gui.py:248 ../textw/partition_text.py:888
#: ../textw/partition_text.py:909 ../textw/partition_text.py:1082
#: ../textw/partition_text.py:1313
msgid "Error With Request"
msgstr "要求された設定にエラーがあります"

#: ../iw/lvm_dialog_gui.py:707 ../iw/lvm_dialog_gui.py:926
#, python-format
msgid ""
"The logical volumes you have configured require %d MB, but the volume group "
"only has %d MB.  Please either make the volume group larger or make the "
"logical volume(s) smaller."
msgstr ""
"設定している論理ボリュームには %d MBが必要となりますが、ボリュームグループに"
"は %d MBしかありません。ボリュームグループを大きくするか、論理ボリュームを小"
"さくしてください。"

#: ../iw/lvm_dialog_gui.py:765
msgid "No free slots"
msgstr "空きスロットがありません"

#: ../iw/lvm_dialog_gui.py:766
#, python-format
msgid "You cannot create more than %s logical volumes per volume group."
msgstr "1 ボリュームグループに対して %s 以上の論理ボリュームは作成できません。"

#: ../iw/lvm_dialog_gui.py:772
msgid "No free space"
msgstr "空き領域はありません"

#: ../iw/lvm_dialog_gui.py:773
msgid ""
"There is no room left in the volume group to create new logical volumes. To "
"add a logical volume you must reduce the size of one or more of the "
"currently existing logical volumes"
msgstr ""
"新規の論理ボリュームを作成する空き領域がボリュームグループには残っていませ"
"ん。論理ボリュームを追加するには、ひとつ以上の既存の論理ボリュームのサイズを"
"縮小する必要があります。"

#: ../iw/lvm_dialog_gui.py:801
#, python-format
msgid "Are you sure you want to delete the logical volume \"%s\"?"
msgstr "本当に論理ボリューム 「%s」 を削除しますか?"

#: ../iw/lvm_dialog_gui.py:937
msgid "Invalid Volume Group Name"
msgstr "無効なボリュームグループ名"

#: ../iw/lvm_dialog_gui.py:948
msgid "Name in use"
msgstr "名前は使用中です"

#: ../iw/lvm_dialog_gui.py:949
#, python-format
msgid "The volume group name \"%s\" is already in use. Please pick another."
msgstr ""
"ボリュームグループ名 \"%s\" はすでに使用されています。別の名前を選択してくだ"
"さい。"

#: ../iw/lvm_dialog_gui.py:992
msgid "Not enough physical volumes"
msgstr "物理ボリュームが足りません"

#: ../iw/lvm_dialog_gui.py:993
msgid ""
"At least one unused physical volume partition is needed to create an LVM "
"Volume Group.\n"
"\n"
"Create a partition or RAID array of type \"physical volume (LVM)\" and then "
"select the \"LVM\" option again."
msgstr ""
"LVM ボリュームグループを作成するには、未使用の物理ボリュームパーティションが"
"少なくとも 1 つ必要です。\n"
"\n"
"「物理ボリューム (LVM)」タイプのパーティションまたは RAID アレイを作成してか"
"ら、 「LVM」 オプションを選択し直してください。"

#: ../iw/lvm_dialog_gui.py:1004
msgid "Make LVM Volume Group"
msgstr "LVM ボリュームグループの作成"

#: ../iw/lvm_dialog_gui.py:1007
#, python-format
msgid "Edit LVM Volume Group: %s"
msgstr "LVM ボリュームグループの編集: %s"

#: ../iw/lvm_dialog_gui.py:1009
msgid "Edit LVM Volume Group"
msgstr "LVM ボリュームグループの編集"

#: ../iw/lvm_dialog_gui.py:1025
msgid "_Volume Group Name:"
msgstr "ボリュームグループ名(_V):"

#: ../iw/lvm_dialog_gui.py:1033
msgid "Volume Group Name:"
msgstr "ボリュームグループ名:"

#: ../iw/lvm_dialog_gui.py:1041
msgid "_Physical Extent:"
msgstr "物理エクステント(_P): "

#: ../iw/lvm_dialog_gui.py:1056
msgid "Physical Volumes to _Use:"
msgstr "使用する物理ボリューム(_U):"

#: ../iw/lvm_dialog_gui.py:1062
msgid "Used Space:"
msgstr "使用中の領域:"

#: ../iw/lvm_dialog_gui.py:1079
msgid "Free Space:"
msgstr "空き領域:"

#: ../iw/lvm_dialog_gui.py:1097
msgid "Total Space:"
msgstr "合計領域:"

#: ../iw/lvm_dialog_gui.py:1126
msgid "Logical Volume Name"
msgstr "論理ボリューム名"

#: ../iw/lvm_dialog_gui.py:1129 ../iw/partition_gui.py:369
#: ../iw/upgrade_swap_gui.py:140 ../textw/partition_text.py:1423
#: ../textw/upgrade_text.py:120
msgid "Mount Point"
msgstr "マウントポイント"

#: ../iw/lvm_dialog_gui.py:1132 ../iw/partition_gui.py:374
msgid "Size (MB)"
msgstr "サイズ (MB)"

#: ../iw/lvm_dialog_gui.py:1146 ../iw/osbootwidget.py:95
msgid "_Add"
msgstr "追加(_A)"

#: ../iw/lvm_dialog_gui.py:1149 ../iw/network_gui.py:457
#: ../iw/osbootwidget.py:99 ../iw/partition_gui.py:1379
msgid "_Edit"
msgstr "編集(_E)"

#: ../iw/lvm_dialog_gui.py:1164
msgid "_Logical Volumes"
msgstr "論理ボリューム (_L)"

#: ../iw/netconfig_dialog.py:143
#, python-format
msgid ""
"An error occurred converting the value entered for \"%s\":\n"
"%s"
msgstr ""
"\"%s\" に入力された値を変換中にエラーが発生:\n"
"%s"

#: ../iw/netconfig_dialog.py:145 ../iw/netconfig_dialog.py:154
#: ../iw/network_gui.py:141 ../iw/network_gui.py:145 ../iw/network_gui.py:149
#: ../iw/network_gui.py:153 ../iw/network_gui.py:161 ../iw/network_gui.py:166
#: ../textw/network_text.py:52 ../textw/network_text.py:57
#: ../textw/network_text.py:63
msgid "Error With Data"
msgstr "データにエラーがあります"

#: ../iw/netconfig_dialog.py:153 ../iw/network_gui.py:154
#: ../textw/network_text.py:53
#, python-format
msgid "A value is required for the field %s."
msgstr "フィールド %s には値が必要です。"

#: ../iw/netconfig_dialog.py:176
msgid "Dynamic IP"
msgstr "動的 IP"

#: ../iw/netconfig_dialog.py:177 ../loader2/net.c:416 ../loader2/net.c:860
#, c-format, python-format
msgid "Sending request for IP information for %s..."
msgstr "%s の IP 情報の要求を送信中..."

#: ../iw/netconfig_dialog.py:198 ../iw/netconfig_dialog.py:201
#: ../textw/network_text.py:93 tmp/netpostconfig.glade.h:12
msgid "IP Address"
msgstr "IP アドレス"

#: ../iw/netconfig_dialog.py:208
msgid "IPv4 CIDR prefix must be between 0 and 32."
msgstr "IPv4 CIDR の接頭辞は 0 から 32 の間にしてください。"

#: ../iw/netconfig_dialog.py:209 ../iw/netconfig_dialog.py:215
#: ../iw/netconfig_dialog.py:223 ../iw/netconfig_dialog.py:226
msgid "IPv4 Network Mask"
msgstr "IPv4 ネットワークマスク"

#: ../iw/netconfig_dialog.py:233 ../iw/netconfig_dialog.py:236
#: ../iw/network_gui.py:34 ../iw/network_gui.py:533
msgid "Gateway"
msgstr "ゲートウェイ"

#: ../iw/netconfig_dialog.py:243
msgid "Nameserver"
msgstr "ネームサーバー"

#: ../iw/netconfig_dialog.py:253
msgid "Error configuring network device:"
msgstr "ネットワークデバイス設定中にエラー発生:"

#: ../iw/netconfig_dialog.py:266
msgid "Error configuring network device"
msgstr "ネットワークデバイス設定中にエラー発生"

#: ../iw/network_gui.py:34 ../iw/network_gui.py:535
msgid "Primary DNS"
msgstr "1 番目の DNS"

#: ../iw/network_gui.py:34 ../iw/network_gui.py:537
msgid "Secondary DNS"
msgstr "2 番目の DNS"

#: ../iw/network_gui.py:35
msgid "_Gateway"
msgstr "ゲートウェイ(_G)"

#: ../iw/network_gui.py:35
msgid "_Primary DNS"
msgstr "1 番目の DNS(_P)"

#: ../iw/network_gui.py:35
msgid "_Secondary DNS"
msgstr "2 番目の DNS(_S)"

#: ../iw/network_gui.py:38 ../textw/network_text.py:532
msgid "Network Configuration"
msgstr "ネットワークの設定"

#: ../iw/network_gui.py:142
msgid ""
"You have not specified a hostname.  Depending on your network environment "
"this may cause problems later."
msgstr ""
"ホスト名を指定していません。ネットワーク環境によっては後で問題となる可能性が"
"あります。"

#: ../iw/network_gui.py:146
#, python-format
msgid ""
"You have not specified the field \"%s\".  Depending on your network "
"environment this may cause problems later."
msgstr ""
"フィールド \"%s\" を指定していません。ネットワーク環境によっては後で問題とな"
"る可能性があります。"

#: ../iw/network_gui.py:150 ../textw/network_text.py:820
#, python-format
msgid ""
"The hostname \"%s\" is not valid for the following reason:\n"
"\n"
"%s"
msgstr ""
"ホスト名 \"%s\" は、次のような理由で有効ではありません。\n"
"\n"
"%s"

#: ../iw/network_gui.py:157 ../textw/network_text.py:43
#, python-format
msgid "Error With %s Data"
msgstr "%s データにエラーがあります"

#: ../iw/network_gui.py:162 ../textw/network_text.py:64
msgid "The IPv4 information you have entered is invalid."
msgstr "入力した IPv4 情報は無効です。"

#: ../iw/network_gui.py:166
msgid ""
"You have no active network devices.  Your system will not be able to "
"communicate over a network by default without at least one device active."
msgstr ""
"アクティブなネットワークデバイスがありません。少なくとも 1 つのデバイスがアク"
"ティブでないとデフォルトではシステムがネットワーク上で通信できません。"

#: ../iw/network_gui.py:291 ../iw/network_gui.py:305
msgid "Disabled"
msgstr "使用不可"

#: ../iw/network_gui.py:398
msgid "Active on Boot"
msgstr "起動時にアクティブ"

#: ../iw/network_gui.py:400 ../iw/osbootwidget.py:66
#: ../iw/partition_gui.py:367 ../textw/bootloader_text.py:196
#: ../textw/bootloader_text.py:262 ../textw/partition_text.py:1423
msgid "Device"
msgstr "デバイス"

#: ../iw/network_gui.py:402
msgid "IPv4/Netmask"
msgstr "IPv4/ネットマスク"

#: ../iw/network_gui.py:404
msgid "IPv6/Prefix"
msgstr "IPv6/プレフィックス"

#: ../iw/network_gui.py:464
msgid "Network Devices"
msgstr "ネットワークデバイス"

#: ../iw/network_gui.py:475
msgid "Set the hostname:"
msgstr "ホスト名を設定:"

#: ../iw/network_gui.py:480
msgid "_automatically via DHCP"
msgstr "DHCP経由で自動設定(_A)"

#: ../iw/network_gui.py:487
msgid "_manually"
msgstr "手動設定(_M)"

#: ../iw/network_gui.py:492
msgid "(e.g., host.domain.com)"
msgstr "(例、 host.domain.com)"

#: ../iw/network_gui.py:498
msgid "Hostname"
msgstr "ホスト名"

#: ../iw/network_gui.py:542
msgid "Miscellaneous Settings"
msgstr "その他の設定"

#: ../iw/network_gui.py:630
msgid "Edit Device "
msgstr "デバイスの編集"

#: ../iw/network_gui.py:634
msgid "Unknown Ethernet Device"
msgstr "不明なイーサネットデバイス"

#: ../iw/network_gui.py:640
msgid "unknown"
msgstr "不明"

#: ../iw/network_gui.py:642
msgid "Hardware address: "
msgstr "ハードウェアのアドレス: "

#: ../iw/network_gui.py:778 ../textw/network_text.py:228 ../loader2/net.c:820
msgid "Missing Protocol"
msgstr "見つからないプロトコル"

#: ../iw/network_gui.py:779 ../textw/network_text.py:229
msgid "You must select at least IPv4 or IPv6 support."
msgstr "少なくとも、 IPv4 または IPv6 サポートを選択しなくてはなりません。"

#: ../iw/network_gui.py:802 ../iw/network_gui.py:843 ../iw/network_gui.py:849
#: ../textw/network_text.py:74 ../loader2/net.c:92
msgid "Invalid Prefix"
msgstr "無効なプレフィックス"

#: ../iw/network_gui.py:803
msgid "IPv4 prefix must be between 0 and 32."
msgstr "IPv4 のプレフィックスは 0 から 32 の間にしてください。"

#: ../iw/network_gui.py:844 ../iw/network_gui.py:850
msgid "IPv6 prefix must be between 0 and 128."
msgstr "IPv6 のプレフィックスは 0 から 128 の間にしてください。"

#: ../iw/osbootwidget.py:50
msgid "Boot loader operating system list"
msgstr "ブートローダーのオペレーティングシステムのリスト"

#: ../iw/osbootwidget.py:66 ../textw/bootloader_text.py:262
msgid "Default"
msgstr "デフォルト"

#: ../iw/osbootwidget.py:66 ../iw/partition_gui.py:368
msgid "Label"
msgstr "ラベル"

#: ../iw/osbootwidget.py:129
msgid "Image"
msgstr "イメージ"

#: ../iw/osbootwidget.py:136
msgid ""
"Enter a label for the boot loader menu to display. The device (or hard drive "
"and partition number) is the device from which it boots."
msgstr ""
"ブートローダーメニューに表示されるラベルを入力します。デバイス(または、ハード"
"ドライブとパーティション番号)とは、起動用に使うデバイスのことです。"

#: ../iw/osbootwidget.py:144
msgid "_Label"
msgstr "ラベル(_L)"

#: ../iw/osbootwidget.py:152
msgid "_Device"
msgstr "デバイス(_D)"

#: ../iw/osbootwidget.py:186
msgid "Default Boot _Target"
msgstr "デフォルトのブートターゲット(_T)"

#: ../iw/osbootwidget.py:215
msgid "You must specify a label for the entry"
msgstr "ブートエントリのラベルを指定してください"

#: ../iw/osbootwidget.py:224
msgid "Boot label contains illegal characters"
msgstr "ブートラベルに不適切な文字があります"

#: ../iw/osbootwidget.py:248
msgid "Duplicate Label"
msgstr "ラベルが重複しています"

#: ../iw/osbootwidget.py:249
msgid "This label is already in use for another boot entry."
msgstr "このラベルは既に別のブートエントリに使用されています。"

#: ../iw/osbootwidget.py:262
msgid "Duplicate Device"
msgstr "デバイスが重複しています"

#: ../iw/osbootwidget.py:263
msgid "This device is already being used for another boot entry."
msgstr "このデバイスは既に別のブートエントリに使用されています。"

#: ../iw/osbootwidget.py:327 ../textw/bootloader_text.py:345
msgid "Cannot Delete"
msgstr "削除できません"

#: ../iw/osbootwidget.py:328 ../textw/bootloader_text.py:346
#, python-format
msgid ""
"This boot target cannot be deleted because it is for the %s system you are "
"about to install."
msgstr ""
"このブートターゲットは、インストールしようとしている %s システム用になってい"
"るため削除できません。"

#: ../iw/partition_dialog_gui.py:66
msgid "Additional Size Options"
msgstr "追加容量オプション"

#: ../iw/partition_dialog_gui.py:71
msgid "_Fixed size"
msgstr "固定容量(_F)"

#: ../iw/partition_dialog_gui.py:73
msgid "Fill all space _up to (MB):"
msgstr "指定限度まで使用(MB)(_U):"

#: ../iw/partition_dialog_gui.py:83
msgid "Fill to maximum _allowable size"
msgstr "最大許容量まで使用(_A)"

#: ../iw/partition_dialog_gui.py:198
msgid "The end cylinder must be greater than the start cylinder."
msgstr "終了シリンダーは開始シリンダーよりも大きくなければなりません。"

#: ../iw/partition_dialog_gui.py:299 ../textw/partition_text.py:712
msgid "Add Partition"
msgstr "パーティションの追加"

#: ../iw/partition_dialog_gui.py:302
#, python-format
msgid "Edit Partition: /dev/%s"
msgstr "パーティションの編集: /dev/%s"

#: ../iw/partition_dialog_gui.py:304
msgid "Edit Partition"
msgstr "パーティションの編集"

#: ../iw/partition_dialog_gui.py:331
msgid "File System _Type:"
msgstr "ファイルシステムタイプ(_T):"

#: ../iw/partition_dialog_gui.py:350
msgid "Allowable _Drives:"
msgstr "使用可能なドライブ(_D):"

#: ../iw/partition_dialog_gui.py:365
msgid "Drive:"
msgstr "ドライブ:"

#: ../iw/partition_dialog_gui.py:409
msgid "_Start Cylinder:"
msgstr "開始シリンダー(_S):"

#: ../iw/partition_dialog_gui.py:427
msgid "_End Cylinder:"
msgstr "終了シリンダー(_E):"

#: ../iw/partition_dialog_gui.py:475
msgid "Force to be a _primary partition"
msgstr "基本パーティションにする(_P)"

#: ../iw/partition_gui.py:318
#, python-format
msgid "Drive %s (Geom: %s/%s/%s) (Model: %s)"
msgstr "ドライブ %s (Geom: %s/%s/%s) (モデル: %s)"

#: ../iw/partition_gui.py:325
#, python-format
msgid "Drive %s (%-0.f MB) (Model: %s)"
msgstr "ドライブ %s (%-0.f MB) (モデル: %s)"

#: ../iw/partition_gui.py:370 ../textw/partition_text.py:1423
msgid "Type"
msgstr "タイプ"

#: ../iw/partition_gui.py:375 ../textw/partition_text.py:1423
msgid "Start"
msgstr "開始"

#: ../iw/partition_gui.py:376 ../textw/partition_text.py:1423
msgid "End"
msgstr "終了"

#: ../iw/partition_gui.py:413
msgid ""
"Mount Point/\n"
"RAID/Volume"
msgstr ""
"マウントポイント/\n"
"RAID/ボリューム"

#: ../iw/partition_gui.py:415
msgid ""
"Size\n"
"(MB)"
msgstr ""
"容量\n"
"(MB)"

#: ../iw/partition_gui.py:547 ../textw/partition_text.py:1417
msgid "Partitioning"
msgstr "パーティション設定"

#: ../iw/partition_gui.py:638
msgid ""
"The partitioning scheme you requested caused the following critical errors."
msgstr ""
"あなたが要求したパーティション設定案は以下の重大なエラーを起こしました。"

#: ../iw/partition_gui.py:640
#, python-format
msgid ""
"You must correct these errors before you continue your installation of %s."
msgstr ""
"%s のインストールを続行する前にこれらのエラーを修正する必要があります。"

#: ../iw/partition_gui.py:646
msgid "Partitioning Errors"
msgstr "パーティション設定エラー"

#: ../iw/partition_gui.py:652
msgid "The partitioning scheme you requested generated the following warnings."
msgstr "あなたが要求したパーティション設定案には以下の警告があります。"

#: ../iw/partition_gui.py:654
msgid "Would you like to continue with your requested partitioning scheme?"
msgstr "このパーティション設定のまま続行しますか?"

#: ../iw/partition_gui.py:659
msgid "Partitioning Warnings"
msgstr "パーティション設定の警告"

#: ../iw/partition_gui.py:681
msgid "Format Warnings"
msgstr "フォーマットの警告"

#: ../iw/partition_gui.py:686
msgid "_Format"
msgstr "フォーマット(_F)"

#: ../iw/partition_gui.py:723
msgid "LVM Volume Groups"
msgstr "LVM ボリュームグループ"

#: ../iw/partition_gui.py:764
msgid "RAID Devices"
msgstr "RAID デバイス"

#: ../iw/partition_gui.py:794 ../iw/partition_gui.py:928
#: ../textw/partition_text.py:114 ../textw/partition_text.py:178
msgid "None"
msgstr "なし"

#: ../iw/partition_gui.py:816 ../loader2/hdinstall.c:334
msgid "Hard Drives"
msgstr "ハードドライブ"

#: ../iw/partition_gui.py:889 ../textw/partition_text.py:160
#: ../textw/partition_text.py:199
msgid "Free space"
msgstr "空き領域"

#: ../iw/partition_gui.py:891 ../textw/partition_text.py:162
msgid "Extended"
msgstr "拡張領域"

#: ../iw/partition_gui.py:893 ../textw/partition_text.py:164
msgid "software RAID"
msgstr "ソフトウェア RAID"

#: ../iw/partition_gui.py:930
msgid "Free"
msgstr "空き"

#: ../iw/partition_gui.py:1024 ../textw/partition_text.py:247
#, python-format
msgid "Could not allocate requested partitions: %s."
msgstr "要求されたパーティションを設定できませんでした: %s"

#: ../iw/partition_gui.py:1033
#, python-format
msgid "Warning: %s."
msgstr "警告: %s"

#: ../iw/partition_gui.py:1215 ../iw/partition_gui.py:1229
msgid "Not supported"
msgstr "サポートされていません"

#: ../iw/partition_gui.py:1216
msgid "LVM is NOT supported on this platform."
msgstr "LVM は、このプラットフォームではサポートされていません。"

#: ../iw/partition_gui.py:1230
msgid "Software RAID is NOT supported on this platform."
msgstr "ソフトウェア RAID はこのプラットフォームではサポートされていません。"

#: ../iw/partition_gui.py:1237
msgid "No RAID minor device numbers available"
msgstr "RAID のマイナーデバイス番号は利用できません"

#: ../iw/partition_gui.py:1238
msgid ""
"A software RAID device cannot be created because all of the available RAID "
"minor device numbers have been used."
msgstr ""
"RAID で使用できるマイナーデバイス番号はすべて使用中のため、ソフトウェア RAID "
"デバイスは作成できません。"

#: ../iw/partition_gui.py:1252
msgid "RAID Options"
msgstr "RAID オプション"

#: ../iw/partition_gui.py:1263
#, python-format
msgid ""
"Software RAID allows you to combine several disks into a larger RAID "
"device.  A RAID device can be configured to provide additional speed and "
"reliability compared to using an individual drive.  For more information on "
"using RAID devices please consult the %s documentation.\n"
"\n"
"You currently have %s software RAID partition(s) free to use.\n"
"\n"
msgstr ""
"ソフトウェア RAID を使用すれば、複数のディスクを結合して 1 つの大きな RAID デ"
"バイスとすることができます。RAID デバイスは、個別のドライブ使用に比べ速度と信"
"頼性を高めるように設定することができます。RAID デバイスの使用方法については %"
"s ドキュメントを参照してください。\n"
" \n"
"現在、%s ソフトウェア RAID パーティションが使用できます。\n"
"\n"

#: ../iw/partition_gui.py:1274
msgid ""
"To use RAID you must first create at least two partitions of type 'software "
"RAID'.  Then you can create a RAID device that can be formatted and "
"mounted.\n"
"\n"
msgstr ""
"RAIDを使用するには、まず「ソフトウェア RAID」タイプのパーティションを少くと"
"も 2 つ作成する必要があります。そうすると、フォーマット及びマウントが可能な "
"RAID デバイスが作成できるようになります。\n"
"\n"

#: ../iw/partition_gui.py:1280
msgid "What do you want to do now?"
msgstr "どうしますか?"

#: ../iw/partition_gui.py:1289
msgid "Create a software RAID _partition."
msgstr "ソフトウェア RAID パーティションを作成します(_P)。"

#: ../iw/partition_gui.py:1292
#, python-format
msgid "Create a RAID _device [default=/dev/md%s]."
msgstr "RAID デバイスを作成します [default=/dev/md%s](_D)。"

#: ../iw/partition_gui.py:1296
#, python-format
msgid "Clone a _drive to create a RAID device [default=/dev/md%s]."
msgstr ""
"RAID デバイスを作成するためにドライブのクローンを作成します [default=/dev/md%"
"s](_D)。"

#: ../iw/partition_gui.py:1335
msgid "Couldn't Create Drive Clone Editor"
msgstr "ドライブクローンエディターを作成できませんでした"

#: ../iw/partition_gui.py:1336
msgid "The drive clone editor could not be created for some reason."
msgstr "ドライブクローンエディターは、何らかの理由で作成できませんでした。"

#: ../iw/partition_gui.py:1378
msgid "Ne_w"
msgstr "新規(_W)"

#: ../iw/partition_gui.py:1381
msgid "Re_set"
msgstr "リセット(_S)"

#: ../iw/partition_gui.py:1382
msgid "R_AID"
msgstr "RAID(_A)"

#: ../iw/partition_gui.py:1383
msgid "_LVM"
msgstr "_LVM(_L)"

#: ../iw/partition_gui.py:1424
msgid "Hide RAID device/LVM Volume _Group members"
msgstr "RAID デバイス/LVM ボリュームグループメンバーを非表示にする(_G)"

#: ../iw/partition_ui_helpers_gui.py:102 ../iw/partition_ui_helpers_gui.py:123
#: ../iw/partition_ui_helpers_gui.py:125 ../textw/partition_text.py:270
#: ../textw/partition_text.py:272 ../textw/partition_text.py:274
#: ../textw/partition_text.py:299
msgid "<Not Applicable>"
msgstr "<利用不可>"

#: ../iw/partition_ui_helpers_gui.py:284
msgid "_Format as:"
msgstr "フォーマットする (_F):"

#: ../iw/partition_ui_helpers_gui.py:307
msgid "Mi_grate filesystem to:"
msgstr "ファイルシステムを移動する:"

#: ../iw/partition_ui_helpers_gui.py:327 tmp/autopart.glade.h:9
msgid "_Resize"
msgstr "サイズを変更 (_R)"

#: ../iw/partition_ui_helpers_gui.py:381
#, python-format
msgid ""
"Partitions of type '%s' must be constrained to a single drive.  To do this, "
"select the drive in the 'Allowable Drives' checklist."
msgstr ""
"タイプ '%s' のパーティションは単独ドライブに入れる必要があります。これは行う"
"には、「選択可能なドライブ」のチェックリストからそのドライブを選択します。"

#: ../iw/partmethod_gui.py:31 ../textw/partmethod_text.py:30
msgid "Disk Partitioning Setup"
msgstr "ディスクパーティション設定"

#: ../iw/partmethod_gui.py:56
msgid "_Automatically partition"
msgstr "自動パーティション設定(_A)"

#: ../iw/partmethod_gui.py:59
msgid "Manually partition with _Disk Druid"
msgstr "Disk Druid を使用して手動パーティション設定(_D)"

#: ../iw/progress_gui.py:38
msgid "Installing Packages"
msgstr "パッケージをインストール中"

#: ../iw/raid_dialog_gui.py:288
msgid ""
"At least two unused software RAID partitions are needed to create a RAID "
"device.\n"
"\n"
"First create at least two partitions of type \"software RAID\", and then "
"select the \"RAID\" option again."
msgstr ""
"RAID デバイスを作成するには、少なくとも 2 つの未使用ソフトウェア RAID パー"
"ティションが必要です。\n"
" \n"
"最初に「ソフトウェア RAID」タイプのパーティションを少なくとも2つ作成してか"
"ら、再度「RAID」オプションを選択してください。"

#: ../iw/raid_dialog_gui.py:302 ../iw/raid_dialog_gui.py:735
#: ../textw/partition_text.py:944
msgid "Make RAID Device"
msgstr "RAID デバイスの作成"

#: ../iw/raid_dialog_gui.py:305
#, python-format
msgid "Edit RAID Device: /dev/md%s"
msgstr "RAID デバイスの編集: /dev/md%s"

#: ../iw/raid_dialog_gui.py:307 ../textw/partition_text.py:942
msgid "Edit RAID Device"
msgstr "RAID デバイスの編集"

#: ../iw/raid_dialog_gui.py:363
msgid "RAID _Device:"
msgstr "RAID デバイス (_D):"

#: ../iw/raid_dialog_gui.py:381
msgid "RAID _Level:"
msgstr "RAID レベル(_L):"

#: ../iw/raid_dialog_gui.py:422
msgid "_RAID Members:"
msgstr "RAID メンバー(_R):"

#: ../iw/raid_dialog_gui.py:439
msgid "Number of _spares:"
msgstr "スペア数(_S):"

#: ../iw/raid_dialog_gui.py:449
msgid "_Format partition?"
msgstr "パーティションをフォーマットしますか(_F)?"

#: ../iw/raid_dialog_gui.py:536
msgid ""
"The source drive has no partitions to be cloned.  You must first define "
"partitions of type 'software RAID' on this drive before it can be cloned."
msgstr ""
"ソースドライブにはクローンを作成するパーティションがありません。クローンを作"
"成する前に、このドライブ上で「ソフトウェア RAID」タイプのパーティションを定義"
"する必要があります。"

#: ../iw/raid_dialog_gui.py:540 ../iw/raid_dialog_gui.py:546
#: ../iw/raid_dialog_gui.py:558 ../iw/raid_dialog_gui.py:571
msgid "Source Drive Error"
msgstr "ソースドライブエラーです"

#: ../iw/raid_dialog_gui.py:547
msgid ""
"The source drive you selected has partitions which are not of type 'software "
"RAID'.\n"
"\n"
"You must remove these partitions before this drive can be cloned. "
msgstr ""
"選択されたソースドライブには「ソフトウェア RAID」タイプではないパーティション"
"があります。\n"
"\n"
"これらのパーティションを削除しないと、このドライブのクローンを作成できませ"
"ん。"

#: ../iw/raid_dialog_gui.py:559
#, python-format
msgid ""
"The source drive you selected has partitions which are not constrained to "
"the drive /dev/%s.\n"
"\n"
"You must remove these partitions or restrict them to this drive before this "
"drive can be cloned. "
msgstr ""
"選択されたソースドライブにはドライブ /dev/%s に入っていないパーティションがあ"
"ります。\n"
"\n"
"これらのパーティションを削除するか、このドライブに制限しないと、このドライブ"
"のクローンを作成できません。"

#: ../iw/raid_dialog_gui.py:572
msgid ""
"The source drive you selected has software RAID partition(s) which are "
"members of an active software RAID device.\n"
"\n"
"You must remove these partitions before this drive can be cloned."
msgstr ""
"選択されたソースドライブにはアクティブなソフトウェア RAID デバイスのメンバー"
"であるソフトウェア RAID パーティションがあります。\n"
"\n"
"これらのパーティションを削除しないと、このドライブのクローンを作成できませ"
"ん。"

#: ../iw/raid_dialog_gui.py:585 ../iw/raid_dialog_gui.py:591
#: ../iw/raid_dialog_gui.py:610
msgid "Target Drive Error"
msgstr "ターゲットドライブのエラー"

#: ../iw/raid_dialog_gui.py:586
msgid "Please select the target drives for the clone operation."
msgstr "クローン作成を行うターゲットドライブを選択してください。"

#: ../iw/raid_dialog_gui.py:592
#, python-format
msgid "The source drive /dev/%s cannot be selected as a target drive as well."
msgstr ""
"ソースドライブ/dev/%sは、同時にターゲットドライブとして選択できません。"

#: ../iw/raid_dialog_gui.py:611
#, python-format
msgid ""
"The target drive /dev/%s has a partition which cannot be removed for the "
"following reason:\n"
"\n"
"\"%s\"\n"
"\n"
"You must remove this partition before this drive can be a target."
msgstr ""
"ターゲットドライブ /dev/%s には、次の理由で削除できないパーティションがありま"
"す。\n"
" \n"
"\"%s\"\n"
"\n"
"このドライブをターゲットにする前に、このパーティションを削除する必要がありま"
"す。"

#: ../iw/raid_dialog_gui.py:672
msgid "Please select a source drive."
msgstr "ソースドライブを選択してください。"

#: ../iw/raid_dialog_gui.py:692
#, python-format
msgid ""
"The drive /dev/%s will now be cloned to the following drives:\n"
"\n"
msgstr ""
"ドライブ /dev/%s が次のドライブに対してクローン作成されます。\n"
"\n"

#: ../iw/raid_dialog_gui.py:697
msgid ""
"\n"
"\n"
"WARNING! ALL DATA ON THE TARGET DRIVES WILL BE DESTROYED."
msgstr ""
"\n"
"\n"
"警告! ターゲットドライブ上のすべてのデータは消失します。"

#: ../iw/raid_dialog_gui.py:700
msgid "Final Warning"
msgstr "最終警告"

#: ../iw/raid_dialog_gui.py:702
msgid "Clone Drives"
msgstr "クローンドライブ"

#: ../iw/raid_dialog_gui.py:711
msgid "There was an error clearing the target drives.  Cloning failed."
msgstr ""
"ターゲットドライブ内を消去している際にエラーが発生しました。クローン作成は失"
"敗しました。"

#: ../iw/raid_dialog_gui.py:745
msgid ""
"Clone Drive Tool\n"
"\n"
"This tool allows you to significantly reduce the amount of effort required "
"to setup RAID arrays.  This tool uses a source drive which has been prepared "
"with the desired partitioning layout, and clones this layout onto other "
"similar sized drives.  Then a RAID device can be created.\n"
"\n"
"NOTE: The source drive must have partitions which are restricted to be on "
"that drive only, and can only contain unused software RAID partitions.  "
"Other partition types are not allowed.\n"
"\n"
"EVERYTHING on the target drive(s) will be destroyed by this process."
msgstr ""
"クローンドライブツール\n"
"\n"
"このツールを使用すると RAID アレイの設定に必要な手間を大幅に省くことができま"
"す。希望するパーティションレイアウトで準備したソースドライブを使い、このレイ"
"アウトを他の類似サイズのドライブにクローニングします。そうすると、 RAID デバ"
"イスを作成することができます。\n"
" \n"
"注意：ソースドライブは、そのドライブ以外に存在するパーティションを持てず、ま"
"た、未使用のソフトウェア RAID パーティションしか持てません。他のパーティショ"
"ンタイプは使用できません。\n"
"\n"
"ターゲットドライブ上にあるものはすべてこのプロセスによって消去されます。"

#: ../iw/raid_dialog_gui.py:765
msgid "Source Drive:"
msgstr "ソースドライブ:"

#: ../iw/raid_dialog_gui.py:773
msgid "Target Drive(s):"
msgstr "ターゲットドライブ:"

#: ../iw/raid_dialog_gui.py:781
msgid "Drives"
msgstr "ドライブ"

#: ../iw/task_gui.py:44
#, python-format
msgid ""
"Unable to read package metadata from repository.  This may be due to a "
"missing repodata directory.  Please ensure that your repository has been "
"correctly generated.\n"
"\n"
"%s"
msgstr ""
"リポジトリからパッケージのメタデータを読み込めません。repodataディレクトリが"
"ないことが原因の可能性があります。リポジトリが正しく生成されているか確認して"
"ください。\n"
"\n"
"%s"

#: ../iw/task_gui.py:54
#, python-format
msgid ""
"Unable to find a group file for %s.  This will prevent manual selection of "
"packages from the repository from working"
msgstr ""
"%s のグループファイルが見つかりません。 このためリポジトリから手作業による"
"パッケージ選択ができません。"

#: ../iw/task_gui.py:80
msgid "Edit Repository"
msgstr "リポジトリの編集"

#: ../iw/task_gui.py:155
msgid "Invalid Repository Name"
msgstr "無効なリポジトリ名です"

#: ../iw/task_gui.py:156
msgid "You must provide a repository name."
msgstr "リポジトリ名を入力しなければなりません。"

#: ../iw/task_gui.py:167
msgid "Invalid Proxy URL"
msgstr "無効なプロキシ URL です。"

#: ../iw/task_gui.py:168
msgid "You must provide an HTTP, HTTPS, or FTP URL to a proxy."
msgstr "プロキシには、HTTP, HTTPS, または FTP のURLを入力する必要があります。"

#: ../iw/task_gui.py:187
msgid "Invalid Repository URL"
msgstr "無効なリポジトリ URL です。"

#: ../iw/task_gui.py:188
msgid "You must provide an HTTP, HTTPS, or FTP URL to a repository."
msgstr ""
"リポジトリには、HTTP, HTTPS, または FTP の URL を入力する必要があります。"

#: ../iw/task_gui.py:232
#, python-format
msgid ""
"The repository %s has already been added.  Please choose a different "
"repository name and URL."
msgstr ""
"リポジトリ %s はすでに追加されています。別のリポジトリ名と URL を選択してくだ"
"さい。"

#: ../iw/timezone_gui.py:62 ../textw/timezone_text.py:95
msgid "Time Zone Selection"
msgstr "タイムゾーンの選択"

#: ../iw/upgrade_bootloader_gui.py:33 ../textw/upgrade_bootloader_text.py:132
msgid "Upgrade Boot Loader Configuration"
msgstr "ブートローダー設定のアップグレード"

#: ../iw/upgrade_bootloader_gui.py:119
msgid "_Update boot loader configuration"
msgstr "ブートローダー設定の更新(_U)"

#: ../iw/upgrade_bootloader_gui.py:120
msgid "This will update your current boot loader."
msgstr "これで現在のブートローダーを更新します。"

#: ../iw/upgrade_bootloader_gui.py:124 ../textw/upgrade_bootloader_text.py:104
msgid ""
"Due to system changes, your boot loader configuration can not be "
"automatically updated."
msgstr "システムが変わったので、ブートローダー設定は自動的には更新されません。"

#: ../iw/upgrade_bootloader_gui.py:127 ../textw/upgrade_bootloader_text.py:108
msgid ""
"The installer is unable to detect the boot loader currently in use on your "
"system."
msgstr "インストーラーは現在システムで使用中のブートローダーを検出できません。"

#: ../iw/upgrade_bootloader_gui.py:134 ../textw/upgrade_bootloader_text.py:117
#, python-format
msgid ""
"The installer has detected the %s boot loader currently installed on %s."
msgstr ""
"インストーラーが現在 %s にインストールされている %s ブートローダーを検出しま"
"した。"

#: ../iw/upgrade_bootloader_gui.py:138
msgid "This is the recommended option."
msgstr "これは推奨されるオプションです。"

#: ../iw/upgrade_bootloader_gui.py:143
msgid "_Create new boot loader configuration"
msgstr "新しいブートローダー設定を作成(_C)"

#: ../iw/upgrade_bootloader_gui.py:145
msgid ""
"This option creates a new boot loader configuration.  If you wish to switch "
"boot loaders, you should choose this."
msgstr ""
"このオプションは新しいブートローダー設定を作成します。ブートローダーを変更し"
"たい場合は、これを選択してください。"

#: ../iw/upgrade_bootloader_gui.py:152
msgid "_Skip boot loader updating"
msgstr "ブートローダーの更新をスキップ(_S)"

#: ../iw/upgrade_bootloader_gui.py:153
msgid ""
"This option makes no changes to boot loader configuration.  If you are using "
"a third party boot loader, you should choose this."
msgstr ""
"このオプションはブートローダー設定をまったく変更しません。他社製ブートロー"
"ダーを使用している場合は、これを選択してください。"

#: ../iw/upgrade_bootloader_gui.py:164
msgid "What would you like to do?"
msgstr "どうしますか?"

#: ../iw/upgrade_migratefs_gui.py:32 ../textw/upgrade_text.py:37
msgid "Migrate File Systems"
msgstr "ファイルシステムの転換"

#: ../iw/upgrade_migratefs_gui.py:62 ../textw/upgrade_text.py:39
#, python-format
msgid ""
"This release of %s supports the an updated file system, which has several "
"benefits over the file system traditionally shipped in %s.  This "
"installation program can migrate formatted partitions without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""
"%s のこのリリースでは最新のファイルシステムをサポートしています。これらは従"
"来 %s に使われていたファイルシステムに比べ優れた点をいくつか備えています。こ"
"のインストールプログラムは、データを失うことなくファイルシステムを更新するこ"
"とができます。\n"
"\n"
"どのパーティションを転換しますか?"

#: ../iw/upgrade_swap_gui.py:33
msgid "Upgrade Swap Partition"
msgstr "swap パーティションのアップグレード"

#: ../iw/upgrade_swap_gui.py:93 ../textw/upgrade_text.py:103
#, python-format
msgid ""
"Recent kernels (2.4 or newer) need significantly more swap than older "
"kernels, up to twice the amount of RAM on the system.  You currently have %"
"dMB of swap configured, but you may create additional swap space on one of "
"your file systems now."
msgstr ""
"最近のカーネル(2.4以降)では旧来のカーネルに比べ、膨大な swap 領域を必要とし、"
"最大でシステムの RAM の 2 倍が必要です。現在、 %dMB の swap 領域を確保してい"
"ますが、ここでファイルシステムのいずれかに追加の swap 領域を作成することがで"
"きます。"

#: ../iw/upgrade_swap_gui.py:100
#, python-format
msgid ""
"\n"
"\n"
"The installer has detected %s MB of RAM.\n"
msgstr ""
"\n"
"\n"
"インストーラーは %s MB の RAM を検出しました。\n"

#: ../iw/upgrade_swap_gui.py:112
msgid "I _want to create a swap file"
msgstr "swap ファイルを作成します(_W)"

#: ../iw/upgrade_swap_gui.py:121
msgid "Select the _partition to put the swap file on:"
msgstr "swap ファイルを配置するパーティションを選択(_P):"

#: ../iw/upgrade_swap_gui.py:140 ../textw/upgrade_text.py:121
msgid "Partition"
msgstr "パーティション"

#: ../iw/upgrade_swap_gui.py:140
msgid "Free Space (MB)"
msgstr "空き領域 (MB)"

#: ../iw/upgrade_swap_gui.py:158
#, python-format
msgid ""
"A minimum swap file size of %d MB is recommended.  Please enter a size for "
"the swap file:"
msgstr ""
"最低でも %d MB の swap ファイルにすることを推奨します。swap ファイルの容量を"
"入力してください。"

#: ../iw/upgrade_swap_gui.py:173
msgid "Swap file _size (MB):"
msgstr " swap ファイルの容量 (MB)(_S):"

#: ../iw/upgrade_swap_gui.py:183
msgid "I _don't want to create a swap file"
msgstr "swap ファイルを作成しません(_D)"

#: ../iw/upgrade_swap_gui.py:193
msgid ""
"A swap file is strongly recommended. Failure to create one could cause the "
"installer to abort abnormally.  Are you sure you wish to continue?"
msgstr ""
"swap ファイルの作成を強く推奨します。 swap ファイルを作成しないと、インストー"
"ラーが異常終了する原因となる場合があります。本当に続行しますか?"

#: ../iw/upgrade_swap_gui.py:201 ../textw/upgrade_text.py:187
msgid "The swap file must be between 1 and 2000 MB in size."
msgstr "swap ファイルは 1 MB から 2000 MB の間の容量にしてください。"

#: ../iw/upgrade_swap_gui.py:208 ../textw/upgrade_text.py:182
msgid ""
"There is not enough space on the device you selected for the swap partition."
msgstr "swap パーティション用に選択したデバイスには十分な領域がありません。"

#: ../iw/zipl_gui.py:35
msgid "z/IPL Boot Loader Configuration"
msgstr "z/IPL ブートローダーの設定"

#: ../iw/zipl_gui.py:59
msgid "The z/IPL boot loader will be installed on your system."
msgstr "z/IPL ブートローダーがシステムにインストールされます。"

#: ../iw/zipl_gui.py:61
msgid ""
"The z/IPL Boot Loader will now be installed on your system.\n"
"\n"
"The root partition will be the one you selected previously in the partition "
"setup.\n"
"\n"
"The kernel used to start the machine will be the one to be installed by "
"default.\n"
"\n"
"If you wish to make changes later after the installation feel free to change "
"the /etc/zipl.conf configuration file.\n"
"\n"
"You can now enter any additional kernel parameters which your machine or "
"your setup may require."
msgstr ""
"z/IPL ブートローダーがシステムにインストールされます。\n"
"\n"
"root パーティションはパーティションの設定で既に選択したパーティションになりま"
"す。\n"
"\n"
"マシンの起動に使用されるカーネルはデフォルトでインストールされるカーネルにな"
"ります。\n"
"\n"
"インストール後に変更を加える場合は、/etc/zipl.conf の設定ファイルを自由に変更"
"できます。\n"
"\n"
"ここでご使用のマシンまたは設定が必要とする追加のカーネルパラメーターを入力す"
"ることができます。"

#: ../iw/zipl_gui.py:88 ../textw/zipl_text.py:68
msgid "Kernel Parameters"
msgstr "カーネルパラメーター"

#: ../iw/zipl_gui.py:91 ../iw/zipl_gui.py:94
msgid "Chandev Parameters"
msgstr "Chandev パラメーター"

#: ../textw/bootloader_text.py:32
msgid "Which boot loader would you like to use?"
msgstr "使用するブートローダーを選択してください。"

#: ../textw/bootloader_text.py:42
msgid "Use GRUB Boot Loader"
msgstr "GRUB ブートローダーを使用する"

#: ../textw/bootloader_text.py:43
msgid "No Boot Loader"
msgstr "ブートローダーを使用しない"

#: ../textw/bootloader_text.py:63
msgid "Skip Boot Loader"
msgstr "ブートローダー(設定)をスキップ"

#: ../textw/bootloader_text.py:64
msgid ""
"You have elected not to install any boot loader, which is not recommended "
"unless you have an advanced need. Booting your system into Linux directly "
"from the hard drive almost always requires a boot loader.\n"
"\n"
"Are you sure you want to skip boot loader installation?"
msgstr ""
"いずれのブートローダーもインストールしないよう選択していますが、特段の必要性"
"がない限り、推奨できません。ハードドライブからシステムを Linux で直接起動する"
"には、ほとんどの場合においてブートローダーが必要となります。\n"
"\n"
"本当にブートローダーのインストールをスキップしますか?"

#: ../textw/bootloader_text.py:96
msgid ""
"A few systems need to pass special options to the kernel at boot time to "
"function properly. If you need to pass boot options to the kernel, enter "
"them now. If you don't need any or aren't sure, leave this blank."
msgstr ""
"システムが正しく機能するには起動時に特殊なオプションをカーネルに対して渡す必"
"要がある場合があります。カーネルにブートオプションを渡す必要がある場合は、こ"
"こでそのオプションを入力してください。オプションが必要ない、またはよくわから"
"ない場合は空白のままにしておいてください。"

#: ../textw/bootloader_text.py:105
msgid "Force use of LBA32 (not normally required)"
msgstr "LBA32 の強制使用(通常は必要なし)"

#: ../textw/bootloader_text.py:128
msgid ""
"If LBA32 is not supported by your system's BIOS, forcing its use can prevent "
"your machine from booting.\n"
"\n"
"Would you like to continue and force LBA32 mode?"
msgstr ""
"システムの BIOS が LBA32 をサポートしていない場合、強制的に使用するとマシンが"
"ブートできなくなる恐れがあります。\n"
"\n"
"続行して LBA32 モードを強制しますか?"

#: ../textw/bootloader_text.py:169
msgid "Where do you want to install the boot loader?"
msgstr "ブートローダーのインストール場所を指定してください。"

#: ../textw/bootloader_text.py:197 ../textw/bootloader_text.py:262
msgid "Boot label"
msgstr "ブートラベル"

#: ../textw/bootloader_text.py:201
msgid "Clear"
msgstr "消去"

#: ../textw/bootloader_text.py:209
msgid "Edit Boot Label"
msgstr "ブートラベルの編集"

#: ../textw/bootloader_text.py:227 ../textw/bootloader_text.py:232
msgid "Invalid Boot Label"
msgstr "無効なブートラベル"

#: ../textw/bootloader_text.py:228
msgid "Boot label may not be empty."
msgstr "ブートラベルは空白にできません。"

#: ../textw/bootloader_text.py:233
msgid "Boot label contains illegal characters."
msgstr "ブートラベルに不適切な文字が含まれています。"

#: ../textw/bootloader_text.py:281
#, python-format
msgid ""
"The boot manager %s uses can boot other operating systems as well.  Please "
"tell me what partitions you would like to be able to boot and what label you "
"want to use for each of them."
msgstr ""
"%s が使用するブートマネージャーは他のオペレーティングシステムの起動にも使用す"
"ることができます。ブート可能にするパーティションとそのパーティション用に使用"
"するブートラベルを設定してください。"

#: ../textw/bootloader_text.py:294
msgid ""
" <Space> select | <F2> select default | <F4> delete | <F12> next screen>"
msgstr "<Space> 選択 | <F2> デフォルト選択 | <F4> 削除 | <F12> 次の画面"

#: ../textw/bootloader_text.py:390
msgid ""
"A boot loader password prevents users from passing arbitrary options to the "
"kernel.  For highest security, you should set a password, but a password is "
"not necessary for more casual users."
msgstr ""
"ブートローダーのパスワードはユーザーが任意のオプションをカーネルに渡してしま"
"うのを防ぎます。安全性を最大限に高めるにはパスワードの設定を推奨しますが、こ"
"れを必要としない一般的なユーザーには必ずしも必要ではありません。"

#: ../textw/bootloader_text.py:400
msgid "Use a GRUB Password"
msgstr "GRUB パスワードを使用する"

#: ../textw/bootloader_text.py:412
msgid "Boot Loader Password:"
msgstr "ブートローダーのパスワード:"

#: ../textw/bootloader_text.py:413 tmp/account.glade.h:1
msgid "Confirm:"
msgstr "確認:"

#: ../textw/bootloader_text.py:442
msgid "Passwords Do Not Match"
msgstr "パスワードが一致していません"

#: ../textw/bootloader_text.py:447
msgid "Password Too Short"
msgstr "パスワードが短かすぎます"

#: ../textw/bootloader_text.py:448
msgid "Boot loader password is too short"
msgstr "ブートローダーのパスワードが短かすぎます"

#: ../textw/complete_text.py:31
msgid ""
"Press <Enter> to end the installation process.\n"
"\n"
msgstr ""
"<Enter> を押してインストールを終了します。\n"
"\n"

#: ../textw/complete_text.py:32
msgid "<Enter> to exit"
msgstr "<Enter> を押して終了します"

#: ../textw/complete_text.py:36
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
"%s%s"
msgstr ""
"おめでとうございます。 %s のインストールは完了しました。\n"
"\n"
"%s%s"

#: ../textw/complete_text.py:39
#, python-format
msgid ""
"For information on errata (updates and bug fixes), visit http://www.redhat."
"com/errata/.\n"
"\n"
"Information on using your system is available in the %s manuals at http://"
"www.redhat.com/docs/."
msgstr ""
"エラータ(更新やバグ修正)に関する詳細は、 http://www.redhat.com/errata/ をご覧"
"ください。\n"
"\n"
"システムの使用方法については、 http://www.redhat.com/docs/ にある %s マニュア"
"ルを参照してください。"

#: ../textw/complete_text.py:45
msgid "Complete"
msgstr "完了"

#: ../textw/complete_text.py:46
msgid "Reboot"
msgstr "再起動する"

#: ../textw/grpselect_text.py:94
msgid "Please select the package groups you would like to install."
msgstr "インストールしたいパッケージグループを選択してください。"

#: ../textw/grpselect_text.py:112
msgid ""
"<Space>,<+>,<-> selection   |   <F2> Group Details   |   <F12> next screen"
msgstr " <Space>,<+>,<-> 選択   |   <F2> グループの詳細   |   <F12> 次の画面"

#: ../textw/grpselect_text.py:124
msgid "No optional packages to select"
msgstr "選択するオプションのパッケージがありません"

#: ../textw/grpselect_text.py:146
msgid "Package Group Details"
msgstr "パッケージグループの詳細"

#: ../textw/keyboard_text.py:46
msgid "Keyboard Selection"
msgstr "キーボードの選択"

#: ../textw/keyboard_text.py:47
msgid "Which model keyboard is attached to this computer?"
msgstr "このコンピューターに接続されるキーボードのモデルを指定してください。"

#: ../textw/network_text.py:58
#, python-format
msgid ""
"You have not specified the field %s.  Depending on your network environment "
"this may cause problems later."
msgstr ""
"フィールド %s を指定していません。ネットワーク環境によっては後で問題となる可"
"能性があります。"

#: ../textw/network_text.py:75
#, python-format
msgid "IPv%d prefix must be between 0 and %d."
msgstr "IPv%d のプレフィックスは 0 から %d の間にしてください。"

#: ../textw/network_text.py:79
msgid "Integer Required for Prefix"
msgstr "プレフィックスには数字を指定してください"

#: ../textw/network_text.py:80
#, python-format
msgid ""
"You must enter a valid integer for the %s.  For IPv4, the value can be "
"between 0 and 32.  For IPv6 it can be between 0 and 128."
msgstr ""
"%s には正しい数字を指定してください。IPv4 では 0 から 32、IPv6 では 0 から "
"128 です。"

#: ../textw/network_text.py:87 tmp/netpostconfig.glade.h:16
msgid "Prefix (Netmask)"
msgstr "プレフィックス (ネットマスク)"

#: ../textw/network_text.py:89 tmp/netpostconfig.glade.h:15
msgid "Prefix"
msgstr "プレフィックス"

#: ../textw/network_text.py:142
msgid "Activate on boot"
msgstr "起動時にアクティブにする"

#: ../textw/network_text.py:149 ../loader2/net.c:730
#: tmp/netpostconfig.glade.h:9
msgid "Enable IPv4 support"
msgstr "IPv4 サポートを有効にする"

#: ../textw/network_text.py:156 ../loader2/net.c:743
#: tmp/netpostconfig.glade.h:10
msgid "Enable IPv6 support"
msgstr "IPv6 サポートを有効にする"

#: ../textw/network_text.py:169
msgid "P-to-P:"
msgstr "P-to-P:"

#: ../textw/network_text.py:187 tmp/netpostconfig.glade.h:7
msgid "ESSID:"
msgstr "ESSID:"

#: ../textw/network_text.py:196
msgid "WEP Key:"
msgstr "WEP キー:"

#: ../textw/network_text.py:209
#, python-format
msgid "Network Configuration for %s"
msgstr "%s 用のネットワーク設定"

#: ../textw/network_text.py:246 ../textw/network_text.py:249
msgid "point-to-point IP address"
msgstr "point-to-point IP アドレス"

#: ../textw/network_text.py:272 tmp/netpostconfig.glade.h:5
msgid "Dynamic IP configuration (DHCP)"
msgstr "動的 IP 設定 (DHCP)"

#: ../textw/network_text.py:275 ../textw/network_text.py:406
msgid "Manual address configuration"
msgstr "手動によるアドレス設定"

#: ../textw/network_text.py:295
#, python-format
msgid "IPv4 Configuration for %s"
msgstr "%s 用の IPv4 設定"

#: ../textw/network_text.py:325 ../textw/network_text.py:337
#: ../textw/network_text.py:340
msgid "IPv4 address"
msgstr "IPv4 アドレス"

#: ../textw/network_text.py:329
msgid "IPv4 network mask"
msgstr "IPv4 ネットマスク"

#: ../textw/network_text.py:355 ../textw/network_text.py:358
#: ../textw/network_text.py:361
msgid "IPv4 prefix (network mask)"
msgstr "IPv4 プレフィックス (ネットマスク)"

#: ../textw/network_text.py:400 tmp/netpostconfig.glade.h:4
msgid "Automatic neighbor discovery"
msgstr "隣接者の自動探索"

#: ../textw/network_text.py:403 tmp/netpostconfig.glade.h:6
msgid "Dynamic IP configuration (DHCPv6)"
msgstr "動的 IP 設定 (DHCPv6)"

#: ../textw/network_text.py:430
#, python-format
msgid "IPv6 Configuration for %s"
msgstr "%s 用のIPv6 設定"

#: ../textw/network_text.py:460 ../textw/network_text.py:471
#: ../textw/network_text.py:474
msgid "IPv6 address"
msgstr "IPv6 アドレス"

#: ../textw/network_text.py:464 ../textw/network_text.py:483
msgid "IPv6 prefix"
msgstr "IPv6 プレフィックス"

#: ../textw/network_text.py:509
msgid "Configure Network Interface"
msgstr "ネットワークインターフェースを設定する"

#: ../textw/network_text.py:510
#, python-format
msgid "Would you like to configure the %s network interface in your system?"
msgstr "システムの %s ネットワークインターフェースを設定しますか?"

#: ../textw/network_text.py:524 ../textw/network_text.py:526
msgid "UNCONFIGURED"
msgstr "未設定"

#: ../textw/network_text.py:535
msgid ""
"The current configuration settings for each interface are listed next to the "
"device name.  Unconfigured interfaces are shown as UNCONFIGURED.  To "
"configure an interface, highlight it and choose Edit.  When you are "
"finished, press OK to continue."
msgstr ""
"現在の個々のインターフェースの設定はデバイス名の横に表示されます。未設定のイ"
"ンターフェースは未設定と表示されます。あるインターフェースを設定するにはそれ"
"を選択し、編集を選びます。完了したら OK を押して続けます。"

#: ../textw/network_text.py:567
msgid "Active on boot"
msgstr "起動時にアクティブにする"

#: ../textw/network_text.py:569
msgid "Inactive on boot"
msgstr "起動時にアクティブにしない"

#: ../textw/network_text.py:572
msgid "DHCP"
msgstr "DHCP"

#: ../textw/network_text.py:577
msgid "Auto IPv6"
msgstr "自動 IPv6"

#: ../textw/network_text.py:579
msgid "DHCPv6"
msgstr "DHCPv6"

#: ../textw/network_text.py:678 ../loader2/net.c:1055
msgid "Gateway:"
msgstr "ゲートウェイ:"

#: ../textw/network_text.py:687
msgid "Primary DNS:"
msgstr "1 番目の DNS:"

#: ../textw/network_text.py:692
msgid "Secondary DNS:"
msgstr "2 番目の DNS:"

#: ../textw/network_text.py:699
msgid "Miscellaneous Network Settings"
msgstr "その他のネットワーク設定"

#: ../textw/network_text.py:716 ../textw/network_text.py:719
msgid "gateway"
msgstr "ゲートウェイ"

#: ../textw/network_text.py:726 ../textw/network_text.py:729
msgid "primary DNS"
msgstr "1 番目の DNS"

#: ../textw/network_text.py:738
msgid "secondary DNS"
msgstr "2 番目の DNS"

#: ../textw/network_text.py:762
msgid "automatically via DHCP"
msgstr "DHCP 経由で自動設定"

#: ../textw/network_text.py:765
msgid "manually"
msgstr "手動設定"

#: ../textw/network_text.py:784
msgid "Hostname Configuration"
msgstr "ホスト名の設定"

#: ../textw/network_text.py:787
msgid ""
"If your system is part of a larger network where hostnames are assigned by "
"DHCP, select automatically via DHCP. Otherwise, select manually and enter a "
"hostname for your system. If you do not, your system will be known as "
"'localhost.'"
msgstr ""
"システムが大きなネットワークの一部になっていて、ホスト名が DHCP で割り当てら"
"れる場合、 「DHCP 経由で自動設定」を選択します。これ以外は、「手動設定」を選"
"択してシステムのホスト名を入力します。ホスト名を入力しなかった場合、"
"「localhost」になります。"

#: ../textw/network_text.py:813 ../textw/network_text.py:819
msgid "Invalid Hostname"
msgstr "無効なホスト名です"

#: ../textw/network_text.py:814
msgid "You have not specified a hostname."
msgstr "ホスト名を指定していません。"

#: ../textw/partition_text.py:50
msgid "Must specify a value"
msgstr "値を指定しなければなりません"

#: ../textw/partition_text.py:53
msgid "Requested value is not an integer"
msgstr "要求された値は整数ではありません"

#: ../textw/partition_text.py:55
msgid "Requested value is too large"
msgstr "要求された値は大きすぎます"

#: ../textw/partition_text.py:122
#, python-format
msgid "RAID Device %s"
msgstr "RAID デバイス %s"

#: ../textw/partition_text.py:250
#, python-format
msgid "Warning: %s"
msgstr "警告: %s "

#: ../textw/partition_text.py:251
msgid "Modify Partition"
msgstr "パーティションの編集"

#: ../textw/partition_text.py:251
msgid "Add anyway"
msgstr "とにかく追加する"

#: ../textw/partition_text.py:289
msgid "Mount Point:"
msgstr "マウントポイント:"

#: ../textw/partition_text.py:341
msgid "File System type:"
msgstr "ファイルシステムのタイプ:"

#: ../textw/partition_text.py:375
msgid "Allowable Drives:"
msgstr "選択可能なドライブ:"

#: ../textw/partition_text.py:431
msgid "Fixed Size:"
msgstr "固定容量:"

#: ../textw/partition_text.py:433
msgid "Fill maximum size of (MB):"
msgstr "最大容量 (MB):"

#: ../textw/partition_text.py:437
msgid "Fill all available space:"
msgstr "使用できる領域をすべて:"

#: ../textw/partition_text.py:460
msgid "Start Cylinder:"
msgstr "開始シリンダー:"

#: ../textw/partition_text.py:473
msgid "End Cylinder:"
msgstr "終了シリンダー:"

#: ../textw/partition_text.py:496
msgid "Volume Group:"
msgstr "ボリュームグループ:"

#: ../textw/partition_text.py:518
msgid "RAID Level:"
msgstr "RAID レベル:"

#: ../textw/partition_text.py:536
msgid "RAID Members:"
msgstr "RAID メンバー:"

#: ../textw/partition_text.py:555
msgid "Number of spares?"
msgstr "使用するスペア数を入力してください。"

#: ../textw/partition_text.py:569
msgid "File System Type:"
msgstr "ファイルシステムのタイプ:"

#: ../textw/partition_text.py:582
msgid "File System Label:"
msgstr "ファイルシステムのラベル:"

#: ../textw/partition_text.py:593
msgid "File System Option:"
msgstr "ファイルシステムのオプション:"

#: ../textw/partition_text.py:596 ../textw/partition_text.py:803
#: ../textw/partition_text.py:1028 ../textw/partition_text.py:1198
#, python-format
msgid "Format as %s"
msgstr "%s としてフォーマット"

#: ../textw/partition_text.py:598 ../textw/partition_text.py:805
#: ../textw/partition_text.py:1030 ../textw/partition_text.py:1200
#, python-format
msgid "Migrate to %s"
msgstr "%s に転換"

#: ../textw/partition_text.py:600 ../textw/partition_text.py:807
#: ../textw/partition_text.py:1032 ../textw/partition_text.py:1202
msgid "Leave unchanged"
msgstr "フォーマットしない"

#: ../textw/partition_text.py:613 ../textw/partition_text.py:783
#: ../textw/partition_text.py:1008 ../textw/partition_text.py:1178
msgid "File System Options"
msgstr "ファイルシステムのオプション"

#: ../textw/partition_text.py:616
msgid ""
"Please choose how you would like to prepare the file system on this "
"partition."
msgstr ""
"このパーティションのファイルシステムをどのようにするか選択してください。"

#: ../textw/partition_text.py:623
msgid "Leave unchanged (preserve data)"
msgstr "フォーマットしない(データを保存)"

#: ../textw/partition_text.py:632
msgid "Format as:"
msgstr "フォーマットする:"

#: ../textw/partition_text.py:652
msgid "Migrate to:"
msgstr "転換先:"

#: ../textw/partition_text.py:753
msgid "Force to be a primary partition"
msgstr "基本パーティションとして強制指定"

#: ../textw/partition_text.py:762
msgid "Not Supported"
msgstr "サポートされていません"

#: ../textw/partition_text.py:763
msgid "You can only edit LVM Volume Groups in the graphical installer."
msgstr ""
"LVM ボリュームグループはグラフィカルインストーラーでしか編集できません。"

#: ../textw/partition_text.py:830 ../textw/partition_text.py:878
msgid "Invalid Entry for Partition Size"
msgstr "パーティション容量には無効な値です"

#: ../textw/partition_text.py:842
msgid "Invalid Entry for Maximum Size"
msgstr "最大容量には無効な値です"

#: ../textw/partition_text.py:861
msgid "Invalid Entry for Starting Cylinder"
msgstr "開始シリンダーには無効な値です"

#: ../textw/partition_text.py:870
msgid "Invalid Entry for End Cylinder"
msgstr "終了シリンダーには無効な値です"

#: ../textw/partition_text.py:981
msgid "No RAID partitions"
msgstr "RAID パーティションがありません"

#: ../textw/partition_text.py:982
msgid "At least two software RAID partitions are needed."
msgstr "少なくとも 2 つのソフトウェア RAID パーティションが必要です。"

#: ../textw/partition_text.py:994 ../textw/partition_text.py:1165
msgid "Format partition?"
msgstr "パーティションをフォーマットしますか?"

#: ../textw/partition_text.py:1056
msgid "Invalid Entry for RAID Spares"
msgstr "RAID スペアには無効な値です"

#: ../textw/partition_text.py:1071
msgid "Too many spares"
msgstr "スペア数が多過ぎます"

#: ../textw/partition_text.py:1072
msgid "You may not use any spares with a RAID0 array."
msgstr "RAID0 アレイではスペアは使えません。"

#: ../textw/partition_text.py:1151
msgid "No Volume Groups"
msgstr "ボリュームグループがありません"

#: ../textw/partition_text.py:1152
msgid "No volume groups exist in which to create a logical volume"
msgstr "論理ボリュームを作成するためのボリュームグループがありません"

#: ../textw/partition_text.py:1276
#, python-format
msgid ""
"The current requested size (%10.2f MB) is larger than the maximum logical "
"volume size (%10.2f MB). "
msgstr ""
"現在の要求サイズ(%10.2f MB)は、最大論理ボリュームサイズ(%10.2f MB)を超えてい"
"ます。"

#: ../textw/partition_text.py:1295
#, python-format
msgid ""
"The current requested size (%10.2f MB) is larger than the available size in "
"the volume group (%10.2f MB)."
msgstr ""
"現在の要求サイズ(%10.2f MB)は、ボリュームグループ内で使用できるサイズ(%10.2f "
"MB)を超えています。"

#: ../textw/partition_text.py:1348
msgid "New Partition or Logical Volume?"
msgstr "新規のパーティションにしますか、新規の論理ボリュームにしますか?"

#: ../textw/partition_text.py:1349
msgid "Would you like to create a new partition or a new logical volume?"
msgstr ""
"新規のパーティションを作成しますか、新規の論理ボリュームを作成しますか?"

#: ../textw/partition_text.py:1351
msgid "partition"
msgstr "パーティション"

#: ../textw/partition_text.py:1351
msgid "logical volume"
msgstr "論理ボリューム"

#: ../textw/partition_text.py:1423
msgid "Size"
msgstr "容量"

#: ../textw/partition_text.py:1427
msgid "New"
msgstr "新規"

#: ../textw/partition_text.py:1429
msgid "Delete"
msgstr "削除"

#: ../textw/partition_text.py:1430
msgid "RAID"
msgstr "RAID"

#: ../textw/partition_text.py:1433
msgid ""
"    F1-Help     F2-New      F3-Edit   F4-Delete    F5-Reset    F12-OK        "
msgstr ""
"  <F1>-ヘルプ   <F2>-新規    <F3>-編集  <F4>-削除    <F5>-リセット  <F12>-"
"OK    "

#: ../textw/partition_text.py:1465
msgid "No Root Partition"
msgstr "ルートパーティションがありません"

#: ../textw/partition_text.py:1466
msgid "Installation requires a / partition."
msgstr "インストールには / パーティションが必要です。"

#: ../textw/partition_text.py:1508
msgid "Partitioning Type"
msgstr "パーティションタイプ"

#: ../textw/partition_text.py:1510
msgid ""
"Installation requires partitioning of your hard drive.  The default layout "
"is reasonable for most users.  You can either choose to use this or create "
"your own."
msgstr ""
"インストールには、ハードドライブのパーティション設定が必要です。デフォルトで"
"は、ほとんどのユーザーに適しているパーティション設定のレイアウトが選択されま"
"す。デフォルトのレイアウトを選択するか、独自のレイアウトを作成します。"

#: ../textw/partition_text.py:1534
msgid "Which drive(s) do you want to use for this installation?"
msgstr "このインストールに使用するドライブを指定してください。"

#: ../textw/partition_text.py:1549
msgid "<Space>,<+>,<-> selection   |   <F2> Add drive   |   <F12> next screen"
msgstr "<Space>,<+>,<-> 選択   |   <F2> ドライブの追加   |   <F12> 次の画面"

#: ../textw/partition_text.py:1616
msgid "Review Partition Layout"
msgstr "パーティションレイアウトの確認"

#: ../textw/partition_text.py:1617
msgid "Review and modify partitioning layout?"
msgstr "パーティション設定のレイアウトを確認、変更を行いますか?"

#: ../textw/partition_text.py:1638 tmp/adddrive.glade.h:3
msgid "Advanced Storage Options"
msgstr "高度なストレージオプション"

#: ../textw/partition_text.py:1639 tmp/adddrive.glade.h:5
msgid "How would you like to modify your drive configuration?"
msgstr "ドライブの構成をどのように変更しますか?"

#: ../textw/partition_text.py:1662
msgid "Add FCP Device"
msgstr "FCP デバイスの追加"

#: ../textw/partition_text.py:1663 tmp/zfcp-config.glade.h:5
msgid ""
"zSeries machines can access industry-standard SCSI devices via Fibre Channel "
"(FCP). You need to provide a 16 bit device number, a 64 bit World Wide Port "
"Name (WWPN), and a 64 bit FCP LUN for each device."
msgstr ""
"zSeries マシンはファイバーチャネル (FCP) 経由で業界基準の SCSI デバイスにアク"
"セスすることができます。それぞれのデバイスに 16 ビットのデバイス番号、64 ビッ"
"トの World Wide Port Name (WWPN) および 64 ビットの FCP LUN を指定する必要が"
"あります。"

#: ../textw/partition_text.py:1686 tmp/iscsi-config.glade.h:5
msgid "Configure iSCSI Parameters"
msgstr "iSCSI パラメーターの設定"

#: ../textw/partition_text.py:1687 tmp/iscsi-config.glade.h:6
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""
"iSCSI ディスクを使用するには、ホスト用に設定している iSCSI イニシエーター名及"
"びiSCSI ターゲットのアドレスを入力してください。"

#: ../textw/partition_text.py:1688
msgid "Target IP Address"
msgstr "ターゲット IP アドレス"

#: ../textw/partition_text.py:1689
msgid "iSCSI Initiator Name"
msgstr "iSCSI イニシエーター名"

#: ../textw/partmethod_text.py:32
msgid "Autopartition"
msgstr "自動パーティション設定"

#: ../textw/partmethod_text.py:33
msgid "Disk Druid"
msgstr "Disk Druid"

#: ../textw/progress_text.py:59
msgid "Package Installation"
msgstr "パッケージのインストール"

#: ../textw/task_text.py:51
msgid "Package selection"
msgstr "パッケージの選択"

#: ../textw/task_text.py:57 tmp/tasksel.glade.h:4
#, no-c-format, python-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage.  What additional tasks would you like your system to "
"support?"
msgstr ""
"%s のデフォルトインストールには一般的なインターネット使用に適用できるソフト"
"ウェアのセットが含まれています。システムにサポートさせる追加タスクを指定して"
"ください。"

#: ../textw/task_text.py:72
msgid "Customize software selection"
msgstr "ソフトウェア選択のカスタマイズ"

#: ../textw/timezone_text.py:74
msgid "In which time zone are you located?"
msgstr "どのタイムゾーンに位置しますか?"

#: ../textw/timezone_text.py:92
msgid "System clock uses UTC"
msgstr "システムクロックに UTC を使用"

#: ../textw/upgrade_bootloader_text.py:112
#: ../textw/upgrade_bootloader_text.py:121
msgid "Update boot loader configuration"
msgstr "ブートローダー設定の更新"

#: ../textw/upgrade_bootloader_text.py:124
msgid "Skip boot loader updating"
msgstr "ブートローダー更新のスキップ"

#: ../textw/upgrade_bootloader_text.py:126
msgid "Create new boot loader configuration"
msgstr "新しいブートローダー設定の作成"

#: ../textw/upgrade_text.py:121
msgid "Free Space"
msgstr "空き領域"

#: ../textw/upgrade_text.py:136
msgid "RAM detected (MB):"
msgstr "検出された RAM 容量 (MB):"

#: ../textw/upgrade_text.py:139
msgid "Suggested size (MB):"
msgstr "推奨容量 (MB):"

#: ../textw/upgrade_text.py:142
msgid "Swap file size (MB):"
msgstr "swap ファイル容量 (MB):"

#: ../textw/upgrade_text.py:150
msgid "Add Swap"
msgstr "swap の追加"

#: ../textw/upgrade_text.py:175
msgid "The value you entered is not a valid number."
msgstr "入力した値は無効な数です。"

#: ../textw/upgrade_text.py:208
msgid "Reinstall System"
msgstr "システムの再インストール"

#: ../textw/upgrade_text.py:222
msgid "System to Upgrade"
msgstr "アップグレード対象システム"

#: ../textw/upgrade_text.py:223
msgid ""
"There seem to be one or more existing Linux installations on your system.\n"
"\n"
"Please choose one to upgrade, or select 'Reinstall System' to freshly "
"install your system."
msgstr ""
"システム上に 既存の Linux インストールがひとつ以上あるようです。\n"
"\n"
"アップグレードするにはひとつを選び、新規にシステムをインストールするには、"
"「システムの再インストール」を選んでください。"

#: ../textw/userauth_text.py:27
msgid "Root Password"
msgstr "root パスワード"

#: ../textw/userauth_text.py:30
msgid ""
"Pick a root password. You must type it twice to ensure you know it and do "
"not make a typing mistake. "
msgstr ""
"ルートパスワードを設定してください。パスワードを2回入力することで、パスワード"
"を覚えていて、タイプミスがないことを確認します。"

#: ../textw/userauth_text.py:64
msgid "The root password must be at least 6 characters long."
msgstr "root パスワードは 6 文字以上にしてください。"

#: ../textw/welcome_text.py:27
#, python-format
msgid "%s"
msgstr "%s"

#: ../textw/welcome_text.py:28
#, python-format
msgid ""
"Welcome to %s!\n"
"\n"
msgstr ""
"%s へようこそ!\n"
"\n"

#: ../textw/zipl_text.py:32
msgid ""
"The z/IPL Boot Loader will be installed on your system after installation is "
"complete. You can now enter any additional kernel and chandev parameters "
"which your machine or your setup require."
msgstr ""
"z/IPL ブートローダーはインストールが完了してからシステム上にインストールされ"
"ます。ここで、ご使用のマシンまたは設定が必要とする追加のカーネル及び chandev "
"パラメーターを入力することができます。"

#: ../textw/zipl_text.py:64
msgid "z/IPL Configuration"
msgstr "z/IPL の設定"

#: ../textw/zipl_text.py:72 ../textw/zipl_text.py:76
msgid "Chandev line "
msgstr "Chandev 行"

#: ../installclasses/fedora.py:34
msgid "_Fedora"
msgstr "Fedora(_F)"

#: ../installclasses/fedora.py:35 ../installclasses/rhel.py:42
#, python-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage. What additional tasks would you like your system to "
"include support for?"
msgstr ""
"%s のデフォルトインストールには一般的なインターネット使用に適用できるソフト"
"ウェアのセットが含まれています。システムにサポートさせる追加タスクを指定して"
"ください。"

#: ../installclasses/fedora.py:44
msgid "Office and Productivity"
msgstr "オフィスとプロダクティビティ"

#: ../installclasses/fedora.py:45 ../installclasses/rhel.py:54
#: ../installclasses/rhel.py:59
msgid "Software Development"
msgstr "ソフトウェア開発"

#: ../installclasses/fedora.py:46 ../installclasses/rhel.py:58
msgid "Web server"
msgstr "ウェブサーバー"

#: ../installclasses/rhel.py:41
msgid "Red Hat Enterprise Linux"
msgstr "Red Hat Enterprise Linux"

#: ../installclasses/rhel.py:51
msgid "Office"
msgstr "オフィス"

#: ../installclasses/rhel.py:52
msgid "Multimedia"
msgstr "マルチメディア"

#: ../installclasses/rhel.py:63
msgid "Virtualization"
msgstr "仮想化"

#: ../installclasses/rhel.py:64
msgid "Clustering"
msgstr "クラスタリング"

#: ../installclasses/rhel.py:65
msgid "Storage Clustering"
msgstr "ストレージのクラスタリング"

#: ../installclasses/rhel.py:69
msgid "Installation Number"
msgstr "インストール番号"

#: ../installclasses/rhel.py:70
msgid ""
"To install the full set of supported packages included in your subscription, "
"please enter your Installation Number"
msgstr ""
"申込分を含めたすべてのサポートパッケージをインストールするには、インストール"
"番号を入力してください。"

#: ../installclasses/rhel.py:73
msgid ""
"If you're unable to locate the Installation Number, consult http://www."
"redhat.com/apps/support/in.html.\n"
"\n"
"If you skip:\n"
"* You may not get access to the full set of packages included in your "
"subscription.\n"
"* It may result in an unsupported/uncertified installation of Red Hat "
"Enterprise Linux.\n"
"* You will not get software and security updates for packages not included "
"in your subscription."
msgstr ""
"インストール番号を見つけることができないときは、 http://www.redhat.com/apps/"
"support/in.html に相談してください。\n"
"\n"
"スキップしたら:\n"
"* 申込分を含めたすべてのパッケージにアクセスできないかもしれません。\n"
"* Red Hat Enterprise Linux の未サポート/未保証のインストールになりかねませ"
"ん。\n"
"* 申込分に含まれていないパッケージのソフトウェアやセキュリティ更新は受け取れ"
"ません。"

#: ../loader2/cdinstall.c:96 ../loader2/cdinstall.c:117
#: ../loader2/mediacheck.c:76
msgid "Media Check"
msgstr "メディアチェック"

#: ../loader2/cdinstall.c:96 ../loader2/cdinstall.c:99
#: ../loader2/cdinstall.c:117 ../loader2/cdinstall.c:125
#: ../loader2/method.c:393
msgid "Test"
msgstr "テスト"

#: ../loader2/cdinstall.c:96 ../loader2/cdinstall.c:100
msgid "Eject Disc"
msgstr "ディスク取り出し"

#: ../loader2/cdinstall.c:97
#, c-format
msgid ""
"Choose \"%s\" to test the disc currently in the drive, or \"%s\" to eject "
"the disc and insert another for testing."
msgstr ""
"現在ドライブに入っているディスクをテストするには \"%s\" を選び、 ディスクを取"
"り出して別のディスクを挿入してテストをするには。 \"%s\" を選びます。"

#: ../loader2/cdinstall.c:118
#, c-format
msgid ""
"If you would like to test additional media, insert the next disc and press "
"\"%s\". Testing each disc is not strictly required, however it is highly "
"recommended.  Minimally, the discs should be tested prior to using them for "
"the first time. After they have been successfully tested, it is not required "
"to retest each disc prior to using it again."
msgstr ""
"追加のメディアをテストする場合は、次のディスクを挿入してから \"%s\" を押して"
"ください。それぞれのディスクのテストは絶対に必要なわけではありませんが、強く"
"推奨します。最低でも、はじめて使用する前にはテストを行ってください。テストが"
"成功したら、再度使用する前に各ディスクをテストする必要はありません。"

#: ../loader2/cdinstall.c:142
#, c-format
msgid ""
"The %s disc was not found in any of your drives. Please insert the %s disc "
"and press %s to retry."
msgstr ""
"いずれのドライブにも %s ディスク は見つかりませんでした。%s ディスクを挿入し"
"てから、%s を押して再試行してください。"

#: ../loader2/cdinstall.c:200
msgid "Disc Found"
msgstr "ディスク が見つかりました"

#: ../loader2/cdinstall.c:201
#, c-format
msgid ""
"To begin testing the media before installation press %s.\n"
"\n"
"Choose %s to skip the media test and start the installation."
msgstr ""
"インストールの前にメディアのテストを開始するには %s を押します。\n"
"\n"
"メディアテストをスキップするには %s を選択して、インストールを開始します。"

#: ../loader2/cdinstall.c:339
#, c-format
msgid ""
"No %s disc was found which matches your boot media.  Please insert the %s "
"disc and press %s to retry."
msgstr ""
"ブートメディアに適合する %s ディスクが見つかりませんでした。%s ディスクを挿入"
"してから %s を 押して再試行してください。"

#: ../loader2/cdinstall.c:344
#, c-format
msgid ""
"The %s disc was not found in any of your CDROM drives. Please insert the %s "
"disc and press %s to retry."
msgstr ""
"いずれの CDROM ドライブにも %s ディスクは見つかりませんでした。%s ディスクを"
"挿入してから、%s を押して再試行してください。"

#: ../loader2/cdinstall.c:350
msgid "Disc Not Found"
msgstr "ディスクが見つかりません"

#: ../loader2/cdinstall.c:422
msgid "Cannot find kickstart file on CDROM."
msgstr "ディスクROM 上にキックスタートファイルが見つかりません。"

#: ../loader2/copy.c:51 ../loader2/method.c:121 ../loader2/method.c:347
#, c-format
msgid "Failed to read directory %s: %s"
msgstr "%s ディレクトリの読み込みに失敗: %s"

#: ../loader2/driverdisk.c:138
msgid "Loading"
msgstr "読み込み中"

#: ../loader2/driverdisk.c:138
msgid "Reading driver disk..."
msgstr "ドライバーディスクを読み込み中..."

#: ../loader2/driverdisk.c:230 ../loader2/driverdisk.c:262
msgid "Driver Disk Source"
msgstr "ドライバーディスクソース"

#: ../loader2/driverdisk.c:231
msgid ""
"You have multiple devices which could serve as sources for a driver disk.  "
"Which would you like to use?"
msgstr ""
"ドライバーディスク用のソースとして使用できるデバイスが複数あります。使用する"
"デバイスを指定してください。"

#: ../loader2/driverdisk.c:263
msgid ""
"There are multiple partitions on this device which could contain the driver "
"disk image.  Which would you like to use?"
msgstr ""
"このデバイスにはドライバーディスクのイメージを格納できるパーティションが複数"
"あります。使用するパーティションを指定してください。"

#: ../loader2/driverdisk.c:300
msgid "Failed to mount partition."
msgstr "パーティションのマウントに失敗しました。"

#: ../loader2/driverdisk.c:308
msgid "Select driver disk image"
msgstr "ドライバーディスクイメージを選択してください"

#: ../loader2/driverdisk.c:309
msgid "Select the file which is your driver disk image."
msgstr "ドライバーディスクイメージのファイルを選択してください。"

#: ../loader2/driverdisk.c:338
msgid "Failed to load driver disk from file."
msgstr "ファイルからのドライバーディスク読み込みに失敗しました。"

#: ../loader2/driverdisk.c:349
#, c-format
msgid "Insert your driver disk into /dev/%s and press \"OK\" to continue."
msgstr "ドライバーディスクを /dev/%s に挿入し、「OK」 を押して続行します。"

#: ../loader2/driverdisk.c:351
msgid "Insert Driver Disk"
msgstr "ドライバーディスクを挿入"

#: ../loader2/driverdisk.c:366
msgid "Failed to mount driver disk."
msgstr "ドライバーディスクのマウントに失敗しました。"

#: ../loader2/driverdisk.c:376
#, c-format
msgid "Driver disk is invalid for this release of %s."
msgstr "この %s のリリースにはドライバーディスクは無効です。"

#: ../loader2/driverdisk.c:439
msgid "Manually choose"
msgstr "手動で選択"

#: ../loader2/driverdisk.c:440
msgid "Load another disk"
msgstr "別のディスクをロード"

#: ../loader2/driverdisk.c:441
msgid ""
"No devices of the appropriate type were found on this driver disk.  Would "
"you like to manually select the driver, continue anyway, or load another "
"driver disk?"
msgstr ""
"このドライバーディスク上では適切なタイプのデバイスが見つかりませんでした。手"
"動でドライバーを選択する、このまま続行する、別のドライバーディスクを読み込"
"む、のいずれかを指定してください。"

#: ../loader2/driverdisk.c:479
msgid "Driver disk"
msgstr "ドライバーディスク"

#: ../loader2/driverdisk.c:480
msgid "Do you have a driver disk?"
msgstr "ドライバーディスクがありますか?"

#: ../loader2/driverdisk.c:489
msgid "More Driver Disks?"
msgstr "他のドライバーディスク?"

#: ../loader2/driverdisk.c:490
msgid "Do you wish to load any more driver disks?"
msgstr "他のドライバーディスクを読み込みますか?"

#: ../loader2/driverdisk.c:535 ../loader2/driverdisk.c:572
#: ../loader2/hdinstall.c:470 ../loader2/kickstart.c:132
#: ../loader2/kickstart.c:142 ../loader2/kickstart.c:185
#: ../loader2/kickstart.c:529 ../loader2/modules.c:336
#: ../loader2/modules.c:348 ../loader2/net.c:1647 ../loader2/net.c:1670
#: ../loader2/nfsinstall.c:342 ../loader2/urlinstall.c:437
#: ../loader2/urlinstall.c:446 ../loader2/urlinstall.c:455
msgid "Kickstart Error"
msgstr "キックスタートエラー"

#: ../loader2/driverdisk.c:536
#, c-format
msgid "Unknown driver disk kickstart source: %s"
msgstr "キックスタートの元となるドライバーディスクが不明です: %s"

#: ../loader2/driverdisk.c:573
#, c-format
msgid ""
"The following invalid argument was specified for the kickstart driver disk "
"command: %s:%s"
msgstr ""
"キックスタートのドライバーディスクコマンドに不正な引数が指定されました: %s %s"

#: ../loader2/driverselect.c:65
#, c-format
msgid ""
"Please enter any parameters which you wish to pass to the %s module "
"separated by spaces.  If you don't know what parameters to supply, skip this "
"screen by pressing the \"OK\" button."
msgstr ""
"%s モジュールに渡すパラメーターを空白で区切って入力してください。どのパラメー"
"ターを使用するのかわからない場合、「OK」 ボタンを押してこの画面をスキップして"
"ください。"

#: ../loader2/driverselect.c:85
msgid "Enter Module Parameters"
msgstr "モジュールパラメーターを入力"

#: ../loader2/driverselect.c:172
msgid "No drivers found"
msgstr "ドライバーが見つかりません"

#: ../loader2/driverselect.c:172
msgid "Load driver disk"
msgstr "ドライバーディスクの読み込み"

#: ../loader2/driverselect.c:173
msgid ""
"No drivers were found to manually insert.  Would you like to use a driver "
"disk?"
msgstr ""
"手動で挿入するドライバーが見つかりませんでした。ドライバーディスクを使用しま"
"すか?"

#: ../loader2/driverselect.c:191
msgid ""
"Please select the driver below which you wish to load.  If it does not "
"appear and you have a driver disk, press F2."
msgstr ""
"読み込むドライバーを下記から選択してください。必要なドライバーが表示されてい"
"ないがドライバーディスクがある場合、<F2> を押してください。"

#: ../loader2/driverselect.c:200
msgid "Specify optional module arguments"
msgstr "オプションのモジュール引数を指定してください"

#: ../loader2/driverselect.c:225
msgid "Select Device Driver to Load"
msgstr "読み込むデバイスドライバーを選択してください"

#: ../loader2/hdinstall.c:80 ../loader2/nfsinstall.c:168
#: ../loader2/urlinstall.c:274
msgid "Media Detected"
msgstr "検出されたメディア"

#: ../loader2/hdinstall.c:81 ../loader2/nfsinstall.c:169
#: ../loader2/urlinstall.c:275
msgid "Local installation media detected..."
msgstr "ローカルのインストールメディアが検出されました..."

#: ../loader2/hdinstall.c:137 ../loader2/nfsinstall.c:293
#: ../loader2/urlinstall.c:192
#, c-format
msgid ""
"The %s installation tree in that directory does not seem to match your boot "
"media."
msgstr ""
"そのディレクトリの %s インストールツリーはブートメディアと適合しないようで"
"す。"

#: ../loader2/hdinstall.c:253
msgid ""
"An error occured reading the install from the ISO images. Please check your "
"ISO images and try again."
msgstr ""
"ISO イメージからのインストールを読み込み中にエラーが発生しました。ISO イメー"
"ジを確認してからやり直してください。"

#: ../loader2/hdinstall.c:335
msgid ""
"You don't seem to have any hard drives on your system! Would you like to "
"configure additional devices?"
msgstr ""
"このシステムにはハードドライブがないようです。追加デバイスの設定を行いますか?"

#: ../loader2/hdinstall.c:349
#, c-format
msgid ""
"What partition and directory on that partition hold the CD (iso9660) images "
"for %s? If you don't see the disk drive you're using listed here, press F2 "
"to configure additional devices."
msgstr ""
"%s 用 ディスク (iso9660) イメージを格納しているパーティション及びそのパーティ"
"ション内のディレクトリを指定してください。一覧に使用しているディスクドライブ"
"がなければ、 F2 を押して追加デバイスの設定を行ってください。"

#: ../loader2/hdinstall.c:372
msgid "Directory holding images:"
msgstr "イメージを格納しているディレクトリ:"

#: ../loader2/hdinstall.c:400
msgid "Select Partition"
msgstr "パーティションを選択してください"

#: ../loader2/hdinstall.c:439
#, c-format
msgid "Device %s does not appear to contain %s CDROM images."
msgstr "デバイス %s には %s ディスクROM イメージが含まれていないようです。"

#: ../loader2/hdinstall.c:471
#, c-format
msgid "Bad argument to HD kickstart method command %s: %s"
msgstr "HD キックスタートコマンド %s に対する引数が不正です: %s"

#: ../loader2/hdinstall.c:541 ../loader2/hdinstall.c:597
msgid "Cannot find kickstart file on hard drive."
msgstr "ハードドライブにキックスタートファイルが見つかりません。"

#: ../loader2/hdinstall.c:584
#, c-format
msgid "Cannot find hard drive for BIOS disk %s"
msgstr "BIOS ディスク %s 用のハードドライブが見つかりません"

#: ../loader2/kbd.c:136
msgid "Keyboard Type"
msgstr "キーボードの種類"

#: ../loader2/kbd.c:137
msgid "What type of keyboard do you have?"
msgstr "ご使用のキーボードの種類を指定してください。"

#: ../loader2/kickstart.c:133
#, c-format
msgid "Error opening kickstart file %s: %s"
msgstr "キックスタートファイル %sを開く際にエラー発生: %s"

#: ../loader2/kickstart.c:143
#, c-format
msgid "Error reading contents of kickstart file %s: %s"
msgstr "キックスタートファイル %s の読み込み時にエラー発生: %s"

#: ../loader2/kickstart.c:186
#, c-format
msgid "Error in %s on line %d of kickstart file %s."
msgstr "%s のエラーが %d 行目(キックスタートファイル %s の)で発生しました。"

#: ../loader2/kickstart.c:285
msgid "Cannot find ks.cfg on removable media."
msgstr "リムーバブルメディアに ks.cfg が見つかりません。"

#: ../loader2/kickstart.c:377
msgid ""
"Unable to download the kickstart file.  Please modify the kickstart "
"parameter below or press Cancel to proceed as an interactive installation."
msgstr ""
"キックスタートファイルをダウンロードすることができません。以下のキックスター"
"トパラメーターを変更するか、対話インストールとして続行するために Cancel を押"
"してください。"

#: ../loader2/kickstart.c:386
msgid "Error downloading kickstart file"
msgstr "キックスタートファイルをダウンロードする際にエラー発生"

#: ../loader2/kickstart.c:530
#, c-format
msgid "Bad argument to shutdown kickstart method command %s: %s"
msgstr "キックスタートコマンド %s のシャットダウンに対する引数が不正です: %s"

#: ../loader2/lang.c:64
#, c-format
msgid "Welcome to %s for %s - Rescue Mode"
msgstr "%s (%s 用)へようこそ - レスキューモード"

#: ../loader2/lang.c:65 ../loader2/loader.c:166
msgid ""
"  <Tab>/<Alt-Tab> between elements  | <Space> selects | <F12> next screen "
msgstr "  <Tab>/<Alt-Tab> 項目間の移動 | <Space> 選択 | <F12> 次の画面 "

#: ../loader2/lang.c:373
msgid "Choose a Language"
msgstr "言語を選択してください"

#: ../loader2/loader.c:123
msgid "Local CD/DVD"
msgstr "ローカル CD/DVD"

#: ../loader2/loader.c:125
msgid "Hard drive"
msgstr "ハードドライブ"

#: ../loader2/loader.c:126
msgid "NFS directory"
msgstr "NFS ディレクトリ:"

#: ../loader2/loader.c:156
#, c-format
msgid "Welcome to %s%n for %s"
msgstr "%s%n (%s 用)へようこそ"

#: ../loader2/loader.c:374 ../loader2/loader.c:410
msgid "Update Disk Source"
msgstr "ディスクソースの更新"

#: ../loader2/loader.c:375
msgid ""
"You have multiple devices which could serve as sources for an update disk.  "
"Which would you like to use?"
msgstr ""
"更新ディスク用のソースとして使用できるデバイスが複数あります。使用するデバイ"
"スを指定してください。"

#: ../loader2/loader.c:411
msgid ""
"There are multiple partitions on this device which could contain the update "
"disk image.  Which would you like to use?"
msgstr ""
"このデバイスにはアップデートディスクイメージを含んでいるかもしれないパーティ"
"ションが複数あります。使用するパーティションを指定してください。"

#: ../loader2/loader.c:429
#, c-format
msgid "Insert your updates disk into /dev/%s and press \"OK\" to continue."
msgstr "更新ディスクを /dev/%s に挿入してから 「OK」 を押して続行します。"

#: ../loader2/loader.c:431
msgid "Updates Disk"
msgstr "更新ディスク"

#: ../loader2/loader.c:446
msgid "Failed to mount updates disk"
msgstr "更新ディスクのマウントに失敗しました"

#: ../loader2/loader.c:449
msgid "Updates"
msgstr "更新"

#: ../loader2/loader.c:449
msgid "Reading anaconda updates..."
msgstr "anaconda の更新を読み込み中..."

#: ../loader2/loader.c:485
msgid ""
"Unable to download the updates image.  Please modify the updates location "
"below or press Cancel to proceed without updates.."
msgstr ""
"アップデートイメージをダウンロードできません。以下のアップデートの場所を編集"
"するか、キャンセルを押してアップデートを中止してください。"

#: ../loader2/loader.c:494
msgid "Error downloading updates image"
msgstr "アップデートイメージのダウンロード中にエラー発生"

#: ../loader2/loader.c:994
#, c-format
msgid "You do not have enough RAM to install %s on this machine."
msgstr "このマシンには %s のインストールに十分な RAM がありません。"

#: ../loader2/loader.c:1151
msgid "Rescue Method"
msgstr "レスキュー方法"

#: ../loader2/loader.c:1152
msgid "Installation Method"
msgstr "インストール方法"

#: ../loader2/loader.c:1154
msgid "What type of media contains the rescue image?"
msgstr "レスキューイメージが格納されているメディアタイプを指定してください。"

#: ../loader2/loader.c:1156
msgid "What type of media contains the packages to be installed?"
msgstr "インストール対象パッケージを含むメディアを指定してください。"

#: ../loader2/loader.c:1180
msgid "No driver found"
msgstr "ドライバーが見つかりません"

#: ../loader2/loader.c:1180
msgid "Select driver"
msgstr "ドライバーを選択してください"

#: ../loader2/loader.c:1181
msgid "Use a driver disk"
msgstr "ドライバーディスクを使用する"

#: ../loader2/loader.c:1182
msgid ""
"Unable to find any devices of the type needed for this installation type.  "
"Would you like to manually select your driver or use a driver disk?"
msgstr ""
"このインストールタイプに必要な種類のデバイスを見つけることができません。  手"
"動でドライバーを選択しますか、またはドライバーディスクを使用しますか?"

#: ../loader2/loader.c:1415
msgid "The following devices have been found on your system."
msgstr "システム上で次のデバイスが検出されました。"

#: ../loader2/loader.c:1417
msgid ""
"No device drivers have been loaded for your system.  Would you like to load "
"any now?"
msgstr ""
"デバイスドライバーはシステムにロードされていません。ドライバーをロードします"
"か?"

#: ../loader2/loader.c:1421
msgid "Devices"
msgstr "デバイス"

#: ../loader2/loader.c:1422
msgid "Done"
msgstr "終了"

#: ../loader2/loader.c:1423
msgid "Add Device"
msgstr "デバイスの追加"

#: ../loader2/loader.c:1600
#, c-format
msgid "loader has already been run.  Starting shell.\n"
msgstr "ローダーはすでに実行されています。シェルの起動中です。\n"

#: ../loader2/loader.c:1946
#, c-format
msgid "Running anaconda %s, the %s rescue mode - please wait...\n"
msgstr ""
"anaconda %s を %s レスキューモードで実行中 - しばらくお待ちください...\n"

#: ../loader2/loader.c:1948
#, c-format
msgid "Running anaconda %s, the %s system installer - please wait...\n"
msgstr ""
"anaconda %s・%s システムインストーラーを実行中 - しばらくお待ちください...\n"

#: ../loader2/mediacheck.c:62
#, c-format
msgid "Unable to find install image %s"
msgstr "インストールイメージ %s を検出できません。"

#: ../loader2/mediacheck.c:68
#, c-format
msgid "Checking \"%s\"..."
msgstr "\"%s\" のチェック中..."

#: ../loader2/mediacheck.c:70
#, c-format
msgid "Checking media now..."
msgstr "現在メディアをチェック中..."

#: ../loader2/mediacheck.c:102
msgid ""
"Unable to read the disc checksum from the primary volume descriptor.  This "
"probably means the disc was created without adding the checksum."
msgstr ""
"プライマリボリュームディスクリプターからディスクチェックサムを読み込めませ"
"ん。 恐らく、このディスクはチェックサムを追加しないで作成されています。"

#: ../loader2/mediacheck.c:109
msgid ""
"The image which was just tested has errors. This could be due to a corrupt "
"download or a bad disc.  If applicable, please clean the disc and try "
"again.  If this test continues to fail you should not continue the install."
msgstr ""
"今テストしたイメージはエラーを含んでいます。欠陥のあるダウンロードまたはディ"
"スク不良が原因の可能性があります。可能ならディスクを清掃してからやり直してく"
"ださい。このテストが失敗を繰り返す場合、インストールは中止すべきです。"

#: ../loader2/mediacheck.c:117
msgid "Success"
msgstr "成功"

#: ../loader2/mediacheck.c:118
msgid ""
"The image which was just tested was successfully verified.  It should be OK "
"to install from this media.  Note that not all media/drive errors can be "
"detected by the media check."
msgstr ""
"テストしたイメージの検証に成功しました。このメディアからのインストールは問題"
"ないでしょう。ただし、すべてのメディア・ドライブのエラーがメディアチェックで"
"発見できるとは限りません。"

#: ../loader2/method.c:390
#, c-format
msgid ""
"Would you like to perform a checksum test of the ISO image:\n"
"\n"
"   %s?"
msgstr ""
"ISO イメージのチェックサムテストを実施しますか: \n"
"\n"
"   %s?"

#: ../loader2/method.c:393
msgid "Checksum Test"
msgstr "チェックサムテスト"

#: ../loader2/modules.c:337
#, c-format
msgid "Bad argument to device kickstart method command %s: %s"
msgstr "デバイスキックスタートのコマンド %s に対する引数が不正です: %s"

#: ../loader2/modules.c:349
msgid "A module name must be specified for the kickstart device command."
msgstr "モジュール名がキックスタートデバイスコマンドに指定されていません。"

#: ../loader2/net.c:93
msgid ""
"Prefix must be between 1 and 32 for IPv4 networks or between 1 and 128 for "
"IPv6 networks"
msgstr ""
"プレフィックスは、IPv4 ネットワークなら 1 から 32 まで、IPv6 ネットワークな"
"ら 1 から 128 までの数値にしなければなりません。"

#: ../loader2/net.c:266
#, c-format
msgid ""
"%s is a wireless network adapter.  Please provide the ESSID and encryption "
"key needed to access your wireless network.  If no key is needed, leave this "
"field blank and the install will continue."
msgstr ""
"%s はワイヤレスネットワークアダプターです。ワイヤレスネットワークへのアクセス"
"に必要な ESSID と暗号キーを入力してください。暗号キーが必要ない場合は、この"
"フィールドを空白にしたままインストールを続行します。"

#: ../loader2/net.c:272
msgid "ESSID"
msgstr "ESSID"

#: ../loader2/net.c:273
msgid "Encryption Key"
msgstr "暗号キー"

#: ../loader2/net.c:276
msgid "Wireless Settings"
msgstr "ワイヤレスの設定"

#: ../loader2/net.c:311
msgid "Nameserver IP"
msgstr "ネームサーバーの IP"

#: ../loader2/net.c:315
msgid "Missing Nameserver"
msgstr "ネームサーバーの欠落"

#: ../loader2/net.c:316
msgid ""
"Your IP address request returned configuration information, but it did not "
"include a nameserver address. If you do not have this information, you can "
"leave the field blank and the install will continue."
msgstr ""
"IP アドレス要求は設定情報を返しましたが、それはネームサーバーのアドレスを含み"
"ませんでした。もしこの情報を持っていないなら、このフィールドを空白にしてイン"
"ストールを続行できます。"

#: ../loader2/net.c:338
msgid "Invalid IP Information"
msgstr "無効な IP 情報"

#: ../loader2/net.c:339
msgid "You entered an invalid IP address."
msgstr "無効な IP アドレスが入力されました。"

#: ../loader2/net.c:589 ../loader2/net.c:695
msgid "Network Error"
msgstr "ネットワークエラー"

#: ../loader2/net.c:590 ../loader2/net.c:696
msgid "There was an error configuring your network interface."
msgstr "ネットワークインターフェースの設定中にエラーが発生しました。"

#: ../loader2/net.c:776
msgid "Configure TCP/IP"
msgstr "TCP/IP の設定"

#: ../loader2/net.c:821
msgid "You must select at least one protocol (IPv4 or IPv6)."
msgstr "少くともプロトコルを 1 つ選択しなければなりません (IPv4 または IPv6)。"

#: ../loader2/net.c:828
msgid "IPv4 Needed for NFS"
msgstr "NFS には IPv4 が必要です"

#: ../loader2/net.c:829
msgid "NFS installation method requires IPv4 support."
msgstr "NFS インストールを行うには IPv4 のサポートが必要になります。"

#: ../loader2/net.c:950
msgid "IPv4 address:"
msgstr "IPv4 アドレス:"

#: ../loader2/net.c:962 ../loader2/net.c:1014 tmp/netconfig.glade.h:1
#: tmp/netpostconfig.glade.h:1
msgid "/"
msgstr "/"

#: ../loader2/net.c:1002
msgid "IPv6 address:"
msgstr "IPv6 アドレス:"

#: ../loader2/net.c:1063
msgid "Name Server:"
msgstr "ネームサーバー:"

#: ../loader2/net.c:1099
msgid ""
"Enter the IPv4 and/or the IPv6 address and prefix (address / prefix).  For "
"IPv4, the dotted-quad netmask or the CIDR-style prefix are acceptable. The "
"gateway and name server fields must be valid IPv4 or IPv6 addresses."
msgstr ""
"IPv4 および/または IPv6 アドレスとプレフィックス (アドレス / プレフィックス) "
"を入力してください。 IPv4 では 4 つの数字をドットで区切ったネットマスクか "
"CIDR 形式のプレフィックスが可能です。ゲートウェイとネームサーバーのフィールド"
"は、必ず有効な IPv4 または IPv6 アドレスでなければなりません。"

#: ../loader2/net.c:1115
msgid "Manual TCP/IP Configuration"
msgstr "手動による TCP/IP 設定"

#: ../loader2/net.c:1224 ../loader2/net.c:1230
msgid "Missing Information"
msgstr "情報が不足しています"

#: ../loader2/net.c:1225
msgid ""
"You must enter both a valid IPv4 address and a network mask or CIDR prefix."
msgstr ""
"有効な IPv4 アドレスとネットマスクまたは CIDR プレフィックスの両方を入力する"
"必要があります。"

#: ../loader2/net.c:1231
msgid "You must enter both a valid IPv6 address and a CIDR prefix."
msgstr ""
"有効な IPv6 アドレスと CIDR プレフィックスの両方を入力する必要があります。"

#: ../loader2/net.c:1552
msgid "Determining host name and domain..."
msgstr "ホスト名とドメインを確定中..."

#: ../loader2/net.c:1648
#, c-format
msgid "Bad argument to kickstart network command %s: %s"
msgstr "キックスタートネットワークコマンド %s に対する引数が不正です: %s"

#: ../loader2/net.c:1671
#, c-format
msgid "Bad bootproto %s specified in network command"
msgstr "ネットワークコマンドで不正な bootproto %s が指定されました"

#: ../loader2/net.c:1859
msgid "Networking Device"
msgstr "ネットワーク構築デバイス"

#: ../loader2/net.c:1860
msgid ""
"You have multiple network devices on this system. Which would you like to "
"install through?"
msgstr ""
"このシステムには複数のネットワークデバイスが存在します。インストールの際に使"
"用するデバイスを選択してください。"

#: ../loader2/nfsinstall.c:57
msgid "NFS server name:"
msgstr "NFS サーバー名:"

#: ../loader2/nfsinstall.c:60
#, c-format
msgid "%s directory:"
msgstr "%s ディレクトリ:"

#: ../loader2/nfsinstall.c:65
#, c-format
msgid "Please enter the server name and path to your %s images."
msgstr "%s イメージへのパスとサーバー名を入力してください。"

#: ../loader2/nfsinstall.c:67
msgid "NFS Setup"
msgstr "NFS 設定"

#: ../loader2/nfsinstall.c:154
msgid "Hostname specified with no DNS configured"
msgstr "ホスト名は DNS 設定しないと指定できません"

#: ../loader2/nfsinstall.c:215 ../loader2/urls.c:223
msgid "Retrieving"
msgstr "取り込み中"

#: ../loader2/nfsinstall.c:284
msgid "That directory could not be mounted from the server."
msgstr "そのディレクトリはサーバーからマウントできませんでした。"

#: ../loader2/nfsinstall.c:297
#, c-format
msgid "That directory does not seem to contain a %s installation tree."
msgstr "そのディレクトリは %s のインストールツリーを含んでいないようです。"

#: ../loader2/nfsinstall.c:343
#, c-format
msgid "Bad argument to NFS kickstart method command %s: %s"
msgstr "NFS キックスタートコマンド %s に対する引数が不正です: %s"

#: ../loader2/telnetd.c:89 ../loader2/telnetd.c:131
msgid "Telnet"
msgstr "Telnet"

#: ../loader2/telnetd.c:89
msgid "Waiting for telnet connection..."
msgstr "telnet 接続の待機中..."

#: ../loader2/telnetd.c:131
msgid "Running anaconda via telnet..."
msgstr "telnet 経由で anaconda を実行中..."

#: ../loader2/urlinstall.c:79
#, c-format
msgid "Unable to retrieve %s://%s/%s."
msgstr "%s://%s/%s の取り込みができません。"

#: ../loader2/urlinstall.c:186
msgid "Unable to retrieve the install image."
msgstr "インストールイメージを取り込むことができません。"

#: ../loader2/urlinstall.c:438
#, c-format
msgid "Bad argument to Url kickstart method command %s: %s"
msgstr "Url キックスタートコマンド %s に対する引数が不正です: %s"

#: ../loader2/urlinstall.c:447
msgid "Must supply a --url argument to Url kickstart method."
msgstr "Url キックスタート方法には、 --url の引数を使用する必要があります。"

#: ../loader2/urlinstall.c:456
#, c-format
msgid "Unknown Url method %s"
msgstr "%s は不明な Url 方法です"

#: ../loader2/urls.c:282
#, c-format
msgid "Please enter the URL containing the %s images on your server."
msgstr "イメージ %s を含んでいるサーバーのアドレスを入力してください。"

#: ../loader2/urls.c:293
msgid "Configure proxy"
msgstr "プロキシの設定"

#: ../loader2/urls.c:308
msgid "URL Setup"
msgstr "URLの設定"

#: ../loader2/urls.c:316
msgid "You must enter a URL."
msgstr "URLを入力してください。"

#: ../loader2/urls.c:322
msgid "URL must be either an ftp or http URL"
msgstr "URL は ftp か http のものである必要があります。"

#: ../loader2/urls.c:333
msgid "Unknown Host"
msgstr "不明なホスト"

#: ../loader2/urls.c:334
#, c-format
msgid "%s is not a valid hostname."
msgstr "%s は有効なホスト名ではありません。"

#: ../loader2/urls.c:380
msgid ""
"If you are using a HTTP proxy server enter the name of the HTTP proxy server "
"to use."
msgstr ""
"HTTP プロキシサーバーを使用している場合、使用する HTTP プロキシサーバーの名前"
"を入力してください。"

#: ../loader2/urls.c:395
msgid "Proxy Name:"
msgstr "プロキシ名:"

#: ../loader2/urls.c:400
msgid "Proxy Port:"
msgstr "プロキシのポート:"

#: ../loader2/urls.c:416
msgid "Further Setup"
msgstr "詳細設定"

#: ../loader2/windows.c:63
msgid "Loading SCSI driver"
msgstr "SCSI ドライバーを読み込み中"

#: ../loader2/windows.c:64
#, c-format
msgid "Loading %s driver..."
msgstr "%s ドライバーを読み込み中..."

#: tmp/account.glade.h:2
msgid "Root Password:"
msgstr "ルートパスワード:"

#: tmp/account.glade.h:3
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""
"root アカウントはシステムの管理に使用されます。 root ユーザーのパスワードを入"
"力してください。"

#: tmp/adddrive.glade.h:1
msgid "Add _ZFCP LUN"
msgstr "ZFCP LUN の追加 (_Z)"

#: tmp/adddrive.glade.h:2
msgid "Add _iSCSI target"
msgstr "iSCSI ターゲットの追加 (_I)"

#: tmp/adddrive.glade.h:4
msgid "Disable _dmraid device"
msgstr "dmraid デバイスを無効にする (_D)"

#: tmp/adddrive.glade.h:6
msgid "_Add drive"
msgstr "ドライブ追加 (_A)"

#: tmp/addrepo.glade.h:1
msgid "<b>Repository _name:</b>"
msgstr "<b>リポジトリ名 (_N):</b>"

#: tmp/addrepo.glade.h:2
msgid "Add Repository"
msgstr "リポジトリの追加"

#: tmp/addrepo.glade.h:3
msgid ""
"Please provide the location where your additional software can be installed "
"from."
msgstr "追加ソフトウェアのインストール元を入力してください。"

#: tmp/addrepo.glade.h:4
msgid "Proxy U_RL:"
msgstr "プロキシの_URL:"

#: tmp/addrepo.glade.h:5
msgid "Proxy pass_word:"
msgstr "プロキシのパスワード:"

#: tmp/addrepo.glade.h:6
msgid "Proxy u_sername:"
msgstr "プロキシのユーザー名 (_U):"

#: tmp/addrepo.glade.h:7
msgid "Repository _Mirror"
msgstr "リポジトリのミラー (_M)"

#: tmp/addrepo.glade.h:8
msgid "Repository _URL"
msgstr "レポジトリの URL (_U)"

#: tmp/addrepo.glade.h:9
msgid "_Proxy configuration"
msgstr "プロキシの設定"

#: tmp/anaconda.glade.h:1
msgid "Reboo_t"
msgstr "再起動(_T)"

#: tmp/anaconda.glade.h:3
msgid "_Next"
msgstr "次(_N)"

#: tmp/autopart.glade.h:1
msgid "<b>Resize _target:</b>"
msgstr "<b>リサイズの対象:</b>"

#: tmp/autopart.glade.h:2
msgid ""
"Installation requires partitioning of your hard drive.  By default, a "
"partitioning layout is chosen which is reasonable for most users.  You can "
"either choose to use this or create your own."
msgstr ""
"インストールには、ハードドライブのパーティション設定が必要です。デフォルトで"
"は、ほとんどのユーザーに適しているパーティション設定のレイアウトが選択されま"
"す。デフォルトのレイアウトを選択するか、自分で作成することができます。"

#: tmp/autopart.glade.h:3
msgid "Re_view and modify partitioning layout"
msgstr "パーティションレイアウトの再確認と変更(_V)"

#: tmp/autopart.glade.h:4
msgid "What drive would you like to _boot this installation from?"
msgstr "このインストールを起動するドライブ (_B):"

#: tmp/autopart.glade.h:5
msgid "Which Partition to resize"
msgstr "リサイズするパーティション"

#: tmp/autopart.glade.h:6
msgid ""
"Which partition would you like to resize to make room for your installation?"
msgstr ""
"インストール用の空き容量を作るために、サイズを変更するパーティションを選択し"
"てください。"

#: tmp/autopart.glade.h:7
msgid "_Advanced storage configuration"
msgstr "高度なストレージ設定 (_A)"

#: tmp/autopart.glade.h:8
msgid "_Encrypt system"
msgstr "システムを暗号化する (_E)"

#: tmp/autopart.glade.h:10
msgid "_Select the drive(s) to use for this installation."
msgstr "このインストールに使用するドライブを選択してください(_S)。"

#: tmp/blwhere.glade.h:1
msgid "/boot"
msgstr "/boot"

#: tmp/blwhere.glade.h:2
msgid "BIOS Drive Order"
msgstr "BIOS ドライブ順"

#: tmp/blwhere.glade.h:3
msgid "Boot loader device"
msgstr "ブートローダーデバイス"

#: tmp/blwhere.glade.h:4
msgid "First BIOS drive:"
msgstr "第一 BIOS ドライブ:"

#: tmp/blwhere.glade.h:5
msgid "Fourth BIOS drive:"
msgstr "第四 BIOS ドライブ:"

#: tmp/blwhere.glade.h:6
msgid "MBR"
msgstr "MBR"

#: tmp/blwhere.glade.h:7
msgid "Second BIOS drive:"
msgstr "第二 BIOS ドライブ:"

#: tmp/blwhere.glade.h:8
msgid "Third BIOS drive:"
msgstr "第三 BIOS ドライブ:"

#: tmp/blwhere.glade.h:9
msgid "Where would you like to install the boot loader for your system?"
msgstr "ブートローダーのインストール場所を指定してください。"

#: tmp/detailed-dialog.glade.h:1
msgid "Info"
msgstr "情報"

#: tmp/detailed-dialog.glade.h:2
msgid "_Details"
msgstr "詳細 (_D)"

#: tmp/exnSave.glade.h:2
msgid "Select A File"
msgstr "ファイルを選択してください"

#: tmp/exnSave.glade.h:3
msgid "Select a destination for the exception information."
msgstr "除外情報の送信先を選択してください。"

#: tmp/exnSave.glade.h:4
msgid "_Disk"
msgstr "ディスク (_D)"

#: tmp/exnSave.glade.h:5
msgid "_Remote"
msgstr "リモート (_R)"

#: tmp/GroupSelector.glade.h:1
msgid ""
"Some packages associated with this group are not required to be installed "
"but may provide additional functionality.  Please choose the packages which "
"you would like to have installed."
msgstr ""
"このグループに関連付けられるいくつかのパッケージは、インストールする必要があ"
"りません。しかし、インストールすることによって追加機能を提供します。どのパッ"
"ケージをインストールするか選択してください。"

#: tmp/GroupSelector.glade.h:2
msgid "_Deselect"
msgstr "選択解除 (_D)"

#: tmp/GroupSelector.glade.h:3
msgid "_Deselect all optional packages"
msgstr "すべての追加パッケージの選択を解除 (_D)"

#: tmp/GroupSelector.glade.h:4
msgid "_Optional packages"
msgstr "追加パッケージ (_O)"

#: tmp/GroupSelector.glade.h:5
msgid "_Select"
msgstr "選択 (_S)"

#: tmp/GroupSelector.glade.h:6
msgid "_Select all optional packages"
msgstr "すべての追加パッケージを選択 (_S)"

#: tmp/GroupSelector.glade.h:7
msgid "dialog1"
msgstr "ダイアログ1"

#: tmp/instkey.glade.h:2
#, no-c-format
msgid "%(instkey)s:"
msgstr "%(instkey):"

#: tmp/instkey.glade.h:4
#, no-c-format
msgid "Please enter your %(instkey)s."
msgstr "%(instkey)を入力してください。"

#: tmp/iscsi-config.glade.h:1
msgid "<b>_Password:</b>"
msgstr "<b>パスワード (_P):</b>"

#: tmp/iscsi-config.glade.h:2
msgid "<b>_Target IP Address:</b>"
msgstr "<b>ターゲット IP アドレス(_T):</b>"

#: tmp/iscsi-config.glade.h:3
msgid "<b>_Username:</b>"
msgstr "<b>ユーザー名 (_U):</b>"

#: tmp/iscsi-config.glade.h:4
msgid "<b>iSCSI Initiator _Name:</b>"
msgstr "<b>iSCSI イニシエーター名(_N):</b>"

#: tmp/iscsi-config.glade.h:7
msgid "_Add target"
msgstr "ターゲットの追加 (_A)"

#: tmp/liveinst.desktop.in.h:1
msgid "Install"
msgstr "インストール"

#: tmp/liveinst.desktop.in.h:2
msgid "Install the live CD to your hard disk"
msgstr "ライブ ディスク をハードディスクにインストール"

#: tmp/liveinst.desktop.in.h:3
msgid "Install to Hard Drive"
msgstr "ハードドライブにインストール"

#: tmp/lukspassphrase.glade.h:1
msgid ""
"Choose a passphrase for this encrypted partition. You will be prompted for "
"the passphrase during system boot."
msgstr ""
"暗号化されたデバイス用のパスフレーズを選択してください。システムのブート中に"
"選択したパスフレーズの入力を求められます。"

#: tmp/lukspassphrase.glade.h:2
msgid "Confirm passphrase:"
msgstr "パスフレーズの確認:"

#: tmp/lukspassphrase.glade.h:3
msgid "Enter passphrase for encrypted partition"
msgstr "暗号化されたパーティション用のパスフレーズを入力してください。"

#: tmp/lukspassphrase.glade.h:4
msgid "Enter passphrase:"
msgstr "パスフレーズ:"

#: tmp/netconfig.glade.h:2
msgid "<b>Gateway:</b>"
msgstr "<b>ゲートウェイ:</b>"

#: tmp/netconfig.glade.h:3
msgid "<b>IPv4 Address:</b>"
msgstr "<b>IPv4 アドレス:</b>"

#: tmp/netconfig.glade.h:4
msgid "<b>IPv6 Address:</b>"
msgstr "<b>IPv6 アドレス:</b>"

#: tmp/netconfig.glade.h:5
msgid "<b>Nameserver:</b>"
msgstr "<b>ネームサーバー:</b>"

#: tmp/netconfig.glade.h:6
msgid "<b>_Interface:</b>"
msgstr "<b>インターフェース (_I):</b>"

#: tmp/netconfig.glade.h:7
msgid "Enable IPv_4 support"
msgstr "IPv4 サポートを有効にする (_4)"

#: tmp/netconfig.glade.h:8
msgid "Enable IPv_6 support"
msgstr "IPv6 サポートを有効にする (_6)"

#: tmp/netconfig.glade.h:9
msgid "Enable network interface"
msgstr "ネットワークインターフェースを有効にする"

#: tmp/netconfig.glade.h:10
msgid ""
"This requires that you have an active network connection during the "
"installation process.  Please configure a network interface."
msgstr ""
"これを行うにはインストール中にアクティブなネットワーク接続が必要になります。"
"ネットワークインターフェースを設定してください。"

#: tmp/netconfig.glade.h:11
msgid "Use _dynamic IP configuration (DHCP)"
msgstr "動的 IP 設定 (DHCP) を使用する (_D)"

#: tmp/netpostconfig.glade.h:2
msgid "<b>Description Goes Here</b>"
msgstr "<b>説明はここに入ります</b>"

#: tmp/netpostconfig.glade.h:3
msgid "<b>Hardware address: DE:AD:00:BE:EF:00</b>"
msgstr "<b>ハードウェアのアドレス: DE:AD:00:BE:EF:00</b>"

#: tmp/netpostconfig.glade.h:8
msgid "Edit Interface"
msgstr "インターフェースを編集"

#: tmp/netpostconfig.glade.h:11
msgid "Encryption Key:"
msgstr "暗号キー:"

#: tmp/netpostconfig.glade.h:13
msgid "Manual configuration"
msgstr "手動による設定"

#: tmp/netpostconfig.glade.h:14
msgid "Point to Point (IP):"
msgstr "ポイントツーポイント (IP):"

#: tmp/tasksel.glade.h:1
msgid "Customize _later"
msgstr "後でカスタマイズする(_l)"

#: tmp/tasksel.glade.h:2
msgid ""
"Please select any additional repositories that you want to use for software "
"installation."
msgstr ""
"ソフトウェアのインストールに使用する予定の追加リポジトリを選択してください。"

#: tmp/tasksel.glade.h:5
msgid ""
"You can further customize the software selection now, or after install via "
"the software management application."
msgstr ""
"ここで、ソフトウェア選択をさらに詳細にカスタマイズすることができます。また、"
"インストール完了後にソフトウェア管理アプリケーションから行うことも可能です。"

#: tmp/tasksel.glade.h:6
msgid "_Add additional software repositories"
msgstr "追加でソフトウェアリポジトリを加える (_A)"

#: tmp/tasksel.glade.h:7
msgid "_Customize now"
msgstr "今すぐカスタマイズする(_C)"

#: tmp/tasksel.glade.h:8
msgid "_Modify repository"
msgstr "リポジトリの編集 (_M)"

#: tmp/zfcp-config.glade.h:1
msgid "<b>Device number:</b>"
msgstr "<b>デバイス番号:</b>"

#: tmp/zfcp-config.glade.h:2
msgid "<b>FCP LUN:</b>"
msgstr "<b>FCP LUN:</b>"

#: tmp/zfcp-config.glade.h:3
msgid "<b>WWPN:</b>"
msgstr "<b>WWPN:</b>"

#: tmp/zfcp-config.glade.h:4
msgid "Add FCP device"
msgstr "FCP デバイスの追加"

#. generated from lang-table
msgid "Afrikaans"
msgstr "アフリカーンス語"

#. generated from lang-table
msgid "Arabic"
msgstr "アラビア語"

#. generated from lang-table
msgid "Assamese"
msgstr "アッサム語"

#. generated from lang-table
msgid "Bengali"
msgstr "ベンガル語"

#. generated from lang-table
msgid "Bengali(India)"
msgstr "ベンガル語 (インド)"

#. generated from lang-table
msgid "Bulgarian"
msgstr "ブルガリア語"

#. generated from lang-table
msgid "Catalan"
msgstr "カタロニア語"

#. generated from lang-table
msgid "Chinese(Simplified)"
msgstr "中国語 (簡体)"

#. generated from lang-table
msgid "Chinese(Traditional)"
msgstr "中国語 (繁体)"

#. generated from lang-table
msgid "Croatian"
msgstr "クロアチア語"

#. generated from lang-table
msgid "Czech"
msgstr "チェコ語"

#. generated from lang-table
msgid "Danish"
msgstr "デンマーク語"

#. generated from lang-table
msgid "Dutch"
msgstr "オランダ語"

#. generated from lang-table
msgid "English"
msgstr "英語"

#. generated from lang-table
msgid "Estonian"
msgstr "エストニア語"

#. generated from lang-table
msgid "Finnish"
msgstr "フィンランド語"

#. generated from lang-table
msgid "French"
msgstr "フランス語"

#. generated from lang-table
msgid "German"
msgstr "ドイツ語"

#. generated from lang-table
msgid "Greek"
msgstr "ギリシャ語"

#. generated from lang-table
msgid "Gujarati"
msgstr "グジャラート語"

#. generated from lang-table
msgid "Hindi"
msgstr "ヒンディー語"

#. generated from lang-table
msgid "Hungarian"
msgstr "ハンガリー語"

#. generated from lang-table
msgid "Icelandic"
msgstr "アイスランド語"

#. generated from lang-table
msgid "Iloko"
msgstr "Iloko"

#. generated from lang-table
msgid "Indonesian"
msgstr "インドネシア語"

#. generated from lang-table
msgid "Italian"
msgstr "イタリア語"

#. generated from lang-table
msgid "Japanese"
msgstr "日本語"

#. generated from lang-table
msgid "Kannada"
msgstr "カンナダ語"

#. generated from lang-table
msgid "Korean"
msgstr "韓国語"

#. generated from lang-table
msgid "Macedonian"
msgstr "マケドニア語"

#. generated from lang-table
msgid "Malay"
msgstr "マレー語"

#. generated from lang-table
msgid "Malayalam"
msgstr "マラヤーラム語"

#. generated from lang-table
msgid "Marathi"
msgstr "マラーティー語"

#. generated from lang-table
msgid "Norwegian"
msgstr "ノルウェー語"

#. generated from lang-table
msgid "Northern Sotho"
msgstr "北ソト語"

#. generated from lang-table
msgid "Oriya"
msgstr "オリヤー語"

#. generated from lang-table
msgid "Persian"
msgstr "ペルシア語"

#. generated from lang-table
msgid "Polish"
msgstr "ポーランド語"

#. generated from lang-table
msgid "Portuguese"
msgstr "ポルトガル語"

#. generated from lang-table
msgid "Portuguese(Brazilian)"
msgstr "ポルトガル語(ブラジル系)"

#. generated from lang-table
msgid "Punjabi"
msgstr "パンジャブ語"

#. generated from lang-table
msgid "Romanian"
msgstr "ルーマニア語"

#. generated from lang-table
msgid "Russian"
msgstr "ロシア語"

#. generated from lang-table
msgid "Serbian"
msgstr "セルビア語"

#. generated from lang-table
msgid "Serbian(Latin)"
msgstr "セルビア語(ラテン系)"

#. generated from lang-table
msgid "Sinhala"
msgstr "Sinhala"

#. generated from lang-table
msgid "Slovak"
msgstr "スロバキア語"

#. generated from lang-table
msgid "Slovenian"
msgstr "スロヴェニア語"

#. generated from lang-table
msgid "Spanish"
msgstr "スペイン語"

#. generated from lang-table
msgid "Swedish"
msgstr "スウェーデン語"

#. generated from lang-table
msgid "Tamil"
msgstr "タミル語"

#. generated from lang-table
msgid "Telugu"
msgstr "テルグ語"

#. generated from lang-table
msgid "Turkish"
msgstr "トルコ語"

#. generated from lang-table
msgid "Ukrainian"
msgstr "ウクライナ語"

#. generated from lang-table
msgid "Vietnamese"
msgstr "ベトナム語"

#. generated from lang-table
msgid "Welsh"
msgstr "ウェールズ語"

#. generated from lang-table
msgid "Zulu"
msgstr "ズールー語"

#~ msgid "Mouse Configuration"
#~ msgstr "マウスの設定"

#~ msgid "/dev/ttyS0 (COM1 under DOS)"
#~ msgstr "/dev/ttyS0 (DOS では COM1)"

#~ msgid "/dev/ttyS1 (COM2 under DOS)"
#~ msgstr "/dev/ttyS1 (DOS では COM2)"

#~ msgid "/dev/ttyS2 (COM3 under DOS)"
#~ msgstr "/dev/ttyS2 (DOS では COM3)"

#~ msgid "/dev/ttyS3 (COM4 under DOS)"
#~ msgstr "/dev/ttyS3 (DOS では COM4)"

#~ msgid "_Model"
#~ msgstr "モデル(_M)"

#~ msgid "_Emulate 3 buttons"
#~ msgstr "3 ボタンマウスのエミュレート(_E)"

#~ msgid "Select the appropriate mouse for the system."
#~ msgstr "システム用の適切なマウスを選択してください。"

#~ msgid "What device is your mouse located on?"
#~ msgstr "マウスを接続するデバイスを指定してください。"

#~ msgid "Which model mouse is attached to this computer?"
#~ msgstr "このコンピューターに接続されるマウスのモデルを選択してください。"

#~ msgid "Emulate 3 Buttons?"
#~ msgstr "3 ボタンマウスをエミュレートしますか?"

#~ msgid "Mouse Selection"
#~ msgstr "マウスの選択"

#~ msgid "NFS"
#~ msgstr "NFS"

#~ msgid "Error pulling second part of kickstart config: %s!"
#~ msgstr "キックスタート設定の第2部分を取り出し中にエラー: %s!"

#~ msgid "Unknown install method"
#~ msgstr "不明なインストール方法です"

#~ msgid ""
#~ "You have specified an install method which isn't supported by anaconda."
#~ msgstr "anaconda がサポートしていないインストール方法が指定されています。"

#~ msgid "unknown install method: %s"
#~ msgstr "不明なインストール方法です: %s"

#~ msgid ""
#~ "You have chosen to remove all partitions (ALL DATA) on the following "
#~ "drives:%s\n"
#~ "Are you sure you want to do this?"
#~ msgstr ""
#~ "次のドライブにあるすべてのパーティション(全データ)を削除することを選択して"
#~ "います:%s\n"
#~ "本当にすべて削除しますか?"

#~ msgid ""
#~ "You have chosen to remove all Linux partitions (and ALL DATA on them) on "
#~ "the following drives:%s\n"
#~ "Are you sure you want to do this?"
#~ msgstr ""
#~ "次のドライブにあるすべての Linux パーティション(及びそのパーティションにあ"
#~ "る全データ)を削除することを選択しています:%s\n"
#~ "本当にすべてを削除しますか?"

#~ msgid "Checking for Bad Blocks"
#~ msgstr "不良ブロックを検査中"

#~ msgid ""
#~ "Bad blocks have been detected on device /dev/%s. We do not recommend you "
#~ "use this device.\n"
#~ "\n"
#~ "Press <Enter> to exit the installer."
#~ msgstr ""
#~ "デバイス /dev/%s 上に不良ブロックが検出されました。このデバイスの使用は推"
#~ "薦できません。\n"
#~ "\n"
#~ "「Enter」 を押してインストーラを終了してください。"

#~ msgid ""
#~ "An error occurred searching for bad blocks on %s.  This problem is "
#~ "serious, and the install cannot continue.\n"
#~ "\n"
#~ "Press <Enter> to exit the installer."
#~ msgstr ""
#~ "%s 上の不良ブロックを検索中にエラーが発生しました。深刻な問題となるためイ"
#~ "ンストールを続行できません。\n"
#~ "\n"
#~ "「Enter」 を押してインストーラを終了してください。"

#~ msgid ""
#~ "Please insert a floppy now. All contents of the disk will be erased, so "
#~ "please choose your diskette carefully."
#~ msgstr ""
#~ "フロッピーディスクを挿入してください。ディスクの内容はすべて消去されますの"
#~ "で、注意してディスクを選択してください。"

#~ msgid ""
#~ "The file %s cannot be opened.  This is due to a missing file or perhaps a "
#~ "corrupt package.  Please verify your installation images and that you "
#~ "have all the required media.\n"
#~ "\n"
#~ "If you exit, your system will be left in an inconsistent state that will "
#~ "likely require reinstallation.\n"
#~ "\n"
#~ msgstr ""
#~ "ファイル %s を開けません。ファイルが欠落しているか、パッケージが破損してい"
#~ "る可能性があります。 インストールイメージ及び必要となるメディアがすべて"
#~ "揃っているか確認してください。\n"
#~ "\n"
#~ "終了するとシステムは矛盾した状態となり、再インストールを必要とする可能性が"
#~ "高くなります。\n"
#~ "\n"

#~ msgid ""
#~ "The installer has tried to mount image #%s, but cannot find it on the "
#~ "server.\n"
#~ "\n"
#~ "Please copy this image to the remote server's share path and click Retry. "
#~ "Click Exit to abort the installation."
#~ msgstr ""
#~ "インストーラーがイメージ #%s のマウントを試行しましたが、サーバー上に見つ"
#~ "かりません。\n"
#~ "\n"
#~ "このイメージをリモートサーバーの共有パスにコピーしてから再試行をクリックし"
#~ "てください。インストールを中止するには「終了」をクリックします。"

#~ msgid ""
#~ "The drive /dev/%s has more than 15 partitions on it.  The SCSI subsystem  "
#~ "in the Linux kernel does not allow for more than 15 partitions at this "
#~ "time.  You will not be able to make changes to the partitioning of this "
#~ "disk or use any partitions beyond /dev/%s15 in %s"
#~ msgstr ""
#~ "ドライブ /dev/%s には 15 を超えるパーティションがあります。現在の Linux "
#~ "カーネルの SCSI サブシステムは 15 を超えるパーティションを認めていません。"
#~ "このディスクのパーティション構成に変更を加えることはできず、/dev/%s15 を超"
#~ "える %s の どのパーティションも使用できません。"

#~ msgid "Low Memory"
#~ msgstr "メモリ不足"

#~ msgid ""
#~ "As you don't have much memory in this machine, we need to turn on swap "
#~ "space immediately. To do this we'll have to write your new partition "
#~ "table to the disk immediately. Is that OK?"
#~ msgstr ""
#~ "このマシンのメモリが不足しているため、ただちに swap 領域を有効にする必要が"
#~ "あります。新しいパーティションテーブルを直ちにディスクに書き込まなければな"
#~ "りません。 よろしいですか?"

#~ msgid "Save to Remote Host"
#~ msgstr "リモートホストに保存"

#~ msgid "Save Crash Dump"
#~ msgstr "クラッシュダンプの保存"

#~ msgid "Searching"
#~ msgstr "検索中"

#~ msgid "Searching for %s installations..."
#~ msgstr "%s インストールを検索中..."

#~ msgid "Connecting..."
#~ msgstr "接続中..."

#~ msgid ""
#~ "The file %s cannot be opened.  This is due to a missing file or perhaps a "
#~ "corrupt package.  Please verify your mirror contains all required "
#~ "packages, and try using a different one.\n"
#~ "\n"
#~ "If you exit, your system will be left in an inconsistent state that will "
#~ "likely require reinstallation.\n"
#~ "\n"
#~ msgstr ""
#~ "ファイル %s を開けません。ファイルが欠落しているか、パッケージが破損してい"
#~ "る可能性があります。 ご使用のミラーに必要なパッケージがすべて含まれている"
#~ "か確認し、また、別のミラーを使用してみてください。\n"
#~ "\n"
#~ "終了すると、システムは矛盾した状態のままになり再インストールが必要となる可"
#~ "能性があります。\n"
#~ "\n"

#~ msgid "VNC Password Error"
#~ msgstr "VNC パスワードのエラー"

#~ msgid ""
#~ "You need to specify a vnc password of at least 6 characters long.\n"
#~ "\n"
#~ "Press <return> to reboot your system.\n"
#~ msgstr ""
#~ "最低 6 文字の長さの vnc パスワードを指定する必要があります。\n"
#~ "\n"
#~ "[Enter] を押してシステムを再起動してください。\n"

#~ msgid "Unable to set vnc password - using no password!"
#~ msgstr "vnc パスワードを設定できません - パスワードを使用していません!"

#~ msgid "Make sure your password is at least 6 characters in length."
#~ msgstr "パスワードが 6 文字以上になっているか確認してください。"

#~ msgid "Please connect to %s to begin the install..."
#~ msgstr "インストールを開始するには %s に接続してください..."

#~ msgid "Please connect to begin the install..."
#~ msgstr "インストールを開始するために接続してください..."

#~ msgid "Set Root Password"
#~ msgstr "root パスワードを設定"

#~ msgid "Advanced Boot Loader Configuration"
#~ msgstr "高度なブートローダーの設定"

#~ msgid "Force LBA32"
#~ msgstr "LBA32 を強制使用"

#~ msgid "_Force LBA32 (not normally required)"
#~ msgstr "LBA32 の強制使用(通常は不必要)(_F)"

#~ msgid ""
#~ "If you wish to add default options to the boot command, enter them into "
#~ "the 'General kernel parameters' field."
#~ msgstr ""
#~ "ブートコマンドにデフォルトのオプションを追加したい場合は「一般カーネルパラ"
#~ "メーター」の領域に入力します。"

#~ msgid "_General kernel parameters"
#~ msgstr "一般カーネルパラメーター(_G)"

#~ msgid "The %s boot loader will be installed on /dev/%s."
#~ msgstr "%s ブートローダーは、/dev/%s 上にインストールされます。"

#~ msgid "No boot loader will be installed."
#~ msgstr "ブートローダーはインストールされません。"

#~ msgid "Configure advanced boot loader _options"
#~ msgstr "高度なブートローダーオプションの設定(_O)"

#~ msgid "Install Boot Loader record on:"
#~ msgstr "ブートローダーの記録をインストールする場所:"

#~ msgid "_Change Drive Order"
#~ msgstr "ドライブの順序を変更(_C)"

#~ msgid ""
#~ "Arrange the drives to be in the same order as used by the BIOS. Changing "
#~ "the drive order may be useful if you have multiple SCSI adapters or both "
#~ "SCSI and IDE adapters and want to boot from the SCSI device.\n"
#~ "\n"
#~ "Changing the drive order will change where the installation program "
#~ "locates the Master Boot Record (MBR)."
#~ msgstr ""
#~ "BIOS で使用されているのと同じ順でドライブを配置します。複数の SCSI アダプ"
#~ "ターや SCSI と IDE アダプターの組合せを使用していて、SCSI デバイスからブー"
#~ "トしたい場合にドライブの順序を変更するとよいかもしれません。\n"
#~ "\n"
#~ "ドライブ順を変更することは、インストールプログラムがマスターブートレコー"
#~ "ド (MBR)の位置を変えることになります。"

#~ msgid "About to Install"
#~ msgstr "インストール準備完了"

#~ msgid "Click next to begin installation of %s."
#~ msgstr "[次] をクリックすると %s. のインストールを開始します。"

#~ msgid ""
#~ "A complete log of the installation can be found in the file '%s' after "
#~ "rebooting your system.\n"
#~ "\n"
#~ "A kickstart file containing the installation options selected can be "
#~ "found in the file '%s' after rebooting the system."
#~ msgstr ""
#~ "インストール作業の完全なログは、システムを再起動した後に %s ファイルの中で"
#~ "見ることが出来ます。\n"
#~ "\n"
#~ "選択したインストールオプションを含むキックスタートファイルは、システムの再"
#~ "起動後、%sファイルの中にあります。"

#~ msgid "About to Upgrade"
#~ msgstr "アップグレード準備完了"

#~ msgid "Click next to begin upgrade of %s."
#~ msgstr "[次]をクリックすると %s. のアップグレードを開始します。"

#~ msgid ""
#~ "A complete log of the upgrade can be found in the file '%s' after "
#~ "rebooting your system."
#~ msgstr ""
#~ "アップグレード作業の完全なログは、システムの再起動後に%sファイルの中で見る"
#~ "ことができます。"

#~ msgid "Drive"
#~ msgstr "ドライブ"

#~ msgid "Model"
#~ msgstr "モデル"

#~ msgid "Netmask"
#~ msgstr "ネットマスク"

#~ msgid ""
#~ "You can configure the boot loader to boot other operating systems by "
#~ "selecting from the list. To add an operating systems that was not "
#~ "automatically detected, click 'Add.' To change the operating system "
#~ "booted by default, select 'Default' next to the desired operating system."
#~ msgstr ""
#~ "一覧から選択することで他のオペレーティングシステムをブートできるようにブー"
#~ "トローダーを設定することができます。 自動的には認識されない他のオペレー"
#~ "ティングシステムを追加するには、「追加」をクリックします。デフォルトでブー"
#~ "トするオペレーティングシステムを変更するには、そうしたいオペレーティングシ"
#~ "ステムの横の「デフォルト」を選択します。"

#~ msgid "Leave _unchanged (preserve data)"
#~ msgstr "変更しないで残す(データを保存)(_U)"

#~ msgid "_Format partition as:"
#~ msgstr "次のパーティションタイプでフォーマット(_F):"

#~ msgid "Check for _bad blocks?"
#~ msgstr "不良ブロックをチェックしますか(_B)?"

#~ msgid "Release notes are missing.\n"
#~ msgstr "リリースノートが見つかりません。\n"

#~ msgid "Release Notes"
#~ msgstr "リリースノート"

#~ msgid "Unable to load file!"
#~ msgstr "ファイルを読み込めません!"

#~ msgid "Installation to begin"
#~ msgstr "インストール開始"

#~ msgid ""
#~ "A complete log of your installation will be in %s after rebooting your "
#~ "system. You may want to keep this file for later reference."
#~ msgstr ""
#~ "インストールに関する全記録は、システムの再起動後 %s に保存されます。後日、"
#~ "参照できるようにこのファイルを保管しておくことをお勧めします。"

#~ msgid "Upgrade to begin"
#~ msgstr "アップグレード開始"

#~ msgid ""
#~ "A complete log of your upgrade will be in %s after rebooting your system. "
#~ "You may want to keep this file for later reference."
#~ msgstr ""
#~ "アップグレードに関する全記録は、システムの再起動後 %s に保存されます。後"
#~ "日、参照できるようにこのファイルを保管しておくことをお勧めします。"

#~ msgid "%s, %s, %s"
#~ msgstr "%s, %s, %s"

#~ msgid "%s, %s"
#~ msgstr "%s, %s"

#~ msgid "Check for bad blocks"
#~ msgstr "不良ブロックをチェックする"

#~ msgid ""
#~ "No hard drives have been found.  You probably need to manually choose "
#~ "device drivers for the installation to succeed.  Would you like to select "
#~ "drivers now?"
#~ msgstr ""
#~ "ハードドライブが検出されませんでした。インストールを成功させるには、恐らく"
#~ "デバイスドライバーを手動で選択する必要があります。ここでドライバーを選択し"
#~ "ますか?"

#~ msgid "FAILED"
#~ msgstr "失敗"

#~ msgid "PASSED"
#~ msgstr "パス"

#~ msgid "It is OK to install from this media."
#~ msgstr "このメディアからのインストールに問題はありません。"

#~ msgid "No checksum information available, unable to verify media."
#~ msgstr "チェックサムの情報がなく、メディアの検証ができません。"

#~ msgid "Media Check Result"
#~ msgstr "メディアチェックの結果"

#~ msgid ""
#~ "%s for the image:\n"
#~ "\n"
#~ "   %s"
#~ msgstr ""
#~ "イメージの %s\n"
#~ "\n"
#~ "   %s"

#~ msgid ""
#~ "The media check %s\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "%s のメディアチェック\n"
#~ "\n"
#~ "%s"

#~ msgid ""
#~ "Please enter the following information:\n"
#~ "\n"
#~ "    o the name or IP number of your %s server\n"
#~ "    o the directory on that server containing\n"
#~ "      %s for your architecture\n"
#~ msgstr ""
#~ "以下の情報を入力してください。\n"
#~ "\n"
#~ "    o %s サーバーの名前または IP アドレス\n"
#~ "    o そのサーバーにあるご使用のアーキテクチャ用の\n"
#~ "       %s を含むディレクトリ\n"

#~ msgid "FTP"
#~ msgstr "FTP"

#~ msgid "Web"
#~ msgstr "ウェブ"

#~ msgid "Web site name:"
#~ msgstr "Web サイト名:"

#~ msgid "Use non-anonymous ftp"
#~ msgstr "非 anonymous (匿名) FTP を使用"

#~ msgid "FTP Setup"
#~ msgstr "FTP の設定"

#~ msgid "HTTP Setup"
#~ msgstr "HTTP の設定"

#~ msgid "You must enter a server name."
#~ msgstr "サーバー名を入力してください。"

#~ msgid ""
#~ "If you are using non anonymous ftp, enter the account name and password "
#~ "you wish to use below."
#~ msgstr ""
#~ "anonymous (匿名)ではない FTP を使用している場合、使用するアカウント名およ"
#~ "びパスワードを以下に入力してください。"

#~ msgid "Account name:"
#~ msgstr "アカウント名:"

#~ msgid "Further HTTP Setup"
#~ msgstr "HTTP の追加設定"

#~ msgid "_Release Notes"
#~ msgstr "リリースノート(_R)"

#~ msgid "Exception Info"
#~ msgstr "例外の情報"

#~ msgid "_Exception details"
#~ msgstr "例外の詳細(_E)"

#~ msgid "_Save to floppy"
#~ msgstr "フロッピーに保存 (_S)"
